-- phpMyAdmin SQL Dump
-- version 4.3.8
-- http://www.phpmyadmin.net
--
-- Host: localhost
-- Generation Time: Nov 07, 2017 at 09:17 PM
-- Server version: 5.5.51-38.2
-- PHP Version: 5.6.30

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;

--
-- Database: `thescie5_theBreakerDb`
--

-- --------------------------------------------------------

--
-- Table structure for table `admin`
--

CREATE TABLE IF NOT EXISTS `admin` (
  `id` int(11) NOT NULL,
  `password` varchar(40) COLLATE utf8_unicode_ci NOT NULL DEFAULT '1234',
  `member` int(11) DEFAULT NULL,
  `pin` varchar(4) COLLATE utf8_unicode_ci DEFAULT NULL
) ENGINE=MyISAM AUTO_INCREMENT=2 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `admin`
--

INSERT INTO `admin` (`id`, `password`, `member`, `pin`) VALUES
(1, '27656cb712571718b93fc2d4842d6a2a', 1, '1234');

-- --------------------------------------------------------

--
-- Table structure for table `advisory_board`
--

CREATE TABLE IF NOT EXISTS `advisory_board` (
  `id` int(11) NOT NULL,
  `last_name` varchar(40) CHARACTER SET latin1 NOT NULL,
  `first_name` varchar(40) CHARACTER SET latin1 NOT NULL,
  `position` varchar(200) CHARACTER SET latin1 NOT NULL,
  `email` varchar(50) COLLATE utf8_unicode_ci DEFAULT NULL,
  `research_institute` varchar(200) COLLATE utf8_unicode_ci DEFAULT NULL,
  `is_editor` tinyint(1) DEFAULT '0'
) ENGINE=MyISAM AUTO_INCREMENT=11 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `advisory_board`
--

INSERT INTO `advisory_board` (`id`, `last_name`, `first_name`, `position`, `email`, `research_institute`, `is_editor`) VALUES
(5, 'Perron', 'Dr. Karl', 'Scientific and Science-communication Advisor – Group leader', '', 'BiOutils, University of Geneva', 0),
(6, 'Preuten', 'Dr. Tobias', 'Editorial Advisor – Review Operation Coordinator', '', 'OA Publishing Frontiers', 0),
(7, 'Caine', 'Alessandra', 'Strategic Advisor – Business Development', '', 'Commonland', 0),
(10, 'Guzzon', 'Oscar', 'Web Analyst Advisor - Librarian ', 'guzzonoscar@hotmail.com', 'State Archives, Bellinzona, Switzerland', 0);

-- --------------------------------------------------------

--
-- Table structure for table `authors`
--

CREATE TABLE IF NOT EXISTS `authors` (
  `id` int(11) NOT NULL,
  `last_name` varchar(40) CHARACTER SET utf8 NOT NULL,
  `first_name` varchar(20) COLLATE utf8_unicode_ci NOT NULL,
  `position` text COLLATE utf8_unicode_ci,
  `email` varchar(50) COLLATE utf8_unicode_ci DEFAULT NULL,
  `research_institute` varchar(250) COLLATE utf8_unicode_ci DEFAULT NULL,
  `field_research` varchar(200) COLLATE utf8_unicode_ci DEFAULT NULL,
  `general_comments` text COLLATE utf8_unicode_ci
) ENGINE=MyISAM AUTO_INCREMENT=118 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `authors`
--

INSERT INTO `authors` (`id`, `last_name`, `first_name`, `position`, `email`, `research_institute`, `field_research`, `general_comments`) VALUES
(30, 'Kilner', 'Rebecca', 'Professor', 'rmk1002@cam.ac.uk', 'Department of Zoology, University of Cambridge, UK', '', ''),
(25, 'Kyndt', 'Tina', 'Research Professor', 'tina.kyndt@ugent.be', 'Department Molecular Biotechnology, Ghent University (UGent), Ghent, Belgium', '', ''),
(26, 'Bowler', 'Chris', 'CNRS Director of Research ', 'cbowler@biologie.ens.fr', 'Ecole Normale Supérieure, PSL Research University, Institut de Biologie de l’Ecole Normale Supérieure (IBENS), Paris, France', '', 'Bowler is one of the Scientific Coordinators of the Tara Oceans project'),
(27, 'Bourrier', 'Vincent', 'Postdoctoral Research Fellow', 'vincent.bourrier@unige.ch', 'Department of Astronomy, Geneva Observatory, University of Geneva, Geneva, Switzerland', '', ''),
(28, 'Ryan', 'Tomás', 'Research Scientist', 'tr1@mit.edu', 'Howard Hughes Medical Institute and RIKEN-MIT Center for Neural Circuit Genetics, Picower Institute for Learning and Memory, Massachusetts Institute of Technology (MIT)', '', ''),
(29, 'Tonegawa', 'Susumu', 'Professor', 'tonegawa@mit.edu', 'Howard Hughes Medical Institute and RIKEN-MIT Center for Neural Circuit Genetics, Picower Institute for Learning and Memory, Massachusetts Institute of Technology (MIT)', '', 'Tonegawa won the <b>Nobel Prize for Physiology or Medicine in 1987</b>'),
(23, 'Rivera-Rivera', 'Carlos J.', 'PhD student', 'Carlos.RiveraRivera@unige.ch', 'Department of Genetics and Evolution, University of Geneva, Switzerland', '', ''),
(24, ' Dell''Aglio', 'Elisa', 'Postdoctoral Research Fellow', 'Elisa.Dellaglio@unige.ch', 'Department of Botany and Plant Biology, University of Geneva, Switzerland', '', ''),
(31, 'Kessler', 'Sébastien', 'Research Associate', 'Sebastien.Kessler@newcastle.ac.uk', 'Institute of Neuroscience, University of Newcastle, Newcastle upon Tyne, England', '', ''),
(32, 'Kramer', 'Dan', 'PhD student', 'daniel.kramer@berkeley.edu', 'Department of Molecular and Cellular Biology, University of Berkley, California, USA', '', ''),
(33, 'Da Rè', 'Caterina', 'Postdoctoral Research Fellow', 'caterina.dare@unige.ch', 'Department of Molecular Biology, University of Geneva, Switzerland', '', ''),
(34, 'Hulbert', 'Justin C.', 'Professor', 'jhulbert@bard.edu', 'Bard College, Annandale-on-Hudson, NY, USA', '', ''),
(35, 'Engelmann', 'Jan', 'Postdoctoral Research Fellow', 'jan_engelmann@eva.mpg.de', 'Max Planck Institute for Evolutionary Anthropology, Germany', '', ''),
(36, 'Robinson', 'Graham', 'Postdoctoral Research Fellow', 'Graham.Robinson@unige.ch', 'Department of Botany and Plant Biology, University of Geneva, Switzerland', '', ''),
(37, 'Straten', 'Per thor', 'Professor', 'Per.Thor.Straten@regionh.dk', 'Centre for Cancer Immune Therapy, Copenhagen University Hospital Herlev, Denmark', '', ''),
(38, 'Pizzato ', 'Massimo', 'Associate Professor', 'massimo.pizzato@unitn.it', 'Centre for Integrated Biology, University of Trento, Italy', '', ''),
(39, 'Ryan', 'Joanne', 'Postdoctoral Research Fellow', 'joanne.ryan@mcri.edu.au', 'Murdoch Childrens Research Institute (MCRI), Melbourne, Australia', '', ''),
(40, 'Costafrolaz', 'Jordan', 'PhD student', 'Jordan.Costafrolaz@unige.ch', 'Microbiology Unit, Department of Botany and Plant Biology, University of Geneva, Switzerland', '', ''),
(42, 'Korpela', 'Katri', 'PhD student', 'katri.korpela@helsinki.fi', 'Department of Bacteriology and Immunology, Immunobiology Research Program, University of Helsinki, Finland', '', ''),
(43, 'Sentandreu', 'Maria', 'Postdoctoral Research Fellow', 'Maria.Sentandreu@unige.ch', 'Department of Botany and Plant Biology, University of Geneva, Switzerland', '', ''),
(44, 'Stefanelli', 'Thomas', 'PhD student ', 'Thomas.Stefanelli@unige.ch', 'Department of Basic Neurosciences, University of Geneva, Switzerland', '', ''),
(45, 'Mendez', 'Pablo', 'Senior Lecturer', 'Pablo.Mendez@unige.ch', ' Department of Basic Neurosciences, University of Geneva, Switzerland', '', ''),
(46, 'Hankey', 'Jon', 'Director of Research and Development', 'jhankey@vtti.vt.edu', 'Research and Development, Virginia Tech Transportation Institute (VTTI), VA, USA', '', ''),
(47, 'Albright', 'Rebecca', 'Postdoctoral Research Scientist', 'ralbright@calacademy.org', 'Department of Global Ecology, Carnegie Institution for Science, Stanford University, CA, USA', '', ''),
(48, 'Green', 'Edward', 'Postdoctoral Research Fellow', 'e.green@dkfz-heidelberg.de', 'Marie Curie Intra European Fellowship, German Cancer Research Center, Germany', '', ''),
(49, 'Guido', 'Daniele', 'PhD student', 'Daniele.Guido@unige.ch', 'Department of Cell Physiology and Metabolism, University of Geneva, Switzerland', '', ''),
(50, 'Thackeray', 'Stephen', 'Freshwater Ecologist', 'sjtr@ceh.ac.uk', 'Centre for Ecology & Hydrology, Lancaster Environment Centre, Library Avenue, Bailrigg, Lancaster, LA1 4AP, UK ', '', ''),
(51, 'Burthe', 'Sarah', 'Animal Population Ecologist', 'sburthe@ceh.ac.uk', 'Coastal Seas Ecology Group, Centre for Ecology & Hydrology, Bush Estate, Penicuik, Midlothian, EH26 0QB, UK', '', ''),
(52, 'Hundey', 'Beth', 'Adjunct Research Professor & eLearning specialist', 'ehundey2@uwo.ca', 'Teaching Support Centre, Western University, London, Canada', '', ''),
(53, 'Moser', 'Katrina', 'Associate Professor', 'kmoser@uwo.ca', 'Department of Geography at Western University, London, Canada', '', ''),
(54, 'Novák', 'Lukáš', 'PhD student', 'niun42@gmail.com', 'Department of Parasitology, Faculty of Science, Charles University in Prague, Czech Republic', '', ''),
(55, 'Schwarzer', 'Martin', 'Postdoctoral Research Fellow', 'martin.schwarzer@ens-lyon.fr', 'Institut de génomique fonctionnelle de Lyon, Ecole Normale Supérieure de Lyon, CNRS and Université Claude Bernard Lyon, Lyon', '', ''),
(56, 'Foster', 'Taryn', 'Postdoctoral Research Fellow', 'taryn.foster@research.uwa.edu.au', 'School of Earth and Environment, The University of Western Australia, Australia', '', ''),
(57, 'Cubo', 'Nieves', 'PhD student', 'ncubo@ucm.es', 'Faculty of Pharmacy, University Complutense of Madrid, Spanish National Research Council, Madrid, Spain', '', ''),
(58, 'Gruber', 'Christian W.', 'Assistant Professor', 'c.gruber@uq.edu.au', ' School of Biomedical Sciences, The University of Queensland, Australia ', '', ''),
(59, 'Matone', 'Alice', 'Postdoctoral Research Fellow', 'matone@cosbi.eu', 'The Microsoft Research - University of Trento Centre for Computational and Systems Biology (COSBI)', '', ''),
(60, 'Idorn', 'Manja', 'PhD student', '', 'Centre for Cancer Immune Therapy, Copenhagen University Hospital Herlev, Denmark', '', ''),
(61, 'Longstaffe', 'Fred', 'Distinguished Professor ', '', 'Department of Earth Science, Western University, London, Canada', '', ''),
(62, 'Gründemann', 'Carsten', 'PD Dr. rer. nat.', 'carsten.gruendemann@uniklinik-freiburg.de', 'Environmental Health Sciences and Hospital Infection control, Universitätsklinikum Freiburg, Germany', '', ''),
(63, 'Laturney', 'Meghan', 'PhD student', 'm.e.laturney@rug.nl', 'Groningen Institute for Evolutionary Life Sciences, University of Groningen, The Netherlands', '', ''),
(65, 'Domyan', 'Eric', 'Assistant Professor', 'Eric.Domyan@uvu.edu', 'Utah Valley University, 800 W. University Parkway, Orem', 'Biology/Biotechnology', ''),
(66, 'Long', 'Stephen', 'Professor', 'slong@illinois.edu', 'Carl R. Woese Institute for Genomic Biology at Illinois, USA', 'Plant Biology and Crop Sciences', ''),
(67, 'Głowacka', 'Katarzyna', 'Postdoctoral Research Fellow', '', 'Carl R. Woese Institute for Genomic Biology at Illinois, USA', 'Plant Biology and Crop Sciences', ''),
(68, 'Kromdijk', 'Johannes', 'Postdoctoral Research Fellow', '', 'Carl R. Woese Institute for Genomic Biology at Illinois', 'Plant Biology and Crop Sciences', ''),
(69, 'Mandriota', 'Stefano', 'Research Director', 'stefano.fdg@gmail.com', 'Laboratoire de cancérogenèse environnementale, Fondation des Grangettes, Geneva, Switzerland', '', ''),
(70, 'Fitzpatrick', 'Teresa', 'Professor', 'Theresa.Fitzpatrick@unige.ch', 'Department of Botany and Plant Biology, University of Geneva, Switzerland', '', ''),
(71, 'Marzo', 'Aude', 'Postdoctoral Research fellow', 'a.marzo@ucl.ac.uk', 'University College London, Department Cellular and Developmental Biology, London, UK', '', ''),
(72, 'Noble', 'Peter', 'Professor', 'panoble@u.washington.edu', 'Department of Periodontics, University of Washington, Seattle, USA', '', ''),
(73, 'Pozhitkov', 'Alex', 'Research Scientist', '', 'Department of Oral Health Sciences, University of Washington, Seattle, USA', '', ''),
(74, 'Matthews', 'Gillian', 'Postdoctoral Research Fellow', 'gmatthew@mit.edu', 'Simons Center for the Social Brain, Massachusetts Institute of Technology (MIT), Boston, USA', '', ''),
(75, 'Salinas', 'Patricia', 'Professor', 'p.salinas@ucl.ac.uk', 'Department of Cell and Developmental Biology, University College London, London WC1E 6BT, UK', '', ''),
(76, 'McLeod', 'Faye ', 'Postdoctoral Research Fellow', 'f.mcleod@ucl.ac.uk', 'Department of Cell and Developmental Biology, University College London, London WC1E 6BT, UK', '', ''),
(77, 'Anderson', 'Rozalyn', 'Associate Professor', 'rmanderson5@wisc.edu', 'Department of Medicine, School of Medicine and Public Health, University of Wisconsin Madison, & Geriatric Research, Education, and Clinical Center, William S. Middleton Memorial Veterans Hospital, Madison WI', '', ''),
(78, 'Bian', 'Lin', 'PhD student', 'linbian2@illinois.edu', 'University of Illinois at Urbana-Champaign, Illinois, USA', '', ''),
(79, 'Breda', 'Carlo', 'Postdoctoral Research Fellow', 'cb172@le.ac.uk', 'Department of Genetics, University of Leicester, Leicester LE1 7RH, UK', 'Neurobiology', ''),
(80, 'Bapteste', 'Eric', 'Professor', 'epbapteste@gmail.com', ' Pierre and Marie Curie University - Paris 6,  Unité de recherche Systématique, Adaptation, Évolution', '', ''),
(81, 'Méheust', 'Raphaël', 'PhD student', 'raphael.meheust@gmail.com', ' Pierre and Marie Curie University - Paris 6, Unité de recherche Systématique, Adaptation, Évolution', '', ''),
(82, 'Liddelow', 'Shane A.', 'Postdoctoral Research Fellow', 'liddelow@stanford.edu', 'Department of Neurobiology, Stanford University, School of Medicine, Stanford, California 94305, USA', '', ''),
(83, 'Chain', 'Patrick', 'Professor', 'pchain@lanl.gov', 'Los Alamos National Laboratory, U.S. Dept. of Energy''s NNSA, USA', '', ''),
(84, 'Ritchie', 'Hannah Kent ', 'PhD student', 'hannah.ritchie@colorado.edu', 'Integrative Physiology Department, University of Colorado Boulder, Boulder CO, USA', '', ''),
(85, 'Wright ', 'Kenneth P. ', 'Professor', 'kenneth.wright@colorado.edu', 'Department of Integrative Physiology, University of Colorado, Boulder, CO, USA', '', ''),
(86, 'Stothard', 'Ellen R.', 'PhD student', 'ellen.stothard@colorado.edu', 'Department of Integrative Physiology, University of Colorado Boulder, Boulder CO, USA', '', ''),
(87, 'Melin', 'Amanda D. ', 'Professor', 'amanda.melin@ucalgary.ca', 'Department of Anthropology and Archaeology &  Alberta Children’s Hospital Research Institute  University of Calgary, AB, Canada', '', ''),
(88, 'Hammond', 'Andrew ', 'Postdoctoral Research Fellow', 'andrew.hammond08@imperial.ac.uk', 'Department of Life Sciences, Faculty of Natural Sciences, Imperial College London, UK', '', ''),
(89, 'Karlsson', 'Xenia', 'M.Sc. student', '', 'Department of Life Sciences, Faculty of Natural Sciences, Imperial College London, UK', '', ''),
(90, 'Wang', 'Ziyin', 'M.Sc. student', '', 'Department of Life Sciences, Faculty of Natural Sciences, Imperial College London, UK', '', ''),
(91, 'Neave', 'Nick', 'Associate Professor and Lecturer', 'nick.neave@northumbria.ac.uk', 'Department of Psychology, Northumbria University, Newcastle, UK', '', ''),
(92, 'Simon', 'Daniel T.', 'Associate Professor', 'daniel.simon@liu.se', 'Laboratory of Organic Electronics, Department of Science and Technology, Linköping University, 60174 Norrköping, Sweden', '', ''),
(93, 'Poxson', 'David J.', 'Postdoctoral Research Fellow', 'david.poxson@liu.se', 'Laboratory of Organic Electronics, Department of Science and Technology, Linköping University, 60174 Norrköping, Sweden', '', ''),
(94, 'Prosee', 'Reinier', 'PhD student', 'Reinier.Prosee@unige.ch', 'Department of Molecular Biology, Section of Biology, Faculty of Science, University of Geneva, Switzerland', '', ''),
(95, 'Zaidi', 'Arslan', 'PhD student', 'saz5078@psu.edu', 'Department of Anthropology, PennState College of the Liberal Arts, The Pennsylvania State University, USA', '', ''),
(96, 'Shriver', 'Mark', 'Professor', 'mds17@psu.edu', 'Department of Anthropology, PennState College of the Liberal Arts, The Pennsylvania State University, USA', '', ''),
(97, 'DeCasien', 'Alexandra R', 'PhD student', 'alex.decasien@nyu.edu', 'Department of Anthropology, New York University, New York Consortium in Evolutionary Primatology, New York, USA', '', ''),
(98, 'Argue', 'Debbie', 'Postdoctoral Research Fellow', 'debbie.argue@anu.edu.au', 'School of Archaeology and Anthropology, College of Arts and Social Science, Australian National University, Canberra, Australia', '', ''),
(99, 'Lorenzo', 'Laura', 'PhD student', 'Laura.Lorenzo@unige.ch', 'Department of Plant Biology, Section of Biology, Faculty of Science, University of Geneva, Switzerland', '', ''),
(100, 'Farrer', 'Andrew', 'PhD student', 'andrew.farrer@adelaide.edu.au', 'Australian Centre for Ancient DNA (ACAD), School of Biological Sciences, The University of Adelaide, Adelaide, SA, 5005 Australia', '', ''),
(101, 'Geigl', 'Eva-Maria', 'Research Director', 'Eva-maria.GEIGL@ijm.fr', 'National Research Center CNRS, Jacques Monod Institute, University Paris Diderot, 15 rue Helene Brion, 75013 Paris, France', '', ''),
(102, 'Grange', 'Thierry', 'Research Director', 'Thierry.GRANGE@ijm.fr', 'National Research Center CNRS, Jacques Monod Institute, University Paris Diderot, 15 rue Helene Brion, 75013 Paris, France', '', ''),
(103, 'Frank', 'Erik', 'PhD student', 'erik@the-franks.de', 'Department of Animal Ecology and Tropical Biology, Biocentre, University of Würzburg, Am Hubland, D-97074 Würzburg, Germany', '', ''),
(104, 'Isley', 'Nicholas A.', 'Postdoctoral Research Fellow', 'nisley@scripps.edu', 'The Scripps Research Institute, BCC-483, 10550 N. Torrey Pines Rd. La Jolla, CA 92037', '', ''),
(105, 'Phillips', 'Jonathan', 'Postdoctoral Research Fellow', 'phillips01@g.harvard.edu', 'Department of Psychology, Harvard University, Cambridge, Massachusetts, USA', '', ''),
(106, 'Schwing', 'Raoul', 'Professor', 'Raoul.Schwing@vetmeduni.ac.at', 'Messerli Research Institute, Comparative Cognition, University of Veterinary Medicine Vienna (Vetmeduni Vienna), Vienna, Austria', '', ''),
(107, 'Wein', 'Amelia', 'PhD student', '', 'Messerli Research Institute, Comparative Cognition, University of Veterinary Medicine Vienna (Vetmeduni Vienna), Vienna, Austria', '', ''),
(108, 'Woodcock', 'Ben A.', 'Ecological Entomologist', '', 'NERC, Centre for Ecology & Hydrology, Wallingford, Oxfordshire OX10 8BB, UK', '', ''),
(110, 'Heard', 'Matthew S.', 'Plant Ecologist', '', 'NERC Centre for Ecology & Hydrology, Wallingford, Oxfordshire OX10 8BB, UK', '', ''),
(111, 'Mitschunas', 'Nadine', 'Field Ecologist', '', 'NERC Centre for Ecology & Hydrology, Wallingford, Oxfordshire OX10 8BB, UK.', '', ''),
(112, 'Raffa', 'Brigitta', 'Undergraduate Student', '', 'Szent-István University, 2103 Gödöllö, Hungary.', '', ''),
(113, 'Vaskor', 'Dora', 'Undergraduate Student', '', 'Szent-István University, 2103 Gödöllö, Hungary.', '', ''),
(114, 'Bihlay', 'Áron', 'Undergraduate Student', '', 'Szent-István University, 2103 Gödöllö, Hungary.', '', ''),
(115, 'Webb', 'Judy A.', 'Associate Researcher', '', 'NERC Centre for Ecology & Hydrology, Wallingford, Oxfordshire OX10 8BB, UK.', '', ''),
(116, 'Pywell', 'Richard F.', 'Biodiversity Science Area Head', '', 'NERC Centre for Ecology & Hydrology, Wallingford, Oxfordshire OX10 8BB, UK.', '', ''),
(117, 'Bath', 'Eleanor', 'Postdoctoral Research Fellow', 'eleanor.bath@chch.ox.ac.uk', 'Christ Church College, Department of Zoology, University of Oxford, Oxford OX1 2JD, UK.', '', '');

-- --------------------------------------------------------

--
-- Table structure for table `author_break`
--

CREATE TABLE IF NOT EXISTS `author_break` (
  `id` int(11) NOT NULL,
  `break_id` int(11) DEFAULT NULL,
  `author_id` int(11) DEFAULT NULL
) ENGINE=MyISAM AUTO_INCREMENT=532 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `author_break`
--

INSERT INTO `author_break` (`id`, `break_id`, `author_id`) VALUES
(325, 61, 48),
(323, 47, 35),
(112, 43, 31),
(76, 73, 42),
(326, 65, 54),
(363, 63, 50),
(361, 57, 43),
(111, 40, 27),
(366, 64, 61),
(316, 60, 47),
(322, 67, 56),
(166, 39, 26),
(122, 42, 30),
(338, 71, 59),
(335, 51, 39),
(337, 69, 58),
(402, 44, 32),
(344, 62, 49),
(333, 49, 60),
(343, 56, 42),
(391, 45, 33),
(345, 66, 55),
(334, 50, 38),
(340, 52, 40),
(341, 68, 57),
(422, 41, 29),
(348, 58, 44),
(346, 46, 34),
(353, 38, 25),
(355, 70, 36),
(354, 48, 36),
(528, 59, 46),
(420, 37, 24),
(77, 74, 40),
(78, 74, 23),
(79, 75, 42),
(80, 75, 35),
(81, 75, 51),
(421, 41, 28),
(332, 49, 37),
(347, 58, 45),
(365, 64, 53),
(364, 64, 52),
(336, 69, 62),
(362, 63, 51),
(121, 36, 23),
(155, 76, 33),
(327, 77, 63),
(158, 78, 52),
(328, 79, 65),
(329, 80, 54),
(358, 81, 66),
(357, 81, 67),
(356, 81, 68),
(400, 82, 69),
(201, 83, 39),
(202, 84, 39),
(203, 85, 68),
(204, 86, 39),
(205, 87, 39),
(206, 88, 46),
(207, 89, 40),
(208, 90, 35),
(210, 91, 35),
(360, 92, 70),
(359, 92, 36),
(312, 93, 73),
(311, 93, 72),
(263, 94, 48),
(264, 95, 39),
(265, 96, 39),
(266, 97, 35),
(267, 98, 73),
(268, 99, 73),
(269, 100, 73),
(270, 101, 23),
(271, 102, 48),
(349, 103, 74),
(398, 104, 75),
(397, 104, 76),
(396, 104, 71),
(308, 105, 77),
(527, 106, 78),
(395, 107, 79),
(370, 108, 81),
(371, 108, 80),
(379, 109, 81),
(378, 109, 80),
(401, 110, 49),
(386, 111, 54),
(394, 112, 82),
(426, 113, 83),
(425, 114, 86),
(424, 114, 84),
(423, 114, 85),
(427, 115, 73),
(428, 116, 73),
(429, 117, 73),
(432, 118, 87),
(441, 119, 88),
(440, 119, 89),
(439, 119, 90),
(529, 120, 91),
(449, 121, 92),
(448, 121, 93),
(452, 122, 94),
(460, 123, 96),
(459, 123, 95),
(462, 124, 97),
(472, 125, 98),
(469, 126, 99),
(471, 127, 100),
(478, 128, 101),
(477, 128, 102),
(481, 129, 103),
(484, 130, 104),
(526, 131, 105),
(493, 132, 106),
(492, 132, 107),
(525, 133, 110),
(524, 133, 111),
(523, 133, 112),
(522, 133, 113),
(521, 133, 114),
(520, 133, 115),
(519, 133, 116),
(518, 133, 108),
(531, 134, 117);

-- --------------------------------------------------------

--
-- Table structure for table `available_breaks`
--

CREATE TABLE IF NOT EXISTS `available_breaks` (
  `id` int(11) NOT NULL,
  `article` text COLLATE utf8_unicode_ci,
  `subject_id` int(11) DEFAULT NULL
) ENGINE=MyISAM AUTO_INCREMENT=105 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `available_breaks`
--

INSERT INTO `available_breaks` (`id`, `article`, `subject_id`) VALUES
(9, 'Xue Z, Hennelly S, Doyle B et al. <a href="http://www.cell.com/molecular-cell/pdfExtended/S1097-2765%2816%2930424-5" target="_blank">A G-Rich Motif in the lncRNABraveheart Interacts with a Zinc-Finger Transcription Factor to Specify the Cardiovascular Lineage.</a> <i>Molecular Cell</i>. 2016;64(1):37-50. doi:10.1016/j.molcel.2016.08.010.', 1),
(10, 'vanWyk M, Pielecka-Fortuna J, Löwel S, Kleinlogel S. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Restoring+the+ON+Switch+in+Blind+Retinas%3A+Opto-mGluR6%2C+a+Next-Generation%2C+Cell-Tailored+Optogenetic+Tool" target="_blank">Restoring the ON Switch in Blind Retinas: Opto-mGluR6, a Next-Generation, Cell-Tailored Optogenetic Tool.</a> PLOS Biology. 2015;13(5):e1002143. doi:10.1371/journal.pbio.1002143.', 1),
(11, 'Yano J, Yu K, Donaldson G et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Indigenous+Bacteria+from+the+Gut+Microbiota+Regulate+Host+Serotonin+Biosynthesis"  target="_blank">Indigenous Bacteria from the Gut Microbiota Regulate Host Serotonin Biosynthesis.</a> Cell. 2015;163(1):258. doi:10.1016/j.cell.2015.09.017.', 6),
(12, 'Kutejova E, Sasai N, Shah A, Gouti M, Briscoe J. <a href="http://www.cell.com/developmental-cell/abstract/S1534-5807(16)30037-5?_returnURL=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1534580716300375%3Fshowall%3Dtrue&cc=y=" target="_blank">Neural Progenitors Adopt Specific Identities by Directly Repressing All Alternative Progenitor Transcriptional Programs.</a> Developmental Cell. 2016;36(6):639-653. doi:10.1016/j.devcel.2016.02.013.', 2),
(13, 'Hein G, Morishima Y, Leiberg S, Sul S, Fehr E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=The+brain%E2%80%99s+functional+network+architecture+reveals+human+motives" target="_blank">The brains functional network architecture reveals human motives.</a> Science. 2016;351(6277):1074-1078. doi:10.1126/science.aac7992.', 2),
(14, 'Albertin C, Simakov O, Mitros T et al. <a href="http://www.nature.com/nature/journal/v524/n7564/full/nature14668.html" target="_blank">The octopus genome and the evolution of cephalopod neural and morphological novelties</a>. <i>Nature</i>. 2015;524(7564):220-224. doi:10.1038/nature14668.', 4),
(15, 'Eagen K, Hartl T, Kornberg R. <a href="http://www.cell.com/cell/abstract/S0092-8674%2815%2901340-9" target="_blank">Stable Chromosome Condensation Revealed by Chromosome Conformation Capture</a>. <i>Cell</i>. 2015;163(4):934-946. doi:10.1016/j.cell.2015.10.026.', 7),
(16, 'Schieber A, Lee Y, Chang M et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=disease%20tolerance%20mediated%20by%20microbiome%20e.%20coli%20involves%20inflammasomes%20and%20igf-1%20signaling&amp;cmd=correctspelling" target="_blank">Disease tolerance mediated by microbiome E. coli involves inflammasome and IGF-1 signaling</a>. <i>Science</i>. 2015;350(6260):558-563. doi:10.1126/science.aac6468.', 1),
(17, 'Civiletto G, Varanita T, Cerutti R et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16.+Civiletto+G%2C+Varanita+T%2C+Cerutti+R+et+al.+Opa1+Overexpression+Ameliorates+the+Phenotype+of+Two+Mitochondrial+Disease+Mouse+Models.+Cell+Metabolism.+2015%3B21(6)%3A845-854.+doi%3A10.1016%2Fj.cmet.2015.04.016." target="_blank">Opa1 Overexpression Ameliorates the Phenotype of Two Mitochondrial Disease Mouse Models</a>. <i>Cell Metabolism</i>. 2015;21(6):845-854. doi:10.1016/j.cmet.2015.04.016.', 1),
(18, 'Dionisi-Vici C, Diodato D, Torre G et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Liver+transplant+in+ethylmalonic+encephalopathy%3A+a+new+treatment+for+an+otherwise+fatal+disease" target="_blank">Liver transplant in ethylmalonic encephalopathy: a new treatment for an otherwise fatal disease</a>. <i>Brain</i>. 2016;139(4):1045-1051. doi:10.1093/brain/aww013.', 1),
(19, 'Kang H, Lee S, Ko I, Kengla C, Yoo J, Atala A. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=A+3D+bioprinting+system+to+produce+human-scale+tissue+constructs+with+structural+integrity" target="_blank">A 3D bioprinting system to produce human-scale tissue constructs with structural integrity</a>. <i>Nat Biotechnol</i>. 2016;34(3):312-319. doi:10.1038/nbt.3413.', 1),
(20, 'Beyaz S, Mana M, Roper J et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=High-fat+diet+enhances+stemness+and+tumorigenicity+of+intestinal+progenitors" target="_blank">High-fat diet enhances stemness and tumorigenicity of intestinal progenitors</a>. <i>Nature</i>. 2016;531(7592):53-58. doi:10.1038/nature17173.', 1),
(21, 'Parenti A, Halbisen M, Wang K, Latham K, Ralston A. <a href="http://www.cell.com/stem-cell-reports/abstract/S2213-6711%2816%2900054-0" target="_blank">OSKM Induce Extraembryonic Endoderm Stem Cells in Parallel to Induced Pluripotent Stem Cells</a>. <i>Stem Cell Reports</i>. 2016;6(4):447-455. doi:10.1016/j.stemcr.2016.02.003.', 1),
(22, 'Nishimoto S, Fukuda D, Higashikuni Y et al. <a href="http://advances.sciencemag.org/content/2/3/e1501332" target="_blank">Obesity-induced DNA released from adipocytes stimulates chronic adipose tissue inflammation and insulin resistance</a>. <i>Science Advances</i>. 2016;2(3):e1501332-e1501332. doi:10.1126/sciadv.1501332.', 1),
(23, 'Joubert B, Felix J, Yousefi P et al. <a href="http://www.cell.com/ajhg/fulltext/S0002-9297%2816%2900070-7" target="_blank">DNA Methylation in Newborns and Maternal Smoking in Pregnancy: Genome-wide Consortium Meta-analysis</a>. <i>The American Journal of Human Genetics</i>. 2016;98(4):680-696. doi:10.1016/j.ajhg.2016.02.019.', 1),
(24, 'Van Meter M, Simon M, Tombline G et al. <a href="http://www.sciencedirect.com/science/article/pii/S221112471631052X" target="_blank">JNK Phosphorylates SIRT6 to Stimulate DNA Double-Strand Break Repair in Response to Oxidative Stress by Recruiting PARP1 to DNA Breaks</a>. <i>Cell Reports</i>. 2016;16(10):2641-2650. doi:10.1016/j.celrep.2016.08.006.', 1),
(25, 'Sevigny J, Chiao P, Bussière T et al. <a href="http://www.nature.com/nature/journal/v537/n7618/full/nature19323.html" target="_blank">The antibody aducanumab reduces Aβ plaques in Alzheimer’s disease</a>. <i>Nature</i>. 2016;537(7618):50-56. doi:10.1038/nature19323.', 1),
(26, 'Collins A, Fullmer M, Gogarten J, Nyholm S. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Comparative+genomics+of+Roseobacter+clade+bacteria+isolated+from+the+accessory+nidamental+gland+of+Euprymna+scolopes" target="_blank">Comparative genomics of Roseobacter clade bacteria isolated from the accessory nidamental gland of Euprymna scolopes</a>. <i>Front Microbiol</i>. 2015;6. doi:10.3389/fmicb.2015.00123.', 4),
(27, 'Meadow J, Altrichter A, Bateman A et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Humans+differ+in+their+personal+microbial+cloud" target="_blank">Humans differ in their personal microbial cloud</a>. <i>PeerJ</i>. 2015;3:e1258. doi:10.7717/peerj.1258.', 6),
(28, 'Bradley P, Gordon N, Walker T et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26686880" target="_blank">Rapid antibiotic-resistance predictions from genome sequence data for Staphylococcus aureus and Mycobacterium tuberculosis</a>. <i>Nature Communications</i>. 2015;6:10063. doi:10.1038/ncomms10063.', 6),
(29, 'Di Prisco G, Annoscia D, Margiotta M et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=A+mutualistic+symbiosis+between+a+parasitic+mite+and+a+pathogenic+virus+undermines+honey+bee+immunity+and+health" target="_blank">A mutualistic symbiosis between a parasitic mite and a pathogenic virus undermines honey bee immunity and health</a>. <i>Proceedings of the National Academy of Sciences</i>. 2016;113(12):3203-3208. doi:10.1073/pnas.1523515113.', 4),
(30, 'Wexler A, Bao Y, Whitney J et al. <a href="http://www.pnas.org/content/113/13/3639.abstract" target="_blank">Human symbionts inject and neutralize antibacterial toxins to persist in the gut</a>. <i>Proceedings of the National Academy of Sciences</i>. 2016;113(13):3639-3644. doi:10.1073/pnas.1525637113.', 6),
(31, 'Gallone B, Steensels J, Prahl T et al. <a href="http://www.sciencedirect.com/science/article/pii/S0092867416310716" target="_blank">Domestication and Divergence of Saccharomyces cerevisiae Beer Yeasts</a>. <i>Cell</i>. 2016;166(6):1397-1410.e16. doi:10.1016/j.cell.2016.08.020.', 6),
(32, 'Albertin C, Simakov O, Mitros T et al. <a href="http://www.nature.com/nature/journal/v524/n7564/full/nature14668.html" target="_blank">The octopus genome and the evolution of cephalopod neural and morphological novelties</a>. <i>Nature</i>. 2015;524(7564):220-224. doi:10.1038/nature14668.', 4),
(33, 'Serobyan V, Xiao H, Namdeo S et al. <a href="http://www.nature.com/articles/ncomms12337" target="_blank">Chromatin remodelling and antisense-mediated up-regulation of the developmental switch gene eud-1 control predatory feeding plasticity</a>. <i>Nature Communications</i>. 2016;7:12337. doi:10.1038/ncomms12337.', 4),
(34, 'Fennessy J, Bidon T, Reuss F et al. <a href="http://www.cell.com/current-biology/fulltext/S0960-9822%2816%2930787-4" target="_blank">Multi-locus Analyses Reveal Four Giraffe Species Instead of One</a>. <i>Current Biology</i>. 2016;26(18):2543-2549. doi:10.1016/j.cub.2016.07.036.', 4),
(35, 'Gasmi L, Boulain H, Gauthier J et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=1.+Gasmi+L%2C+Boulain+H%2C+Gauthier+J+et+al.+Recurrent+Domestication+by+Lepidoptera+of+Genes+from+Their+Parasites+Mediated+by+Bracoviruses.+PLoS+Genet.+2015%3B11(9)%3Ae1005470.+doi%3A10.1371%2Fjournal.pgen.1005470." target="_blank">Recurrent Domestication by Lepidoptera of Genes from Their Parasites Mediated by Bracoviruses</a>. <i>PLoS Genet</i>. 2015;11(9):e1005470. doi:10.1371/journal.pgen.1005470.', 4),
(36, 'Sanders J, Beichman A, Roman J et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Baleen+whales+host+a+unique+gut+microbiome+with+similarities+to+both+carnivores+and+herbivore" target="_blank">Baleen whales host a unique gut microbiome with similarities to both carnivores and herbivores</a>. <i>Nature Communications</i>. 2015;6:8285. doi:10.1038/ncomms9285.', 6),
(37, 'Dannemann M, Andrés A, Kelso J. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Introgression+of+Neandertal-+and+Denisovan-like+Haplotypes+Contributes+to+Adaptive+Variation+in+Human+Toll-like+Receptors" target="_blank">Introgression of Neandertal- and Denisovan-like Haplotypes Contributes to Adaptive Variation in Human Toll-like Receptors</a>. <i>The American Journal of Human Genetics</i>. 2016;98(1):22-33. doi:10.1016/j.ajhg.2015.11.015.', 4),
(38, 'Poirotte C, Kappeler P, Ngoubangoye B, Bourgeois S, Moussodji M, Charpentier M. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Morbid+attraction+to+leopard+urine+in+Toxoplasma-infected+chimpanzees" target="_blank">Morbid attraction to leopard urine in Toxoplasma-infected chimpanzees</a>. <i>Current Biology</i>. 2016;26(3):R98-R99. doi:10.1016/j.cub.2015.12.020.', 4),
(39, 'Bugnyar T, Reber S, Buckner C. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Ravens+attribute+visual+access+to+unseen+competitors" target="_blank">Ravens attribute visual access to unseen competitors</a>. <i>Nature Communications</i>. 2016;7:10506. doi:10.1038/ncomms10506.', 4),
(41, 'Adhikari K, Fontanil T, Cal S et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=A+genome-wide+association+scan+in+admixed+Latin+Americans+identifies+loci+influencing+facial+and+scalp+hair+features" target="_blank">A genome-wide association scan in admixed Latin Americans identifies loci influencing facial and scalp hair features</a>. <i>Nature Communications</i>. 2016;7:10815. doi:10.1038/ncomms10815.', 4),
(42, 'Daza J, Stanley E, Wagner P, Bauer A, Grimaldi D. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Mid-Cretaceous+amber+fossils+illuminate+the+past+diversity+of+tropical+lizards" target="_blank">Mid-Cretaceous amber fossils illuminate the past diversity of tropical lizards</a>. <i>Science Advances</i>. 2016;2(3):e1501080-e1501080. doi:10.1126/sciadv.1501080.', 4),
(43, 'Zink KLieberman D.<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Impact+of+meat+and+Lower+Palaeolithic+food+processing+techniques+on+chewing+in+humans" target="_blank"> Impact of meat and Lower Palaeolithic food processing techniques on chewing in humans</a>. <i>Nature</i>. 2016;531(7595):500-503. doi:10.1038/nature16990.', 4),
(44, 'Ludlow C, Cromie G, Garmendia-Torres C et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Independent+Origins+of+Yeast+Associated+with+Coffee+and+Cacao+Fermentation" target="_blank">Independent Origins of Yeast Associated with Coffee and Cacao Fermentation</a>. <i>Current Biology</i>. 2016;26(7):965-971. doi:10.1016/j.cub.2016.02.012.', 6),
(45, 'Lahr D, Bosak T, Lara E, Mitchell E. <a href="https://peerj.com/articles/1234/" target="_blank">The Phanerozoic diversification of silica-cycling testate amoebae and its possible links to changes in terrestrial ecosystems.</a> <i>PeerJ</i>. 2015;3:e1234. doi:10.7717/peerj.1234.', 4),
(46, 'Chou M, Amo R, Kinoshita M et al. <a href="http://science.sciencemag.org/content/352/6281/87" target="_blank">Social conflict resolution regulated by two dorsal habenular subregions in zebrafish</a>. <i>Science</i>. 2016;352(6281):87-90. doi:10.1126/science.aac9508.', 4),
(47, 'Llamas B, Fehren-Schmitz L, Valverde G et al.<a href="http://advances.sciencemag.org/content/2/4/e1501385" target="_blank"> Ancient mitochondrial DNA provides high-resolution time scale of the peopling of the Americas</a>. <i>Science Advances</i>. 2016;2(4):e1501385-e1501385. doi:10.1126/sciadv.1501385.', 4),
(49, 'Kutejova E, Sasai N, Shah A, Gouti M, Briscoe J. <a href="http://www.sciencedirect.com/science/article/pii/S1534580716300375" target="_blank">Neural Progenitors Adopt Specific Identities by Directly Repressing All Alternative Progenitor Transcriptional Programs</a>. <i>Developmental Cell</i>. 2016;36(6):639-653. doi:10.1016/j.devcel.2016.02.013.', 2),
(50, 'Hein G, Morishima Y, Leiberg S, Sul S, Fehr E. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=The+brain%E2%80%99s+functional+network+architecture+reveals+human+motives" target="_blank">The brains functional network architecture reveals human motives</a>. <i>Science</i>. 2016;351(6277):1074-1078. doi:10.1126/science.aac7992.', 2),
(51, 'Pattabiraman D, Bierie B, Kober K et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Activation+of+PKA+leads+to+mesenchymal-to-epithelial+transition+and+loss+of+tumor-initiating+ability" target="_blank">Activation of PKA leads to mesenchymal-to-epithelial transition and loss of tumor-initiating ability</a>. <i>Science</i>. 2016;351(6277):aad3680-aad3680. doi:10.1126/science.aad3680.', 1),
(52, 'Telley L, Govindan S, Prados J et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Sequential+transcriptional+waves+direct+the+differentiation+of+newborn+neurons+in+the+mouse+neocortex" target="_blank">Sequential transcriptional waves direct the differentiation of newborn neurons in the mouse neocortex</a>. <i>Science</i>. 2016;351(6280):1443-1446. doi:10.1126/science.aad8361.', 2),
(53, 'Tang H, Hammack C, Ogden S et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Zika+Virus+Infects+Human+Cortical+Neural+Progenitors+and+Attenuates+Their+Growth" target="_blank">Zika Virus Infects Human Cortical Neural Progenitors and Attenuates Their Growth</a>. <i>Cell Stem Cell</i>. 2016;18(5):587-590. doi:10.1016/j.stem.2016.02.016.', 2),
(54, 'Flore G, Di Ruberto G, Parisot J et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Gradient+COUP-TFI+Expression+Is+Required+for+Functional+Organization+of+the+Hippocampal+Septo-Temporal+Longitudinal+Axis" target="_blank">Gradient COUP-TFI Expression Is Required for Functional Organization of the Hippocampal Septo-Temporal Longitudinal Axis</a>. <i>Cereb Cortex</i>. 2016:bhv336. doi:10.1093/cercor/bhv336.', 2),
(55, 'Flore G, Di Ruberto G, Parisot J et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Gradient+COUP-TFI+Expression+Is+Required+for+Functional+Organization+of+the+Hippocampal+Septo-Temporal+Longitudinal+Axis" target="_blank">Gradient COUP-TFI Expression Is Required for Functional Organization of the Hippocampal Septo-Temporal Longitudinal Axis</a>. <i>Cereb Cortex</i>. 2016:bhv336. doi:10.1093/cercor/bhv336.', 2),
(56, 'Marsh S, Abud E, Lakatos A et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=The+adaptive+immune+system+restrains+Alzheimer%E2%80%99s+disease+pathogenesis+by+modulating+microglial+function" target="_blank">The adaptive immune system restrains Alzheimer’s disease pathogenesis by modulating microglial function</a>. <i>Proceedings of the National Academy of Sciences</i>. 2016;113(9):E1316-E1325. doi:10.1073/pnas.1525466113.', 2),
(57, 'Bei F, Lee H, Liu X et al. <a href="http://www.sciencedirect.com/science/article/pii/S009286741501555X" target="_blank">Restoration of Visual Function by Enhancing Conduction in Regenerated Axons.</a> <i>Cell</i>. 2016;164(1-2):219-232. doi:10.1016/j.cell.2015.11.036.', 2),
(58, 'Kropff E, Carmichael J, Moser M, Moser E. <a href="http://www.nature.com/nature/journal/v523/n7561/full/nature14622.html" target="_blank">Speed cells in the medial entorhinal cortex. </a><i>Nature</i>. 2015;523(7561):419-424. doi:10.1038/nature14622.', 2),
(59, 'Hoban A, Stilling R, Ryan F et al. <a href="http://www.nature.com/tp/journal/v6/n4/full/tp201642a.html" target="_blank">Regulation of prefrontal cortex myelination by the microbiota.</a> <i>Translational Psychiatry</i>. 2016;6(4):e774. doi:10.1038/tp.2016.42.', 2),
(61, 'Springmann M, Godfray H, Rayner M, Scarborough P. <a href="http://www.pnas.org/content/113/15/4146.abstract" target="_blank">Analysis and valuation of the health and climate change cobenefits of dietary change</a>. <i>Proceedings of the National Academy of Sciences</i>. 2016;113(15):4146-4151. doi:10.1073/pnas.1523119113.', 3),
(62, 'Falchi F, Cinzano P, Duriscoe D et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=The+new+world+atlas+of+artificial+night+sky+brightness." target="_blank">The new world atlas of artificial night sky brightness.</a> <i>Science Advances</i>. 2016;2(6):e1600377-e1600377. doi:10.1126/sciadv.1600377.', 3),
(63, 'Seddon A, Macias-Fauria M, Long P, Benz D, Willis K. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26886790" target="_blank">Sensitivity of global terrestrial ecosystems to climate variability</a>. <i>Nature</i>. 2016;531(7593):229-232. doi:10.1038/nature16986.', 3),
(64, 'Moffitt S, Hill T, Roopnarine P, Kennett J. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Response+of+seafloor+ecosystems+to+abrupt+global+climate+change" target="_blank">Response of seafloor ecosystems to abrupt global climate change</a>. <i>Proceedings of the National Academy of Sciences</i>. 2015;112(15):4684-4689. doi:10.1073/pnas.1417130112.', 3),
(65, 'Georgeson L, Maslin M, Poessinouw M, Howard S. <a href="http://www.nature.com/nclimate/journal/vaop/ncurrent/full/nclimate2944.html" target="_blank">Adaptation responses to climate change differ between global megacities</a>. <i>Nature Climate Change</i>. 2016;6(6):584-588. doi:10.1038/nclimate2944.', 3),
(66, 'Loconsole G, Saponari M, Boscia D et al. <a href="http://link.springer.com/article/10.1007/s10658-016-0894-x" target="_blank">Intercepted isolates of Xylella fastidiosa in Europe reveal novel genetic diversity</a>. <i>Eur J Plant Pathol</i>. 2016;146(1):85-94. doi:10.1007/s10658-016-0894-x.', 5),
(67, 'Pedmale U, Huang S, Zander M et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Cryptochromes+Interact+Directly+with+PIFs+to+Control+Plant+Growth+in+Limiting+Blue+Light" target="_blank">Cryptochromes Interact Directly with PIFs to Control Plant Growth in Limiting Blue Light</a>. <i>Cell</i>. 2016;164(1-2):233-245. doi:10.1016/j.cell.2015.12.018.', 5),
(71, 'Hiruma K, Gerlach N, Sacristán S et al. <a href="http://www.cell.com/cell/abstract/S0092-8674%2816%2930130-1" target="_blank">Root Endophyte Colletotrichum tofieldiae Confers Plant Fitness Benefits that Are Phosphate Status Dependent</a>. <i>Cell</i>. 2016;165(2):464-474. doi:10.1016/j.cell.2016.02.028.', 5),
(72, 'Yang H, Wang Y, Chen X et al. <a href="http://www.nature.com/articles/ncomms12809" target="_blank">Plasmonic twinned silver nanoparticles with molecular precision</a>. <i>Nature Communications</i>. 2016;7:12809. doi:10.1038/ncomms12809.', 7),
(73, 'Vermeij W, Doll&eacute; M, Reiling E et al. <a href="http://www.nature.com/nature/journal/v537/n7620/full/nature19329.html" target="_blank">Restricted diet delays accelerated ageing and genomic stress in DNA-repair-deficient mice</a>. <em>Nature</em>. 2016;537(7620):427-431. doi:10.1038/nature19329.', 1),
(74, 'Malaspinas A, Westaway M, Muller C et al. <a href="http://www.nature.com/nature/journal/v538/n7624/full/nature18299.html" target="_blank">A genomic history of Aboriginal Australia</a>. <em>Nature</em>. 2016;538(7624):207-214. doi:10.1038/nature18299.', 4),
(76, 'Frangeul L, Pouchelon G, Telley L, Lefort S, Luscher C, Jabaudon D. <a href="http://www.nature.com/nature/journal/v538/n7623/full/nature19770.html" target="_blank">A cross-modal genetic framework for the development and plasticity of sensory pathways</a>. <em>Nature</em>. 2016;538(7623):96-98. doi:10.1038/nature19770.', 4),
(77, 'Low-D&eacute;carie E, Kolber M, Homme P et al. <a href="http://www.pnas.org/content/112/46/14307.full" target="_blank">Community rescue in experimental metacommunities</a>. <em>Proceedings of the National Academy of Sciences</em>. 2015;112(46):14307-14312. doi:10.1073/pnas.1513125112.', 3),
(78, 'Yates L, Gerstung M, Knappskog S et al. <a href="http://www.nature.com/nm/journal/v21/n7/full/nm.3886.html" target="_blank">Subclonal diversification of primary breast cancer revealed by multiregion sequencing</a>. <em>Nature Medicine</em>. 2015;21(7):751-759. doi:10.1038/nm.3886.', 1),
(79, 'Kotsantis P, Silva L, Irmscher S et al. <a href="http://www.nature.com/articles/ncomms13087" target="_blank">Increased global transcription activity as a mechanism of replication stress in cancer</a>. <em>Nature Communications</em>. 2016;7:13087. doi:10.1038/ncomms13087.', 1),
(80, 'Wang G, Romero-Gallo J, Benoit S et al. <a href="http://mbio.asm.org/content/7/4/e01022-16" target="_blank">Hydrogen Metabolism inHelicobacter pyloriPlays a Role in Gastric Carcinogenesis through Facilitating CagA Translocation</a>. <em>mBio</em>. 2016;7(4):e01022-16. doi:10.1128/mbio.01022-16.', 6),
(81, 'Hore T, von Meyenn F, Ravichandran M et al. <a href="http://www.pnas.org/content/113/43/12202.abstract" target="_blank">Retinol and ascorbate drive erasure of epigenetic memory and enhance reprogramming to na&iuml;ve pluripotency by complementary mechanisms</a>. <em>Proceedings of the National Academy of Sciences</em>. 2016;113(43):12202-12207. doi:10.1073/pnas.1608679113', 1),
(82, 'Agarwal G, Besuchet N, Audergon B, Paik J. <a href="http://www.nature.com/articles/srep34224" target="_blank">Stretchable Materials for Robust Soft Actuators towards Assistive Wearable Devices</a>. <em>Scientific Reports</em>. 2016;6:34224. doi:10.1038/srep34224.', 7),
(83, 'Agarwal G, Besuchet N, Audergon B, Paik J. <a href="http://www.nature.com/articles/srep34224" target="_blank">Stretchable Materials for Robust Soft Actuators towards Assistive Wearable Devices</a>. <em>Scientific Reports</em>. 2016;6:34224. doi:10.1038/srep34224.', 7),
(84, 'Woodruff G, Reyna S, Dunlap M et al. <a href="http://www.sciencedirect.com/science/article/pii/S2211124716312566" target="_blank">Defective Transcytosis of APP and Lipoproteins in Human iPSC-Derived Neurons with Familial Alzheimer&rsquo;s Disease Mutations</a>. <em>Cell Reports</em>. 2016;17(3):759-773. doi:10.1016/j.celrep.2016.09.034.', 2),
(85, 'Preston CEhrsson H. <a href="https://academic.oup.com/cercor/article/26/12/4450/2741261/Illusory-Obesity-Triggers-Body-Dissatisfaction" target="_blank">Illusory Obesity Triggers Body Dissatisfaction Responses in the Insula and Anterior Cingulate Cortex</a>. <em>Cerebral Cortex</em>. 2016;26(12):4450-4460. doi:10.1093/cercor/bhw313.', 2),
(86, 'Katsouri L, Lim Y, Blondrath K et al. <a href="http://www.pnas.org/content/113/43/12292.full" target="_blank">PPAR&gamma;-coactivator-1&alpha; gene transfer reduces neuronal loss and amyloid-&beta; generation by reducing &beta;-secretase in an Alzheimer&rsquo;s disease model</a>. <em>Proceedings of the National Academy of Sciences</em>. 2016;113(43):12292-12297. doi:10.1073/pnas.1606171113.', 2),
(87, 'Liao H, Chen Y, Vilgalys R. <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006348" target="_blank">Metatranscriptomic Study of Common and Host-Specific Patterns of Gene Expression between Pines and Their Symbiotic Ectomycorrhizal Fungi in the Genus Suillus</a>. <em>PLOS Genetics</em>. 2016;12(10):e1006348. doi:10.1371/journal.pgen.1006348.', 4),
(88, 'Sato T, Tremaine M, Parreiras L et al. <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006372" target="_blank">Directed Evolution Reveals Unexpected Epistatic Interactions That Alter Metabolic Regulation and Enable Anaerobic Xylose Use by Saccharomyces cerevisiae</a>. <em>PLOS Genetics</em>. 2016;12(10):e1006372. doi:10.1371/journal.pgen.1006372.', 4),
(89, 'Svoronos N, Perales-Puchalt A, Allegrezza M et al. <a href="http://cancerdiscovery.aacrjournals.org/content/early/2016/09/30/2159-8290.CD-16-0502.article-info" target="_blank">Tumor Cell&ndash;Independent Estrogen Signaling Drives Disease Progression through Mobilization of Myeloid-Derived Suppressor Cells</a>. <em>Cancer Discovery</em>. 2016;7(1):72-85. doi:10.1158/2159-8290.cd-16-0502', 1),
(91, 'Disatnik M, Joshi A, Saw N et al. <a href="http://jem.rupress.org/content/early/2016/11/01/jem.20160776/tab-article-info" target="_blank">Potential biomarkers to follow the progression and treatment response of Huntington&rsquo;s disease</a>. <em>The Journal of Experimental Medicine</em>. 2016;213(12):2655-2669. doi:10.1084/jem.20160776.', 1),
(92, 'Bethune M, Gee M, Bunse M et al. <a href="https://elifesciences.org/content/5/e19095" target="_blank">Domain-swapped T cell receptors improve the safety of TCR gene therapy</a>. <em>eLife</em>. 2016;5. doi:10.7554/elife.19095.', 1),
(93, 'Gu Z, Churchman M, Roberts K et al. <a href="http://www.nature.com/articles/ncomms13331" target="_blank">Genomic analyses identify recurrent MEF2D fusions in acute lymphoblastic leukaemia</a>. <em>Nature Communications</em>. 2016;7:13331. doi:10.1038/ncomms13331.', 1),
(94, 'Wong, M., Giraldo, J., Kwak, S., Koman, V., Sinclair, R., Lew, T., Bisker, G., Liu, P. and Strano, M. (2016). <a href="http://www.nature.com/nmat/journal/v16/n2/full/nmat4771.html" target="_blank">Nitroaromatic detection and infrared communication from wild-type plants using plant nanobionics</a>. <em>Nature Materials</em>, 16(2), pp.264-272.', 5),
(95, 'Tieman D, Zhu G, Resende M et al. <a href="science.sciencemag.org/content/355/6323/391/tab-pdf" target="_blank">A chemical genetic roadmap to improved tomato flavor</a>. <em>Science</em>. 2017;355(6323):391-394. doi:10.1126/science.aal1556', 5),
(96, 'Gagliano M, Vyazovskiy V, Borb&eacute;ly A, Grimonprez M, Depczynski M. <a href="https://www.nature.com/articles/srep38427" target="_blank">Learning by Association in Plants</a>. <em>Scientific Reports</em>. 2016;6(1). doi:10.1038/srep38427.', 5),
(98, 'Rogers R, Slatkin M. <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006601" target="_blank">Excess of genomic defects in a woolly mammoth on Wrangel island</a>. <em>PLOS Genetics</em>. 2017;13(3):e1006601. doi:10.1371/journal.pgen.1006601.', 4),
(99, 'Carroll M, Brown N, Goodall C, Downs A, Sheenan T, Anderson K. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0175933" target="_blank">Honey bees preferentially consume freshly-stored pollen</a>. <em>PLOS ONE</em>. 2017;12(4):e0175933. doi:10.1371/journal.pone.0175933.', 4),
(100, 'Parker H, Dreger D, Rimbault M et al. <a href="http://www.cell.com/cell-reports/abstract/S2211-1247(17)30456-4" target="_blank">Genomic Analyses Reveal the Influence of Geographic Origin, Migration, and Hybridization on Modern Dog Breed Development</a>. <em>Cell Reports</em>. 2017;19(4):697-708. doi:10.1016/j.celrep.2017.03.079.', 4),
(101, 'Hahamy A, Macdonald S, van den Heiligenberg F et al. <a href="http://www.cell.com/current-biology/abstract/S0960-9822(17)30352-4" target="_blank">Representation of Multiple Body Parts in the Missing-Hand Territory of Congenital One-Handers</a>. <em>Current Biology</em>. 2017;27(9):1350-1355. doi:10.1016/j.cub.2017.03.053.', 1),
(104, 'Meyer M, Aldenderfer M, Wang Z et al. <a href="http://science.sciencemag.org/content/355/6320/64.full" target="_blank">Permanent human occupation of the central Tibetan Plateau in the early Holocene</a>. <i>Science</i>. 2017.', 4);

-- --------------------------------------------------------

--
-- Table structure for table `breaks`
--

CREATE TABLE IF NOT EXISTS `breaks` (
  `id` int(11) NOT NULL,
  `title` varchar(200) COLLATE utf8_unicode_ci NOT NULL,
  `content` text COLLATE utf8_unicode_ci,
  `reading_time` float DEFAULT NULL,
  `original_article` text COLLATE utf8_unicode_ci,
  `subject_id` int(11) DEFAULT NULL,
  `date_created` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `author1_id` int(11) DEFAULT NULL,
  `author2_id` int(11) DEFAULT NULL,
  `editor_id` int(11) DEFAULT NULL,
  `imzy_link` varchar(250) COLLATE utf8_unicode_ci DEFAULT NULL
) ENGINE=MyISAM AUTO_INCREMENT=135 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `breaks`
--

INSERT INTO `breaks` (`id`, `title`, `content`, `reading_time`, `original_article`, `subject_id`, `date_created`, `author1_id`, `author2_id`, `editor_id`, `imzy_link`) VALUES
(37, 'Why some forests burn better than others', '<p>Forest fires are dramatic ecological events as they can wipe out most of the plants and animals within several kilometers in only a few hours and often also pose a major threat to human settlements. They have a positive ecological impact when occurring naturally because they clear the organic matter residues and favor new plant growth. However, exceptionally frequent or devastating fires - caused by climate change or, more frequently, human negligence - can lead to the extinction of some species in a certain area and completely reshape the landscape. They can also directly impact on human communities located nearby and/or whose economy depends on forest goods, especially wood and paper. Thus, a better understanding of how fires spread and what triggers them is important to protect the environment as well as economical interests.</p>\r\n\r\n<p>The extent and spread of forest fires highly depend on weather conditions, including wind and dryness, and on the characteristics of plants themselves – and their remains. <a href="http://en.wikipedia.org/wiki/Plant_litter" target="_blank">Plant litter</a> - the organic matter that accumulates on the soil - is in fact the most fire-prone forest layer, but not all plant litters are high-quality fuels. Their flammability depends on several traits such as silica content and concentration of highly combustible organic compounds, but also their spatial arrangement.</p>\r\n\r\n<p>This is why a Dutch research group developed an experiment to assess the natural burning behavior of the litter of 39 forest tree species belonging to the group of <a href="http://www.theplantlist.org/browse/G/" target="_blank">gymnosperms</a> – non-flowering trees such as <a href="http://en.wikipedia.org/wiki/Pine" target="_blank">pine</a>, <a href="http://en.wikipedia.org/wiki/Cedrus" target="_blank">cedar</a>, and <a href="http://en.wikipedia.org/wiki/Cycad" target="_blank">cycads</a>. Many of these trees have leaves in the shape of needles and form big forests that are known to be most prone to catch fire (see the 14th Forest Fires in Europe <a href="http://bookshop.europa.eu/is-bin/INTERSHOP.enfinity/WFS/EU-Bookshop-Site/en_GB/-/EUR/ViewPublication-Start?PublicationKey=LBNA26791" target="_blank">report </a>of the European Commission).</p>\r\n\r\n<p>The scientists mimicked the spatial arrangement of litters in natural forests and observed what happened when they burned. To this end, plant litter was collected directly from botanical gardens and forests, paying particular attention to gathering the matter “as it was”, with little or no manipulation (<em>e.g.</em> leaves with or without twigs and bark residues). Samples were collected from the ground, sometimes after gentle shaking of the canopy to increase the amount. Then, litter particles were set on fire in the lab after being gently dropped inside a small container and allowed to take a spontaneous configuration.</p>\r\n\r\n<p>The most striking observation was that the litter of some trees of the <a href="http://en.wikipedia.org/wiki/Pinaceae" target="_blank"><em>Pinaceae</em></a> family, including the samples of larch, fir and spruce, almost did not catch fire at all, while all other samples, including pines (<a href="http://en.wikipedia.org/wiki/Pine" target="_blank"><em>Pinus</em></a>) burned completely or nearly completely. This observation was also in disagreement with previous data showing that leaves of those plants successfully burned when taken individually.</p>\r\n\r\n<p>What is behind this unexpected behavior? The authors observed that those plants drop single needles instead of brunches and twigs, like all other tested trees, leading to high litter density or litterbed packing ratio (<em>i.e.</em> the volume of litter material/volume of litterbed). This might restrict air flow, thus resulting in insufficient oxygen availability to sustain the fire.</p>\r\n\r\n<p>To test this hypothesis, the authors selected two other gymnosperms, whose litter had been shown to be highly flammable and repeated the fire experiments after cutting their litter into smaller pieces, increasing its density. Again, they observed that the cut samples reached lower temperatures and did not burn completely – or even failed to burn. Even the increase in litter surface area to mass due to the artificial cuts did not diminish this effect. The authors conclude that the particle size of the litter is a major determinant of the rate of heat release and, thus, flammability, a so-called “first-order effector”.</p>\r\n\r\n<p>Besides the percentage of burned sample, the scientists measured other parameters like maximum temperature, total burning time, and fire front speed. They conclude that particle size, albeit having a major effect, is not the only factor controlling flammability in a mimicked natural environment. Indeed, at a given particle size, the burning behavior was also dependent on additional species-based factors that influence, in particular, fire temperature and sustainability.</p>\r\n\r\n<p>In conclusion, this study thoroughly characterized a key parameter of plant litter flammability and shed new light on other important litter traits. These data can help in building better models to predict the probability of forest fires, based on climate conditions and on the type of plant species present. This may help to contain or even prevent disastrous forest fires in the future, securing important economical and recreational resources – and the survival of countless plants, animals and, not least, humans. The study also illustrates how reproducing natural conditions in a laboratory can help in coming up with realistic models of complex natural phenomena like fire.</p>', 4, 'Cornwell W, Elvira A, van Kempen L, van Logtestijn R, Aptroot A, Cornelissen J. <strong><a href="http://onlinelibrary.wiley.com/doi/10.1111/nph.13317/abstract" target="_blank">Flammability across the gymnosperm phylogeny: the importance of litter particle size</a></strong>. <i>New Phytologist</i>. 2015;206(2):672-681. doi:10.1111/nph.13317.', 3, '2015-05-19 15:04:07', 24, NULL, 11, 'https://doi.org/10.25250/thescbr.brk002'),
(36, 'How humans gave acne to the grapevine', '<p>Many organisms, ourselves included, host diverse communities of microorganisms that live on and within us. Plenty of these are bacteria and with time, some adapt to live and depend so intimately with their hosts that eventually, life without them is almost impossible. However, in some rare cases, bacteria can switch and adapt to a different host, much like other organisms may adapt to new ecosystems. So far, this has been observed mainly between hosts that are closely related. For example, bacteria depending on one insect species can switch to another insect species and adapt to the new host.</p>\r\n\r\n<p>One bacterium that lives closely with us humans is <a href="http://en.wikipedia.org/wiki/Propionibacterium_acnes" target="_blank"><em>Propionibacterium acnes</em></a><em>  (P. acnes)</em>, usually present in our skin, sometimes causing acne. In this study, the authors report the discovery of a strain of <em>P. acnes</em> far from its customary human host and instead well adapted to living in a plant, the grapevine (<em>Vitis vinifera L.</em>). The bacterium was initially detected when its genetic material (its DNA) was found in a DNA sample of grapevine. Further tests were done to prove that the detection of <em>P. acnes</em> was not due to contamination from human handling of the samples, which is the most common source of such findings. Using a technique called “fluorescence <em>in situ</em> hybridization” (<a href="http://en.wikipedia.org/wiki/Fluorescence_in_situ_hybridization" target="_blank">FISH</a>), the authors were able to directly visualize the <em>P. acnes</em> bacteria inside grapevine tissues. Using this technique, they tagged <em>P. acnes’</em> DNA with a fluorescent dye, making the bacteria shine brightly when irradiated with a laser and easy to spot with a microscope. This is the first ever reported bacterial host transfer between humans and a plant.</p>\r\n\r\n<p>Once the natural presence of the <em>P. acnes</em> bacteria in grapevine was confirmed, it was time to ask questions: Are the <em>P. acnes</em> strains in humans and in grapevine different? If so, how are they related? Has the grapevine strain specialized to its plant host? When and how did they begin to have different hosts?</p>\r\n\r\n<p>A <a href="http://en.wikipedia.org/wiki/Phylogenetics" target="_blank">comparison of the differences and similarities in the DNA sequences</a> of a <a href="http://en.wikipedia.org/wiki/Ribosomal_DNA" target="_blank">ribosomal gene</a> from <em>P. acnes</em> in humans and grapevine showed that they are, in fact, different strains. Based on these data, the new strain was named <em>P. acnes</em> type Zappae (<em>P. Z</em><em>appae</em> for short), in honor of the eccentric composer <a href="http://en.wikipedia.org/wiki/Frank_Zappa" target="_blank">Frank Zappa</a>. However, when the sequence comparisons were repeated with two additional genes<em>, P. Z</em><em>appae</em> did not appear to be different to <em>P. acne, </em>in contrast to what was found earlier. The authors argue that the different results are caused by an evolutionary phenomenon called <a href="http://biologos.org/blog/evolution-basics-species-trees-gene-trees-and-incomplete-lineage-sorting" target="_blank">incomplete lineage sorting</a>. This phenomenon arises when different strains have not (yet) diverged completely in their DNA sequence: some genes are still virtually identical in both species while others differ substantially. This can be typically observed in events of recent diversification.</p>\r\n\r\n<p>One of the genes studied, <em>recA</em>, showed a curious pattern of evolution in <em>P. Z</em><em>appae</em>: the gene sequence had changed very rapidly, which often happens when genes are not used anymore. Previous studies have shown that the <em>recA</em> gene is often unused in bacteria that have evolved into endocellular symbionts. In this state, not only do the bacteria and the host both nourish and feed from each other, but the bacteria actually live inside the hosts'' cells! In humans,<em> P. acnes</em> is not an endocellular symbiont. This could therefore mean that <em>P. Z</em><em>appae</em> is specializing in its plant host.</p>\r\n\r\n<p>Finally, using the rate of DNA sequence change in the <em>recA</em> gene to estimate its speed of evolution, the authors could calculate how old the <em>P.</em> <em>Zappae</em> strain is. Using a few different methods of analysis to be sure of the quality of their results, the authors found that <em>P. Z</em><em>appae</em> appeared around 7,500-6,300 years ago, a date that matches with the time in which humans began to cultivate grapevine.</p>\r\n\r\n<p>Based on these results, they conclude that around 7,000 years ago, humans transferred <em>P. acnes</em> to grapevine when cultivating it. This strain is now <em>P.</em> <em>Zappae</em>, a version of <em>P. acnes</em> that has managed to adapt to a very different host.</p>\r\n', 3.5, 'Campisano A, Ometto L, Compant S et al. <strong><a href="http://mbe.oxfordjournals.org/content/31/5/1059.short" target="_blank">Interkingdom Transfer of the Acne-Causing Agent, Propionibacterium acnes, from Human to Grapevine</a></strong>. <i>Molecular Biology and Evolution</i>. 2014;31(5):1059-1065. doi:10.1093/molbev/msu075.', 4, '2015-02-28 16:01:25', 23, NULL, 11, 'https://doi.org/10.25250/thescbr.brk001'),
(38, 'GMOs are not a human invention: sweet potato is a naturally transgenic food crop', '<p><a href="https://en.wikipedia.org/wiki/Sweet_potato" target="_blank">Sweet potato</a> is one of the most important food crops for human consumption in the world. It is especially grown and consumed in Sub-Saharan Africa, parts of Asia, and the Pacific islands. Additionally, it is one of the earliest domesticated crops, documented by archeological findings in caves of the Chilca Canyon in Peru that are 8,000 to 10,000 years old.</p>\r\n\r\n<p>While searching the sweet potato <a href="https://en.wikipedia.org/wiki/Genome" target="_blank">genome</a> – this is the entire DNA-code – for viral diseases, researchers in Peru unexpectedly discovered DNA sequences of the bacterium ‘<a href="https://en.wikipedia.org/wiki/Agrobacterium" target="_blank"><em>Agrobacterium’</em></a>. Instead of contributing this peculiar finding to bacterial contamination of the plant samples (as is usually done in similar studies), they decided to study these sequences in more detail. They teamed up with different laboratories spread all over the world, including ours at Ghent University in Belgium, to provide independent evidence for their observations. Different complementary DNA analyses were executed, undoubtedly demonstrating that two large pieces of bacterial DNA were stably integrated in the sweet potato genome. Moreover, it was shown that the added bacterial genes were active inside the sweet potato plant tissue. The fact that these genes are still active in sweet potato, after probably thousands of years, suggests that they provided a positive characteristic, which was selected for by farmers during the domestication of this plant. </p>\r\n\r\n<p>The presence of the two bacterial DNA pieces was further investigated across a large set of different sweet potato varieties grown in the world. The DNA analyses confirmed that one of the two foreign DNA sequences was present in each of the more than 200 tested sweet potato varieties. The second piece of bacterial DNA was present in about one fourth of the tested samples, and even in some related wild species.</p>\r\n\r\n<p>This is not the first time that researchers find bacterial, fungal, or viral DNA in the genome of plants or animals. Genome analyses in recent years find more and more examples of possible “<a href="https://en.wikipedia.org/wiki/Horizontal_gene_transfer" target="_blank">horizontal gene transfers</a>”. In an event of horizontal gene transfer, genes are exchanged between different species – in contrast to normal gene transfer from parents to progeny, which occurs within one species. Already in the late 1970s, it was discovered that <em>Agrobacterium</em> is able to transfer a part of its own DNA, the so-called <a href="https://en.wikipedia.org/wiki/Transfer_DNA" target="_blank">transfer DNA or T-DNA</a>, to a few specific plant cells. <em>Agrobacterium</em> uses this process during its pathogenicity on plants, to induce the formation of <a href="http://global.britannica.com/science/crown-gall" target="_blank">crown galls</a> or <a href="https://en.wikipedia.org/wiki/Hairy_root_culture" target="_blank">hairy root disease</a>. It is this T-DNA that we have found inside the genome of sweet potato, although all analyzed plants were completely healthy.</p>\r\n\r\n<p>The mechanism used by <em>Agrobacterium</em> to incorporate its own T-DNA in the genetic material of plants forms the basis of the plant <a href="https://en.wikipedia.org/wiki/Genetically_modified_crops" target="_blank">GMO (genetically modified organism)</a> technology. While <em>Agrobacterium</em> naturally introduces its T-DNA only in a few plant cells, biotechnologists have succeeded to <a href="https://en.wikipedia.org/wiki/Agrobacterium#Uses_in_biotechnology" target="_blank">regenerate complete plants</a> from these modified cells. GMO plants are currently grown on more than 10% of the world’s agricultural lands, but are still debated very controversially. The finding of T-DNAs in sweet potato now reveals that this plant has naturally acquired <em>Agrobacterium</em> genes in the course of its evolution, most likely already thousands of years ago. The natural presence of <em>Agrobacterium</em> T-DNA in sweet potato and its stable inheritance during evolution is a beautiful example of the possibility of DNA exchange across species barriers. It demonstrates that genetic modification also happens in nature.</p>\r\n\r\n<p>Given the fact that sweet potato has been cultivated and eaten for thousands of years, this discovery might change the paradigm about the ‘unnatural’ status of GMO food. </p>\r\n', 3, 'Kyndt T, Quispe D, Zhai H, Jarret R, Ghislain M, Liu Q, Gheysen G, Kreuze J. <a href="http://www.pnas.org/content/112/18/5844" target="_blank">The genome of cultivated sweet potato containsAgrobacteriumT-DNAs with expressed genes: An example of a naturally transgenic food crop.</a> <em>Proceedings of the National Academy of Sciences</em>. 2015;112(18):5844-5849. doi:10.1073/pnas.1419685112.', 5, '2015-07-06 15:06:15', 25, NULL, 11, 'https://doi.org/10.25250/thescbr.brk003'),
(39, 'Tara Oceans Expedition sequences the ocean', '<p> The <em><a href="http://oceans.taraexpeditions.org/en/m/about-tara/" target="_blank">Tara Oceans</a></em> consortium recently published five scientific papers in the journal Science presenting the initial wave of scientific results from the first six years of the project.<sup>1-5</sup> The findings show the extraordinary diversity of <a href="https://en.wikipedia.org/wiki/Plankton" target="_blank">plankton</a> in the world’s oceans, uncover many of the interactions between them, and reveal how plankton impact and are influenced by the environment. One of the papers<sup>1</sup> describes an ocean microbial reference gene catalog containing 40 million <a href="https://en.wikipedia.org/wiki/Gene" target="_blank">genes</a> from marine microbes (bacteria, viruses, <a href="https://en.wikipedia.org/wiki/Archaea" target="_blank">Archaea</a> and <a href="https://en.wikipedia.org/wiki/Picoeukaryote" target="_blank">picoeukaryotes</a>). Derived from samples of seawater collected from all over the world and at depths down to 1,000 metres the authors show that this publicly available <a href="https://en.wikipedia.org/wiki/Nucleic_acid_sequence" target="_blank">DNA sequence</a> dataset is more than 1,000 times larger than what was previously available and is likely to have captured all the abundant microbial genes present in the areas sampled. </p>\r\n\r\n<p>The <em>Tara</em> Oceans project has grasped the attention and imagination of both scientists and the general public, with TV, film and radio presentations, as well as thousands of articles being written in the international press (<a href="http://oceans.taraexpeditions.org/en/" target="_blank">http://oceans.taraexpeditions.org/en/</a>). Starting as a grass-roots initiative by a group of academic scientists, the research-enabled 36 metre schooner <em>Tara</em> spent almost 4 years circumnavigating the globe and going around the <a href="https://en.wikipedia.org/wiki/Arctic_Circle" target="_blank">Arctic Circle</a>. Overall, <em>Tara</em> Oceans sampled plankton at more than 210 sites and at multiple depth layers in all the major oceanic regions. The 35,000 samples collected from the expedition now form the basis for extensive processing, analysis, and data integration on land. One of the wettest experiments ever has generated more than 7 terabytes of information, one of the largest contiguous sets of DNA sequence available to the scientific community.<sup>6-8</sup> <p>\r\n\r\n<p>The enormous 40 million gene set described by Sunagawa <em>et al</em>.<sup>1</sup> is shown to derive principally from around 35,000 species of marine bacteria. Around 80% of these sequences are novel, and most of the novelty is shown to derive from oceanic regions traditionally undersampled (such as the Southern Ocean), and from the twilight zone below the upper +/- 100 metres of the water column where sunlight can pass. The microbial communities are quite different from one location to another, and the authors found that temperature is the main factor determining their composition. In fact just by looking at the species content they could predict the temperature of the water with a high precision – quite a sophisticated thermometer! The implication of this finding is that temperature changes in a future ocean impacted by climate change are likely to affect the functioning of the whole <a href="https://en.wikipedia.org/wiki/Marine_ecosystem" target="_blank">marine ecosystem</a>, impacting the food chain and the <a href="https://en.wikipedia.org/wiki/Biogeochemistry" target="_blank">biogeochemical</a> cycles that depend on this microbial world. </p>\r\n\r\n<p>An interesting aspect of the work was to compare the ocean <a href="https://en.wikipedia.org/wiki/Microbiota#Projects" target="_blank">microbiome</a> (the community of all microorganisms) with the human gut microbiome, on the premise that both represent single contiguous ecosystems. The 40 million genes in the global ocean compare to 10 million genes in the human gut microbiome, so the numbers are quite similar. Furthermore, in terms of abundance close to ¾ of ocean and gut genes are the same, in spite of them being very different ecosystems and containing different microbes. On the other hand, around 90% of these gut microbiome genes have putative functions assigned to them, which compares to less than 60% of the ocean microbiome. These striking differences reflect the priorities of funding <a href="https://en.wikipedia.org/wiki/Medical_research" target="_blank">biomedical research</a> at the expense of environmental research, even though a healthy ecosystem promotes human health too. Another striking difference between the ocean and gut microbiomes is that whereas many gut microbiome genes are involved in signaling and cell-cell communication, the primary signal from the ocean microbiome is one of nutrient transport, energy utilization and basic <a href="https://en.wikipedia.org/wiki/Metabolism" target="_blank">metabolism</a>.</p>\r\n\r\n<p>Life evolved in the ocean, and the complex consortium of organisms that make us human is not unlike the organisms that co-evolved to adapt to life in marine plankton ecosystems. The striking similarities between the two ecosystems are food for thought to ponder about our origins and to remember the importance of marine plankton ecosystems for generating and maintaining planet Earth habitable for us. In spite of our <a href="https://en.wikipedia.org/wiki/Anthropocentrism" target="_blank">anthropocentric</a> focus on biomedical research related to human healthcare, we should not forget the microbial life support systems that made our planet habitable for us in the first place and that continue to do so. In today’s world of changing climate, the strong sensitivity of marine microbes to temperature should be a wake-up call to stop exploiting marine resources and treating the ocean as a garbage site for everything we want to get rid of.</p>', 4, 'Sunagawa, S., Coelho, L., Chaffron, S., Kultima, J., Labadie, K., Salazar, G., Djahanschiri, B., Zeller, G., Mende, D., Alberti, A., Cornejo-Castillo, F., Costea, P., Cruaud, C., d''Ovidio, F., Engelen, S., Ferrera, I., Gasol, J., Guidi, L., Hildebrand, F., Kokoszka, F., Lepoivre, C., Lima-Mendez, G., Poulain, J., Poulos, B., Royo-Llonch, M., Sarmento, H., Vieira-Silva, S., Dimier, C., Picheral, M., Searson, S., Kandels-Lewis, S., Bowler, C., de Vargas, C., Gorsky, G., Grimsley, N., Hingamp, P., Iudicone, D., Jaillon, O., Not, F., Ogata, H., Pesant, S., Speich, S., Stemmann, L., Sullivan, M., Weissenbach, J., Wincker, P., Karsenti, E., Raes, J., Acinas, S., Bork, P., Boss, E., Bowler, C., Follows, M., Karp-Boss, L., Krzic, U., Reynaud, E., Sardet, C., Sieracki, M. and Velayoudon, D. (2015). <a href="http://science.sciencemag.org/content/348/6237/1261359.full" target="_blank">Structure and function of the global ocean microbiome</a>. <em>Science</em>, 348(6237), pp.1261359-1261359.', 3, '2015-11-24 16:13:27', 26, NULL, 11, 'https://doi.org/10.25250/thescbr.brk004'),
(40, 'An exoplanet surrounded by an atmosphere larger than its star', '<p>In the past 20 years, scientists have discovered many planets around other stars than the <a href="https://en.wikipedia.org/wiki/Sun" target="_blank">Sun</a>. Nearly half of these so-called <a href="https://en.wikipedia.org/wiki/Exoplanet" target="_blank">exoplanets</a> orbit extremely close to their star (more than ten times closer than <a href="https://en.wikipedia.org/wiki/Mercury_%28planet%29" target="_blank">Mercury</a> revolves around the Sun in our Solar System), raising many questions about the state of their atmosphere: theoretical studies show that the huge amount of energy that the planets receive from their stars can swell the atmosphere, up to the point where some of its gas is far enough from the planet to escape its gravity and drift away. This phenomenon is called ‘evaporation’. </p>\r\n\r\n<p>Because it is light and very abundant, neutral <a href="https://en.wikipedia.org/wiki/Hydrogen" target="_blank">hydrogen</a> escapes more easily from the atmosphere of an exoplanet than any other element, and can tell us a lot about the properties of high-altitude regions in the atmosphere. Until very recently, evaporation was only known for “<a href="https://en.wikipedia.org/wiki/Hot_Jupiter" target="_blank">hot Jupiters</a>”, massive giant planets with a mass similar to that of <a href="https://en.wikipedia.org/wiki/Jupiter" target="_blank">Jupiter</a> but orbiting very close to their stars. These planets are so massive that although gas escapes from their atmosphere with rates in the order of several thousand tons per second, they only lose a few percent of their mass over their whole lifetime. The evaporating atmosphere appears in form of a hydrogen cloud.</p>\r\n\r\n<p>In 2014, J. Kulow and her collaborators published an intriguing signal hinting at the existence of a hydrogen cloud around the “warm Neptune” <a href="https://en.wikipedia.org/wiki/Gliese_436_b" target="_blank">GJ436b</a>, a smaller and colder planet than hot Jupiters. At the same time, we were collecting several sets of observations of this planet with the Hubble Space Telescope. After analyzing all the collected data, we found that the transit of the hydrogen cloud starts about two hours before, and ends more than three hours after the transit of the planet. “Transit” is the moment when a planet crosses the disk of its star as seen from Earth. When this occurs, one can measure a small diminution of the stellar light because the planetary disk occults part of the stellar disk. We also found that the cloud is much larger than previously thought, and occults the maximum surface of the star when the planet is at the center of its transit. At this point, the cloud decreases the stellar light by nearly 60 %, far beyond the 0.69 % caused by the planet itself, and about four times higher than for known evaporating hot Jupiters! We inferred from this detection that GJ436b is surrounded by a giant cloud of neutral hydrogen, which trails behind the planet like the tail of a comet.</p>\r\n\r\n<p>Using our 3D numerical model of evaporation, EVE (EVaporating Exoplanets), we simulated the structure of this cloud on the computer and compared the results with the actual observations. This way, we found that GJ436b loses between 100 to 1000 tons of neutral hydrogen per second. At first, this appeared surprising because these rates are lower than for hot Jupiters, even though the cloud we observed around GJ436b is larger than any other surrounding hot Jupiters. The difference lies in the fact that GJ436b is orbiting an <a href="https://en.wikipedia.org/wiki/Red_dwarf" target="_blank">M-dwarf</a> (or Red Dwarf) star that is called <a href="https://en.wikipedia.org/wiki/Gliese_436_b" target="_blank">GJ436</a>, at about the same distance as evaporating hot Jupiters orbit Sun-like stars. Because M-dwarfs are colder than the Sun, they emit less energy (e.g. heat and light), and GJ436b is much less irradiated (which is why it is called a ‘warm’ Neptune). This irradiation also takes the form of <a href="https://en.wikipedia.org/wiki/Radiation_pressure" target="_blank">radiation pressure</a>: when <a href="https://en.wikipedia.org/wiki/Photon" target="_blank">photons</a> emitted by the star encounter the hydrogen atoms escaping from the planet’s atmosphere, the impact exerts a small pressure on the atoms that repels them away from the star. In the case of hot Jupiters, their very bright stars emit so many photons that radiation pressure is stronger than the gravity of the star and blows the escaping atmosphere away from the planet. Because GJ436 is darker, emitting fewer photons, its smaller radiation pressure cannot blow the gas away. Instead, it brakes its fall toward the star and allows it to diffuse within a large volume around the planet, thus creating the observed giant cloud of hydrogen.</p>\r\n\r\n<p>GJ436b, due to its relatively high mass and dark star, will never lose its atmosphere completely; however,exoplanets with lower mass and/or more energetic stars may have evolved more dramatically: if such planets completely evaporated their atmospheres, this could lead to the formation of rocky, atmosphere-less remnants. Some of the exoplanets that are observed today with sizes close to that of the Earth, but very close to their star, could be such remnants. With the American and European surveys of exoplanets, like <a href="https://en.wikipedia.org/wiki/Transiting_Exoplanet_Survey_Satellite" target="_blank">TESS</a> and <a href="https://en.wikipedia.org/wiki/PLATO_%28spacecraft%29" target="_blank">PLATO</a>, coming in the next decades, we expect to discover many examples of evaporating exoplanets, from moderate cases like GJ436b to planets suffering catastrophic loss of their atmosphere. This will help us to understand the formation and evolution of Earth-like planets.</p>\r\n', 4, 'Ehrenreich D, Bourrier V, Wheatley P et al. <a href="http://www.nature.com/nature/journal/v522/n7557/full/nature14501.html" target="_blank"><strong>A giant comet-like cloud of hydrogen escaping the warm Neptune-mass exoplanet GJ 436b</strong></a><strong>.</strong> <i>Nature</i>. 2015;522(7557):459-461. doi:10.1038/nature14501.', 3, '2015-11-26 16:14:51', 27, NULL, 11, 'https://doi.org/10.25250/thescbr.brk005'),
(41, 'Where is the Engram?', '<i>The author Tonegawa won the <b>Nobel Prize for Physiology or Medicine in 1987.</b></i>\r\n<br><br>\r\n<p>The human brain is composed of over 100 billion electrically-active brain cells (called <a href="https://en.wikipedia.org/wiki/Neuron">neurons</a>), and what makes the neuron a special cell type is that it sends out extensions (known as <a href="https://en.wikipedia.org/wiki/Axon">axons</a>) that form connections with other neurons. These connections are known as <a href="https://en.wikipedia.org/wiki/Synapse">synapses</a>, and allow for the direct communication of neurons with one another through the release of chemical substances called <a href="https://en.wikipedia.org/wiki/Neurotransmitter">neurotransmitters</a>. Populations of neurons form intricate wiring networks in the brain, resulting in trillions of different synapses. Intriguingly the structure and strength of synaptic connections can change with experience. Because of this plasticity, and the inherent complexity of synaptic networks, it has long been assumed that memories are stored through these connections. Yet it has been difficult for scientists to study this in detail because looking for the neurons involved in any particular memory is like looking for a particular grain of sand at the beach. The challenge therefore is to identify which neurons are holding a given memory, which we refer to as the memory <a href="https://en.wikipedia.org/wiki/Engram_%28neuropsychology%29">engram</a>, and then to study their synaptic connections.</p>\r\n\r\n<p>In <a href="https://en.wikipedia.org/wiki/Arthur_Conan_Doyle">Arthur Conan Doyle</a>’s “<a href="https://en.wikipedia.org/wiki/A_Scandal_in_Bohemia">A Scandal in Bohemia</a>”, Sherlock Holmes devises a plan to retrieve a compromising photograph of the King of Bohemia hidden in Irene Adler’s home. Dr. Watson questions how Holmes will know where to look for it. Holmes retorts – “I will not look...I will get her to show me”. By releasing smoke, the photograph’s location is revealed through Adler’s behaviour because “when a woman thinks that her house is on fire, her instinct is…to rush to the thing which she values most”. If we are to identify a specific memory we can take a similar approach, by hijacking the biological mechanisms of memory allocation. One of the ways that the brain allocates information is through specific genes that are induced by brain activity to produce specific proteins. We modified these brain-activity induced genes to produce a <a href="https://en.wikipedia.org/wiki/Fluorescence">fluorescent</a> protein that can easily be observed. This way, we genetically engineered a mouse whose brain cells lit up brightly when a certain memory was triggered and were able to identify some of the cells that store a specific memory—the memory engram cells.</p>\r\n\r\n<p>Having identified these cells, we could study their biology directly. In doing so, evidence was found implying that engram cells carrying a specific memory have more synaptic input than other neurons. Furthermore, when these particular synapses were experimentally disrupted by drug treatment, the mice displayed <a href="https://en.wikipedia.org/wiki/Amnesia">amnesia</a> for the associated memories; in other words, they could no longer remember the events stored in these places. Thus, these engram cell properties seem to be important for memory function.</p>\r\n\r\n<p>But are these memories lost or just not accessible? The ultimate evidence for a memory being present is the behavioural fact of the animal recalling the memory. We developed a method to directly switch on a particular memory in the mouse by specifically activating the associated engram cells. Upon activation of these cells, memory recall occurs as shown by the expected behaviour of the mouse. When we stimulated engram cells whose synapses had been disrupted before, leading to amnesia, the mice still showed the expected behaviour, i.e. they remembered what was stored in these cells. This means that the memory survived the amnesia and was still present in the brain.</p>\r\n\r\n<p>How, then, is the memory stored in the brain in the absence of the strengthened engram synapses? Further analysis revealed that engram cells showed a unique pattern of synaptic connectivity with other engram cells in different brain regions. Though the strength of these synapses was compromised by the amnesia, the basic pattern of connection in the network remained intact. This is why the memory could be recovered by directly switching-on the engram cells.</p>\r\n\r\n<p>When learning occurs, small populations of neurons are simultaneously activated in different brain regions relevant to that experience, and become engram cells. These engram cell populations form preferential networks across brain regions which likely represent the storage of the memory information itself. In parallel, the synaptic connections between engram cells are reinforced which facilitates efficient access and allows them to be easily retrieved when a memory is needed. These dual properties of engram cells provide adaptive value to the organism: stable synaptic networks allow long-term memories to be stored for a lifetime. On the other hand, dynamic changes in synaptic strength permit the organism to quickly and reversibly alter the relative accessibility of existing memories, without the cost of losing them entirely. Based on these findings, it is likely that many cases of clinical amnesia such as Alzheimer’s disease, stroke, and traumatic brain injury are caused by disruption of the brain’s access to specific memory engrams. “Lost” memories may still be present in the brains of patients suffering from these conditions, and could be retrievable if novel clinical interventions are developed. The trick, as Sherlock Holmes might suggest, will be to get the brain to show us where they are.</p>\r\n', 4, 'Ryan T, Roy D, Pignatelli M, Arons A, Tonegawa S. <strong><a href="http://www.sciencemag.org/content/348/6238/1007.short" target="_blank">Engram cells retain memory under retrograde amnesia</a></strong>. <i>Science</i>. 2015;348(6238):1007-1013. doi:10.11', 2, '2015-12-14 16:17:10', 29, NULL, 11, 'https://doi.org/10.25250/thescbr.brk006'),
(42, 'The lingering effects of parental care and its role in evolutionary change', '<p>For centuries, European culture has been enriched by depictions in art and literature of the diverse ways in which parents can exert a long-lasting influence on their children. We now know that animal parents can have similarly lingering effects on their offspring and a relatively new and growing area of research in <a href="https://en.wikipedia.org/wiki/Evolutionary_biology" target="_blank">evolutionary biology</a> sets out to understand the causes and consequences of these influences. Formally known as ‘parental effects’, and defined as non-genetic influences of the parent on their young, the phenomenon is of particular interest because it can potentially speed up the pace of evolutionary change that allows animals to adapt to changing environments. This may prove vital for the continued survival of animal populations whose habitats are undergoing rapid man-made alterations.</p>\r\n\r\n<p>Recent research in this area has revealed two general ways in which so-called parental effects might facilitate evolution. They can influence offspring <a href="https://en.wikipedia.org/wiki/Fitness_%28biology%29" target="_blank">fitness</a>, promoting some individuals while dooming others. And they can potentially provide a mechanism for non-genetic inheritance. For example, individuals hatching from large eggs might then be more likely to produce large eggs themselves upon reaching maturity. Parental effects that cascade across the generations like this can accelerate the pace at which an adaptation for a new environment spreads through the population.</p>\r\n\r\n<p>In this study, we were interested specifically in understanding how parental effects might modulate the fitness gained from particular acts of animal behaviour. An animal’s behaviour is one of the first traits to change and evolve when it enters a novel environment, and we wanted to know how parents might contribute to this process. We chose parental care as our behaviour of interest, because its adaptive value can readily be assessed by balancing the fitness benefits gained (by assessing offspring performance) against any fitness costs incurred by the parent from carrying out parental care (by assessing parental performance after reproduction). We wanted to know (i) how parental effects might change the fitness costs and benefits associated with parental care for any given individual, and (ii) whether handicapping individuals in this way might have associated fitness consequences for their social partner, with whom they raise young.</p>\r\n\r\n<p>Our experiments exploited the unusual natural history of the burying beetle, <a href="https://en.wikipedia.org/wiki/Nicrophorus_vespilloides" target="_blank"><em>Nicrophorus vespilloides</em></a>. Like other <a href="https://en.wikipedia.org/wiki/Burying_beetle" target="_blank"><em>Nicrophorus</em></a> beetles, this species uses the carcass of a small vertebrate for reproduction. It exhibits elaborate biparental care, though one parent (of either sex) can raise offspring singlehandedly. Parents prepare the carcass for reproduction by stripping it of fur or feathers, rolling the flesh into a ball and interring it in a shallow grave. There the corpse becomes an edible nest for the larvae, which feed themselves upon the flesh and are, additionally, fed from time to time by the parents. Within a week of hatching, just a few bones remain of the carcass, the parents fly off in search of new mating opportunities and the larvae crawl away to <a href="https://en.wikipedia.org/wiki/Pupa" target="_blank">pupate</a> in the soil. Roughly five weeks later, they are sexually mature adults and the life cycle begins again. In our previous work, we discovered that we could remove parents at any stage from just prior to hatching to larval dispersal and the larvae would survive. This gave us a way to manipulate the magnitude of the parental effect experienced by offspring – by changing the extent of care they received. The short generation time (outlined above) meant that we could easily track the consequences when the larvae matured and exhibited parental behaviour themselves, to see how care supplied in one generation influenced the supply ofcare in the next.</p>\r\n\r\n<p>Our experiments showed that adults which received low levels of care as larvae were less successful at raising larger broods and suffered greater mortality afterwards: they were low-quality parents. Well-nourished offspring, conversely, matured into high-quality parents. Furthermore, when raising offspring with higher-quality parents, low-quality individuals caused their partners to suffer raised levels of mortality following reproduction, presumably because they had to work harder to compensate for the inferior partner’s shortcomings.</p>\r\n\r\n<p>The evolution of new adaptive behaviour depends both on a mechanism for inheritance of behavioural traits from generation to generation and on the net fitness gained by performing that behaviour. Our study shows that parental effects not only influence the inheritance of a behavioural trait (because, for example, well-cared for larvae mature into high-quality parents), but that also they alter the fitness gains associated with that behaviour, with knock-on fitness consequences for their social partner.</p>\r\n\r\n<p>Thus, our experiment indicates how parental effects can potentially provide a mechanism for the rapid evolution of new adaptive social behaviour in the face of environmental change.</p>\r\n', 4, 'Kilner R, Boncoraglio G, Henshaw J et al. <a href="http://elifesciences.org/content/early/2015/09/22/eLife.07340" target="_blank">Parental effects alter the adaptive value of an adult behavioural trait</a>. <i>eLife</i>. 2015;4:e07340. doi:10.7554/eLife.0', 4, '2016-01-27 16:28:55', 30, NULL, 11, 'https://doi.org/10.25250/thescbr.brk007'),
(43, 'A dangerous habit: bees prefer pesticide-contaminated nectar', '<p>The impact of pesticides on pollinators is an important factor for the future of world food security, as well as a hotly debated and controversial topic. Pollinating insects like bees help to increase the yields of many food crops but, in doing so, are inadvertently exposed to pesticides in floral nectar and pollen.</p>\r\n\r\n<p><a href="https://en.wikipedia.org/wiki/Neonicotinoid" target="_blank">Neonicotinoids</a> represent a class of neuro-active <a href="https://en.wikipedia.org/wiki/Insecticide" target="_blank">insecticides</a> with a chemical structure similar to that of <a href="https://en.wikipedia.org/wiki/Nicotine" target="_blank">nicotine</a>, and are widely used to protect crops from insect pests. They have been introduced in the early 1990s as a seed treatment for crops such as <a href="https://en.wikipedia.org/wiki/Rapeseed" target="_blank">oilseed rape</a> or sunflowers that are pollinated by bees. As the plant grows, it absorbs some of the insecticide from the seed treatment and the substance spreads to all plant parts including the pollen and nectar consumed by pollinators like bees<sup>5</sup> (read also the Break: <b><a href="https://www.thesciencebreaker.com/break.php?breakId=57" target="_blank">The use of neonicotinoid pesticides affects wild bee populations</b></a>).<p>\r\n\r\n<p>Several reports have shown that low concentrations of neonicotinoids are not directly lethal for social bees but can alter their behaviour and reduce survival of entire colonies<sup>1-3</sup>. These studies have raised major public concern over bee welfare and, in 2013, led to a <a href="http://ec.europa.eu/food/archive/animal/liveanimals/bees/neonicotinoids_en.htm" target="_blank">two-year ban</a> on the use of the three most commonly used neonicotinoids (imidacloprid (<a href="https://en.wikipedia.org/wiki/Imidacloprid" target="_blank">IMD</a>), thiamethoxam (<a href="https://en.wikipedia.org/wiki/Thiamethoxam" target="_blank">TMX</a>), clothianidin (<a href="https://en.wikipedia.org/wiki/Clothianidin" target="_blank">CLO</a>)) on flowering crops by the European Union. On the other hand it has been argued by critics that these pesticides are only deleterious to bees at concentrations higher than those present in the nectar and pollen of treated plants. Furthermore, it has been suggested that bees could choose to forage on other available flowers growing at the periphery of treated crops and hence avoid or reduce exposure<sup>4-5</sup>. Because this hypothesis had never been verified experimentally, it was time to test whether bees are, in fact, able to detect neonicotinoid pesticides and avoid contaminated nectars.</p>\r\n\r\n<p>The experiments were performed on the foragers of two social bee species: the honeybee, <a href="https://en.wikipedia.org/wiki/Western_honey_bee" target="_blank"><em>Apis mellifera</em></a>, and the buff-tailed bumblebee, <a href="https://en.wikipedia.org/wiki/Bombus_terrestris" target="_blank"><em>Bombus terrestris</em></a>. Bees kept in complete darkness in experimental cages were offered a choice between a sucrose solution and a sucrose solution laced with a specific concentration (1-1000 nM) of imidacloprid (IMD), thiamethoxam (TMX) or clothianidin (CLO). The food consumption was estimated 24 hours later by weighing the feeding tubes. Using this experimental set up, we found that bees did not avoid the neonicotinoid pesticides. Unexpectedly, we found that bees of both species actually preferred to eat more of the sucrose solutions laced with IMD or TMX than sucrose alone.</p>\r\n\r\n<p>We also investigated whether the <a href="https://en.wikipedia.org/wiki/Gustatory_system#Taste_as_a_form_of_chemoreception" target="_blank">gustatory neurons</a> in the bees’ mouthparts (the equivalent of the taste cells on the human tongue) are able to perceive the neonicotinoids. To this end, we used electrophysiological recordings allowing us to quantify the electrical signals generated by the neurons. To date, we haven’t found any gustatory neurones responding to stimulation with IMD, TMX and CLO. Similarly, no evidence was found that low doses of the nenonicotinoid pesticides affect the response of the gustatory neurones responding to sucrose.</p>\r\n\r\n<p>In summary, our data indicate that bees are not deterred by neonicotinoids because they may not be able to detect them in the nectar. Instead, bees do actually prefer solutions containing low concentrations of IMD or TMX, even though the consumption of these pesticides caused them to eat less food overall. These results are alarming because they show that bees cannot control their exposure to neonicotinoids through intake of nectar. In addition, bees may not only endanger themselves, but eventually bring more neonicotinoid-laced nectar back to the colony, exposing the entire hive. This implies that treating flowering crops with IMD and TMX presents a sizeable hazard to bees and strategies such as planting flowers in field margins may not be enough to reduce the risk of bee poisoning. A global reduction of the use of pesticides might be the only way to limit the pollinator population decline.</p>\r\n', 3, 'Kessler S, Tiedeken E, Simcock K et al. <a href="http://www.nature.com/nature/journal/v521/n7550/full/nature14414.html" target="_blank">Bees prefer foods containing neonicotinoid pesticides.</a> <i>Nature</i>. 2015;521(7550):74-76. doi:10.1038/nature14414', 3, '2016-02-12 16:30:00', 31, NULL, 11, 'https://doi.org/10.25250/thescbr.brk008');
INSERT INTO `breaks` (`id`, `title`, `content`, `reading_time`, `original_article`, `subject_id`, `date_created`, `author1_id`, `author2_id`, `editor_id`, `imzy_link`) VALUES
(44, 'Fighting back antibiotic resistance: a new hope from the soil', '<p>Bacteria live in a hectic world. They need to find food and a happy place to live all while dividing every 20 or so minutes. To complicate things further, they must also outcompete the other bacteria they share a space with. In the human gut, for example, there may be up to 1,000 different species of bacteria<sup>5</sup>. That''s a lot of competition! In order to outcompete each other for nutrients, some bacteria produce molecules that seek out and destroy other species of bacteria. This is one of the ways that <a data-mce-href="https://en.wikipedia.org/wiki/Antibiotics" href="https://en.wikipedia.org/wiki/Antibiotics">antibiotics</a> are created. Most of the antibiotics that humans use are derived from different bacterial strains, or other small organisms like fungi.</p>\r\n\r\n<p>Because bacteria grow and divide very fast, they can quickly adapt to new surroundings. When we use an antibiotic to get rid of disease-causing bacteria, we eliminate all the bacteria in that colony that are susceptible to the antibiotic. However, there will be a small portion immune to the antibiotic that will survive and continue to grow. This leaves us with a new colony of the same disease-carrying bacteria that are now resistant to the antibiotic. Antibiotic resistance is a major issue<sup>2,3</sup> and doesn''t appear to be going away any time soon. This means we are in a constant need of new antibiotics.</p>\r\n\r\n<p>To combat antibiotic resistance, scientists search for new bacteria that naturally produce antibiotics, as synthesizing new antibiotics from scratch is a costly and extremely difficult task. In order to find new antibiotics, researchers need to keep the bacteria growing in a lab, in a controlled environment, to test each for their ability to produce antibiotics. There is a problem with this, however - the majority (nearly 99%) of bacteria scientists find won''t grow successfully in the lab using the usual, artificial techniques<sup>1</sup>. This leaves us with a small percentage of bacteria that we can successfully grow. But, we have been drawing from this well for so long that the pool of untested bacteria is rapidly shrinking.</p>\r\n\r\n<p>To combat this, about 5 years ago, scientists developed a new technique that allows us to grow strains of bacteria that we were unable to culture before - expanding our pool of testable bacteria. The technique involves a tool named <a data-mce-href="https://en.wikipedia.org/wiki/Isolation_chip" href="https://en.wikipedia.org/wiki/Isolation_chip">iChip</a> that has thousands of tiny wells which allow researchers to cultivate individual colonies of rare bacteria directly in the soil they call home. In this way, they can grow up to 10,000 unique strains of bacteria in their natural environment. This was a breakthrough in our ability to grow bacteria in the lab. It increased the amount of bacteria that we could cultivate by about 50 fold.</p>\r\n\r\n<p>Recently, a team of researchers used this technique to search for new antibiotics<sup>4</sup>. In a new strain of bacteria that couldn''t be grown before the iChip, they discovered a new antibiotic they named <a data-mce-href="https://en.wikipedia.org/wiki/Teixobactin" href="https://en.wikipedia.org/wiki/Teixobactin">teixobactin</a>. This drug has a unique mechanism, as it attacks a molecule that is vital for bacteria to build its membrane. The process that teixobactin disrupts is so basic and essential, it makes it nearly impossible for susceptible bacteria to mutate and develop a resistance. Their research showed that, compared to other popular antibiotics, bacteria aren''t developing a resistance. In an experiment where they measure the amount of antibiotic it takes to kill a colony of bacteria in successive generations (usually each successive generation will be more resistant), the same amount of antibiotic was as effective in the first generation as it was to the 25<sup>th</sup>. This is amazing especially compared to other antibiotics that required 256 times the normal dose to kill the same amount of bacteria at the 25<sup>th</sup> generation. They then tested teixobactin in several different disease models in mice, including <a data-mce-href="https://en.wikipedia.org/wiki/Pneumonia" href="https://en.wikipedia.org/wiki/Pneumonia">pneumonia</a>, to find it was more efficient in eradicating the disease-causing bacteria than other effective antibiotics. So, not only is this antibiotic difficult to build a resistance to, but it is also more efficient at eradicating bacteria. A real win win!</p>\r\n\r\n<p>This discovery represents a major breakthrough in the field, and our lives in general. It shows that the iChip method is successful in growing difficult-to-culture strains of bacteria. It has vastly increased the pool of bacteria we can search through to find new antibiotics. This research also shows that some of the antibiotics we find could be extremely useful, with potentially new targets that make building a resistance nearly impossible. It may be too soon to tell, but hopefully these findings are just the prologue to many more future discoveries.</p>\r\n', 4, 'Ling L, Schneider T, Peoples A et al. <a href="http://www.nature.com/nature/journal/v517/n7535/full/nature14098.html" target="_blank">A new antibiotic kills pathogens without detectable resistance.</a> <i>Nature</i>. 2015;517(7535):455-459. doi:10.1', 6, '2016-02-24 16:33:04', 32, NULL, 11, 'https://doi.org/10.25250/thescbr.brk009'),
(45, 'The colour beige: heating up the fat', '<p>In mammals, <a data-mce-href="https://en.wikipedia.org/wiki/Adipose_tissue" href="https://en.wikipedia.org/wiki/Adipose_tissue" target="_blank">adipose tissue</a> is a specialized tissue that functions as the major storage site for fat. When glucose supplies are low (for example when an organism is food-deprived), the stored fat can be used to produce energy, so that the organism may be constantly supplied of the fuel it needs to survive. For this purpose, mammals possess two different types of adipose tissue: (i) white fat, used to store metabolic energy (necessary for organisms'' physiological activities) and (ii) brown fat, specialized in storing energy that will be used to generate heat in mammalian new-borns and adult hibernating mammals.</p>\r\n\r\n<p>Across the animal kingdom, brown fat is essential to avoid hypothermia and is of great importance as cold is a major death risk factor for premature infants. Brown fat is indeed the site of the <a data-mce-href="https://en.wikipedia.org/wiki/Thermogenesis" href="https://en.wikipedia.org/wiki/Thermogenesis">thermogenesis</a>, a process of heat production mainly regulated by the <a data-mce-href="https://en.wikipedia.org/wiki/Sympathetic_nervous_system" href="https://en.wikipedia.org/wiki/Sympathetic_nervous_system">sympathetic nervous system</a> and triggered by cold conditions.</p>\r\n\r\n<p>In developed countries, food availability and high sugar-content diets have dramatically increased the rate of food-related diseases, the most widespread of them being <a data-mce-href="https://en.wikipedia.org/wiki/Obesity" href="https://en.wikipedia.org/wiki/Obesity">obesity</a>. Obesity itself represents a major epidemiologic challenge associated with a higher incidence of a number of diseases including diabetes, cardiovascular disease, and cancer. Since brown fat can be readily "eliminated" by the organism through thermogenesis, several research groups investigated (i) the presence and abundance of brown fat in adults, and (ii) the possibility of increasing the total amount of brown fat, or by turning white fat into brown fat.</p>\r\n\r\n<p>While tackling the above-mentioned questions, a group of scientists from different Universities has analysed the fat depots in mouse models. This led to the isolation of a new type of fat cells termed <em>beige fat</em>. This particular name has been chosen because these cells showed features of both white and brown fat cells. Importantly, given the fact that beige cells shared brown fat''s capacity to produce heat, they have been described as a novel type of thermogenic fat cells. The experiments that lead to the identification of the beige adipocytes, started from the observation that some <a data-mce-href="https://en.wikipedia.org/wiki/Subcutaneous_tissue" href="https://en.wikipedia.org/wiki/Subcutaneous_tissue">subcutaneous</a> (white) adipose depots had characteristics more similar to those of brown fat cells than to those of white adipose tissues. Moreover, the level of a protein significantly responsible for heat production called <a data-mce-href="https://en.wikipedia.org/wiki/Thermogenin" href="https://en.wikipedia.org/wiki/Thermogenin">UCP1 or thermogenin</a> (normally abundant in brown cells and very low in white cells) was demonstrated to be different between white tissues derived from different locations. This evidence led the scientists to reconsider and rename several tissues (previously defined as white fat) as beige fat.</p>\r\n\r\n<p>Among the different techniques that scientists used to discriminate between different tissues, a procedure called <a data-mce-href="https://en.wikipedia.org/wiki/Reverse_transcription_polymerase_chain_reaction" href="https://en.wikipedia.org/wiki/Reverse_transcription_polymerase_chain_reaction">RT-PCR</a>, able to measure the activity of different genes, was used to compare the activity of <em>UCP1</em> (the gene producing thermogenin) and others genes in the three fat tissues. This provided the group of scientists with the evidence that beige cells show a unique genetic profile. Following these results, the amount of chemical and thermal energy produced by different types of fat cells, was assessed using a technique called <a data-mce-href="https://en.wikipedia.org/wiki/Respirometry" href="https://en.wikipedia.org/wiki/Respirometry">micro-respirometry</a>. This was done to further discriminate between white and brown/beige cells. Then, human brown fat biopsies were analysed for the presence of both brown and beige markers. In order to identify molecules that are typically present in beige or brown tissues, their distribution was visualized using <a data-mce-href="https://en.wikipedia.org/wiki/Immunofluorescence" href="https://en.wikipedia.org/wiki/Immunofluorescence">immunofluorescence</a>, a technique that takes advantage of engineered fluorescent proteins able to recognize and label the target molecules. In doing this, the authors were able to specifically target and visualize beige fat cells and they were able to map their distribution in the different biopsies.</p>\r\n\r\n<p>In summary, this research demonstrated for the first time, the unexpected existence of beige fat and its abundance in adult mammals. Before such results were published, it was assumed that there was too little brown fat present in human adults to use it as an indirect mean to reduce body weight. Because of that, the discovery of abundant beige fat with similar characteristics to those of brown fat, has been a remarkable breakthrough in this field of research. Consistently, the results reported in this research are still used as a reference for several biomedical research studies that investigate the use of brown and beige cells for the treatment of obesity and metabolic diseases<sup>1-2</sup>.</p>\r\n', 3.5, 'Wu J, Boström P, Sparks L et al. <a href="http://www.cell.com/abstract/S0092-8674%2812%2900595-8" target="_blank">Beige Adipocytes Are a Distinct Type of Thermogenic Fat Cell in Mouse and Human.</a> <i>Cell</i>. 2012;150(2):366-376. doi:10.1016/j.cell.2012.05.016', 1, '2016-04-01 15:34:57', 33, NULL, 11, 'https://doi.org/10.25250/thescbr.brk010'),
(46, 'Attempts to Forget the Past Make It Harder to Remember the Present', '<p>An old photograph, the smell of warm apple pie, or even a long-forgotten name overheard at a party all have the potential to unleash a flood of related memories. In many ways, our conscious experience is dictated by the reminders we encounter in daily life. Sometimes, however, we are motivated to exert conscious control over which thoughts are permitted to rise to awareness, either by actively seeking to remember particular instances or by suppressing inappropriate, unpleasant, or otherwise distracting thoughts. While everyday life presents numerous opportunities to engage in these control behaviors, individuals coping in the aftermath of a major life trauma may be especially likely to turn to memory suppression to curb flashbacks. Just how effective are these suppression attempts? Do they reduce the likelihood that the unwanted memories will threaten to rise to the surface in the future? And, importantly, are there unintended consequences of memory suppression?</p>\r\n\r\n<p>Our recent research has revealed that trying to suppress the past has both direct effects on the unwanted memories themselves and profound side effects for unrelated memories. When confronted by strong and specific reminders of unwanted memories, individuals tend to engage portions of the brain''s frontal lobe known to help with stopping inappropriate behaviors (like remembering something that we don''t want to remember)<sup>1</sup>. The <a data-mce-href="https://en.wikipedia.org/wiki/Prefrontal_cortex" href="https://en.wikipedia.org/wiki/Prefrontal_cortex" target="_blank">prefrontal cortex</a> does so by sending inhibitory ("stop") signals to the <a data-mce-href="https://en.wikipedia.org/wiki/Hippocampus" href="https://en.wikipedia.org/wiki/Hippocampus" target="_blank">hippocampus</a> - a region of the brain associated with retrieving old memories, as well as forming new ones<sup>2,3</sup>. We can directly measure the associated decreases in hippocampal activity with <a data-mce-href="https://en.wikipedia.org/wiki/Functional_magnetic_resonance_imaging" href="https://en.wikipedia.org/wiki/Functional_magnetic_resonance_imaging" target="_blank">functional magnetic resonance imaging (fMRI)</a> and establish the lingering difficulty in retrieving memories that had been suppressed previously using surprise recall tests at later points in time.</p>\r\n\r\n<p>Such targeted memory suppression effects may be all well and good if one is trying to prevent a particular unwanted memory from coming back to mind. The issue is that the brain''s response can be rather sluggish, meaning that other memories - innocent bystanders that later we might like to remember - can get lost in the process of ramping up or down the hippocampal memory control system. In other words, suppressing unwanted memories not only makes it harder to remember those particular memories later, it also has the power to induce <a data-mce-href="https://en.wikipedia.org/wiki/Amnesia" href="https://en.wikipedia.org/wiki/Amnesia" target="_blank">amnesia</a> for other experiences that happen just before or after one tries to suppress unrelated memories.</p>\r\n\r\n<p>To test for this side effect, we first asked participants to study a number of arbitrary word pairs (e.g., LEAP-BALLET). Participants were then required to either retrieve or suppress the second word from these pairs, given the first. In between these tasks, we presented participants with novel experiences, like a picture of a peacock standing in a parking lot. While participants weren''t aware of this at the time, our primary research question was whether suppressing words (like BALLET) around the introduction of the bystander pictures would impair subsequent memory for details about those unrelated experiences. This was assessed in a surprise final memory test, in which participants were asked to recall the associated bystander objects (e.g., the peacock) for each presented background scene (e.g., the parking lot).</p>\r\n\r\n<p>We were amazed to find that participants'' memory for these bystander pictures was reduced by nearly half, simply because they had engaged in an unrelated word-suppression task seconds before or after an image of the peacock in the parking lot appeared. Interestingly, they had no problem remembering that they had seen a peacock - they just couldn''t remember where they had seen it: Was it in a parking lot or a zoo, for instance? The observed amnesia effect proved robust in our series of seven experiments with nearly 400 participants, surviving long delays and growing with suppression practice. Moreover, it was possible to predict the magnitude of the amnesia effect by observing hippocampal activity in participants'' brains.</p>\r\n\r\n<p>Together with previously reported forms of forgetting (e.g. inattention, decay over time, interference from related memories, and brain damage), this new type of suppression-induced amnesia may add to explanations as to why individuals recovering from major traumas often have generalized memory impairments for experiences unrelated to the traumatic events themselves<sup>4-6</sup>. To the extent that suppressing memories of the past creates a temporary learning impairment for the present, individuals regularly employing such a suppression strategy to cope with chronic memory intrusions may be particularly susceptible to the amnesic side effects.</p>\r\n\r\n<p>By learning more about the neural and cognitive mechanisms that allow us to remember when we want to remember and forget when we want to forget, we can identify memory control strategies that could help us to let go of traumatic experiences of the past without losing memories of the present.</p>\r\n', 4, 'Hulbert, J.C., Henson, R.N., Anderson, M.C. <a href="http://www.nature.com/ncomms/2016/160315/ncomms11003/full/ncomms11003.html" target="_blank">Inducing amnesia through systemic suppression</a>. Nature Communications. 2016; 7: 11003. doi:10.1038/ncomms11003', 2, '2016-04-08 15:36:28', 34, NULL, 1, 'https://doi.org/10.25250/thescbr.brk011'),
(47, 'Chimpanzees Trust Their Friends', '<p>Human friendships are often characterized by preferential intentions and attitudes including trusting expectations of close social relations. Humans largely trust only their friends with crucial resources or important secrets. In this study, we investigated whether <a data-mce-href="https://en.wikipedia.org/wiki/Chimpanzee" href="https://en.wikipedia.org/wiki/Chimpanzee" target="_blank">chimpanzees</a> show a comparable pattern and extend trust selectively toward those individuals they are closely bonded with.</p>\r\n\r\n<p>A large body of research shows that chimpanzees cooperate with each other in a wide variety of contexts including grooming, meat-sharing, group-hunting, and communal border patrols. Many different psychological mechanisms might account for such collaborative behaviors. For example, chimpanzees may cooperate with dominant individuals out of a fear of being punished. Alternatively, chimpanzees might show trust-based cooperation, i.e. chimpanzees might cooperate because they expect their partners to behave in ways that are consistent with their welfare. In other words, they might cooperate because they expect their partners to be trustworthy.</p>\r\n\r\n<p>In short, while chimpanzee cooperation is well-established, the mechanisms underlying such behaviors remain poorly understood. As trust is the bedrock for human cooperation in many cases, we investigated whether the same might be true for one of our closest living relatives.</p>\r\n\r\n<p>To do so, we adapted the <a data-mce-href="https://en.wikibooks.org/wiki/Bestiary_of_Behavioral_Economics/Trust_Game" href="https://en.wikibooks.org/wiki/Bestiary_of_Behavioral_Economics/Trust_Game" target="_blank">human trust game</a> (used mostly by behavioral economists) for chimpanzees. In our version of the trust game, two chimpanzees participated, a truster and a partner - sitting opposite each other in two different rooms. Trusters could either pull the trust-rope or the no-trust rope - they had to make a choice. Pulling the no-trust rope resulted in immediate access to a small amount of food. Pulling the trust rope resulted in a large food box moving to a partner. The partner could then eat half of the contents of the food box (a large amount of food) and then either prove trustworthy by sending the other half back, or prove untrustworthy by not sending it back. So the no-trust option is safe but contains only a small amount of food. The trust option, on the other hand, is risky but potentially has high-payoffs (a lot of food). The trust option is risky because the partner might not send the food back, leaving the chimpanzee empty-handed.</p>\r\n\r\n<p>Chimpanzees made a choice between pulling the trust rope and the no-trust rope in twenty-four consecutive trials. In half of these trials, they interacted with a friend, in the other half with a non-friend. Social relationships were determined prior to the study by observing naturally occurring cooperative behaviors. Based on 352 hours of observation of such behaviors as grooming and eating together, we determined for each chimpanzee a friend and a non-friend.</p>\r\n\r\n<p>The main finding of the current study is that chimpanzees were much more likely to pull the trust rope when they interacted with a friend compared to a non-friend. Chimpanzees, like humans, thus seem to trust their friends more than their non-friends.</p>\r\n\r\n<p>A second finding of the current study is that, if trusted, friends and non-friends are equally likely to prove trustworthy (by sending the food back). This is surprising because chimpanzees still trust their friends significantly more than their non-friends. This is an unexpected finding as it suggests that chimpanzees do not regulate their behavior toward friends exclusively as a function of received rewards. Instead, they seem to form deep emotional bonds with their friends which are, at least to some extent, independent of short-term payoffs. Close relationships thus do not seem to be based on short-term forms of reciprocity but on more long-term and emotion-based mechanisms - a pattern that can be observed in human friendships as well.</p>\r\n\r\n<p>These findings suggest that trust within closely bonded <a data-mce-href="https://en.wikipedia.org/wiki/Dyad_%28sociology%29" href="https://en.wikipedia.org/wiki/Dyad_%28sociology%29" target="_blank">dyads</a> is not unique to humans, but rather shares evolutionary roots with the social relationships of our closest primate relatives. Human friendships do not represent an anomaly in the animal kingdom. Other animals, such as chimpanzees, form close and long-term emotional bonds with select individuals. These animal friendships show important parallels with close relationships in humans. One shared characteristic is the tendency to selectively trust friends in costly situations.</p>\r\n', 3.5, 'Engelmann JM, Herrmann E. <a href="http://www.cell.com/current-biology/abstract/S0960-9822%2815%2901431-1" target="_blank">Chimpanzees Trust Their Friends</a>. Current Biology. 2016;26(2):252-256. doi:10.1016/j.cub.2015.11.037.', 4, '2016-04-14 15:38:16', 35, NULL, 1, 'https://doi.org/10.25250/thescbr.brk012'),
(48, 'Attractive in the dark — how petunias may help to feed humanity', '<p>Many plants, including staple crops, need insects to reproduce. Changing climate and human interference threaten the sensitive relationships between plants and their pollinators. Many aspects of these relationships are not well understood. However, this knowledge may be crucial to sustain and increase crop production to feed an ever-growing human population.</p>\r\n<p><a data-mce-href="https://en.wikipedia.org/wiki/Petunia" href="https://en.wikipedia.org/wiki/Petunia" target="_blank">Petunias</a> are very popular garden and container plants around the globe. They are of South American origin, and were first described in 1803 from flowers collected in Uruguay. Favoured by gardeners for their diverse range of shapes and colours, petunias have also found their way into science: researchers use them to study flower pollination.</p>\r\n<p>The flowers of common garden petunia release substances, known as floral volatiles, to attract nocturnal pollinators, such as the <a data-mce-href="https://en.wikipedia.org/wiki/Sphingidae" href="https://en.wikipedia.org/wiki/Sphingidae" target="_blank">hawkmoth</a>. The plant depends on these insects for its reproduction. Release of the attractants must be timed so that the moths can find the flowers at night; but, they should not be produced wastefully during the day. All this is regulated by the plant''s internal, or circadian, clock. Many living beings, including humans, possess such a circadian clock. It tells us, for example, when to be tired, and is responsible for the infamous jet-lag. However, the precise mechanisms underlying the regulation of attractant release from petunia flowers have remained largely unknown, until recently.</p>\r\n<p>Researchers at the University of Washington in Seattle were keen to shed light on this regulation. They monitored production of different components related to the release of floral volatiles. They observed that even when the plants were kept in constant darkness the attractants were still released in regular 24-hour intervals - indicating control by the circadian clock.</p>\r\n<p>In order to learn more about the processes behind that control, the researchers benefitted from another plant that is commonly used for scientific studies: the thale cress, <a data-mce-href="https://it.wikipedia.org/wiki/Arabidopsis_thaliana" href="https://it.wikipedia.org/wiki/Arabidopsis_thaliana" target="_blank"><em>Arabidopsis thaliana</em>.</a></p>\r\n<p>The way the circadian clock of this plant works is well studied. An important part of the system is a protein that can switch certain mechanisms on or off depending on the circumstances. It is called LHY (for <u>L</u>ate Elongated <u>Hy</u>pocotyl). The researchers suspected that a similar factor was responsible for the regulation of timely attractant release in petunias. They found a close match, which they called PhLHY (for <em><u>P</u>etunia</em> <em><u>h</u>ybrida</em>, the Latin name of the common garden petunia). They discovered that PhLHY is mainly produced in the morning (a characteristic shared with its cousin from <em>Arabidopsis</em>). They also showed that excess PhLHY has the same effect on the plant as its distant cousin; namely, developmental problems, delayed flowering, and elongated stems (called <em>hypocotyl</em> in seedlings; hence the name of the protein). In short, both proteins appear to have the same biological role.</p>\r\n<p>The way PhLHY functions is intriguing: in its presence, the production of pollinator attractants is suppressed; however, since it is mainly produced in the morning, the suppressive effect wears off over the day. In the night, when almost all PhLHY is gone, the production and release of attractants are at a maximum - just in time for the hawkmoths!</p>\r\n<p>The identification of PhLHY provides the first mechanistic link between the circadian clock and emission of floral pollinator attractants. It may be an important step towards a better understanding of the processes that regulate plant-insect relationships.</p>\r\n<p>The scientists performing this work suggest that with this knowledge production of floral volatiles could be increased in certain crops to attract more pollinators. Even more interestingly, the timing of their release could be regulated so as to attract particular insects. In this way, crops could be engineered to capitalise on pollinators that predominate in a given region. This may open up new possibilities for farmers to cultivate areas that currently produce poor yields.</p>\r\n', 3, 'Fenske, M., Hewett Hazelton, K., Hempton, A., Shim, J., Yamamoto, B., Riffell, J. and Imaizumi, T. (2015). <a href="http://www.pnas.org/content/112/31/9775.abstract" target="_blank">Circadian clock gene LATE ELONGATED HYPOCOTYL directly regulates the timing of floral scent emission in Petunia</a>. <i>Proceedings of the National Academy of Sciences</i>, 112(31), pp.9775-9780.', 5, '2016-04-27 15:39:19', 36, NULL, 11, 'https://doi.org/10.25250/thescbr.brk013'),
(49, 'One run a day keeps the...cancer away!', '<p>A healthy life-style, including regular exercise, has long been associated with the prevention of diabetes and heart attack. Moreover, exercise helps to lower other major disease risk factors, e.g. obesity and high blood pressure. When it comes to cancer, it is suggested that exercise confers some protection, but to which extent, why, and how is largely unknown.</p>\r\n<p>We recently set out to study if exercise would impact tumor incidence and growth in mice. Obviously, if one wishes to study a potential role of exercise using animal models it is crucial that the chosen animal is willing to exercise. With a simple running wheel in the cage, a mouse will voluntarily run between 4 and 8 kilometres per night, suggesting that mice are indeed good models for this kind of research.</p>\r\n<p>We took several approaches to try and answer the question of the role of exercise in relation to cancer: first, we infected mice with skin cancer (melanoma) and lung cancer, respectively, and in both cases saw a significant delay in tumor growth in exercising animals. Secondly, we studied if exercise could lower the risk for development of cancer in the first place. Again, it was very clear that exercising mice had a lowered risk for development of tumors. Moreover, exercising mice that <em>did</em> get tumors had smaller tumors.</p>\r\n<p>Exercise really had an astonishing impact on both tumor growth and development! How does that come about?</p>\r\n<p>It is often believed that tumors consist of cancer cells only; however, they actually comprise numerous different cell types, including cells of the immune system. Interestingly, tumors from exercising mice contained much more immune cells, in particular a cell type called natural killer cells (NK cells). We could show that these NK cells play a key role by using mice that do not have them: in these mice, there was no anti-tumor effect of exercise.</p>\r\n<p>Thus, natural killer cells are crucial; but how does exercise lead to increased NK cells numbers?</p>\r\n<p>Exercise induces the secretion of a huge number of substances, amongst others the hormone adrenalin. Adrenalin exerts its function by interacting with receptors - so-called &beta;-adrenergic receptors. When we blocked this interaction, NK cell levels did no longer increase by exercise and the anti-tumor effect was abolished.</p>\r\n<p>So - the line of events leading to tumor control by exercise was revealed: (i) exercise leads to secretion of adrenalin; (ii) adrenalin interacts with &beta;-adrenergic receptors (in the spleen); (iii) NK cells are mobilized and enter the blood stream; (iv) transported by the blood, NK cells reach the tumors; there, they kill cancer cells and attract more immune cells to the tumors.</p>\r\n<p>Is this relevant for humans? We don''t know that yet, but we think so. We know that exercise in humans also leads to increased NK cell numbers. Thus, exercise could play a role in preventing cancer in man.</p>\r\n<p>To study such an effect in humans is not trivial because tumors develop over many years and are influenced by a range of different factors. One idea would be to take tumor biopsies before and after exercise. An increased number of NK cells after exercise - as seen in the mice - would strongly support the notion of an impact in humans.</p>\r\n<p>Potentially, regular exercise could delay or prevent cancer in humans - time will tell. But could exercise have an immediate impact on cancer patients and treatments today?</p>\r\n<p>Well - maybe it could. Several new treatments are based on a functional immune system, and it has been shown that patients with "immune-cell rich" tumors are more prone to benefit from those treatments. Consequently, huge efforts are undertaken to find ways by which the number of immune cells in tumors can be increased. If exercise could fulfil this role, it would likely be associated with a higher probability of response to treatment.</p>\r\n<p>It really goes without saying that exercise is healthy. But with the potential described above, exercise could soon be incorporated into routine oncology, not only being important for living a long and healthy life but also as an integral part of optimal cancer treatment.</p>\r\n', 3.5, 'Pedersen, L., Idorn, M., Olofsson, G., Lauenborg, B., Nookaew, I., Hansen, R., Johannesen, H., Becker, J., Pedersen, K., Dethlefsen, C., Nielsen, J., Gehl, J., Pedersen, B., thor Straten, P. and Hojman, P. (2016).<a href="http://www.cell.com/cell-metabolism/abstract/S1550-4131%2816%2930003-1" target="_blank">Voluntary Running Suppresses Tumor Growth through Epinephrine- and IL-6-Dependent NK Cell Mobilization and Redistribution.</a> <em>Cell Metabolism</em>, 23(3), pp.554-562.', 1, '2016-05-05 16:06:16', 37, NULL, 11, 'https://doi.org/10.25250/thescbr.brk014'),
(50, 'SERINC5: a blood cell guardian against HIV', '<p>HIV-1 is a virus that currently affects 36 million people worldwide. While the trend of the epidemic has slowed down in recent years thanks to a drug cocktail capable of efficiently inhibiting virus replication, neither a preventive vaccine nor an eradication therapy exist. New targets are therefore needed to develop therapeutic strategies complementary to the existing ones, in order to eradicate the infection.</p>\r\n\r\n<p>HIV carries proteins that the virus needs to infect new cells within infected individuals (<a data-mce-href="https://aidsinfo.nih.gov/education-materials/fact-sheets/19/73/the-hiv-life-cycle" href="https://aidsinfo.nih.gov/education-materials/fact-sheets/19/73/the-hiv-life-cycle" target="_blank">click here to read more about how HIV replicates within cells</a>). One of these proteins, called Nef, was found to be particularly important for the development of the syndrome linked to this virus, AIDS. This was demonstrated by rare cases of patients infected with HIV lacking the Nef protein. These patients remained healthy for a long time despite carrying the virus in their blood. This observation raised interest on the mechanism of action of Nef, and several research groups investigated its activity at the cellular and molecular levels. We now know that Nef is a multifunctional protein that carries out several activities for the virus'' particles. Among these activities, however, one has remained enigmatic: the importance of Nef for maximizing the ability of HIV to infect new cells.</p>\r\n\r\n<p>The aim of our research was to understand how the Nef protein enhances HIV infectivity. We first observed that the importance of Nef for the infectivity of HIV depends on the cell type in which the virus is generated. In fact, while Nef does not appear to improve the infectivity of HIV generated in cells from some tissue types, it is required by viruses produced in most <a data-mce-href="https://en.wikipedia.org/wiki/Haematopoiesis" href="https://en.wikipedia.org/wiki/Haematopoiesis" target="_blank">blood-derived cell types</a>. Such cell type-dependency suggested the existence of a factor in the cell which generates the virus, capable of inhibiting HIV and being neutralized by Nef. This factor must then be present in most blood-derived cells but absent in other cell types. To identify this key-regulator we screened for genes able to generate a certain factor only in blood-derived cell types but not in the cell lines where Nef is not required to produce infectious viruses. The investigation took advantage of a technique called <a data-mce-href="https://en.wikipedia.org/wiki/Massive_parallel_sequencing" href="https://en.wikipedia.org/wiki/Massive_parallel_sequencing" target="_blank">massive parallel sequencing</a>, capable of retrieving the sequence of all genes that are actively generating factors in a specific cell. The analysis identified SERINC5 as the only gene whose level of activity could accurately predict the effect of Nef on the infectious power of the virus. All subsequent experiments demonstrated that SERINC5 acts as a powerful inhibitor of HIV infectivity and, consistently, that this inhibitory effect is counteracted by Nef.</p>\r\n\r\n<p>The SERINC5 gene encodes for a protein inserted into the cell membrane and localized at the surface of the cell. HIV, like many other viruses, is surrounded by a membrane that the virus derives from the cell where it is generated. Our experiments have shown that when SERINC5 is present on the cell surface, it is incorporated into the membrane of the emerging virus particles. Then, with a mechanism yet to be understood, SERINC5 inhibits the infectivity of the newly produced virus by interfering with its ability to penetrate into a new cell. However, HIV developed a mechanism to overcome such inhibition. In fact, Nef protects the virus by removing the SERINC5 protein from the host cell surface, allowing the virus to emerge from the cell with a membrane that does not carry the cellular inhibitor.</p>\r\n\r\n<p>SERINC5 belongs to a family of five genes encoding similar membrane proteins, with functions that remain largely unknown. In addition to SERINC5, SERINC3 was also similarly found to act against HIV, suggesting a prevalent anti-retroviral role for this group of proteins.</p>\r\n\r\n<p>Our research has further unveiled that other retroviruses, which infect different animals (for example mice and horses), are also inhibited by SERINC5 and, like HIV, these viruses have evolved factors capable of overcoming its inhibiting activity. SERINC5, therefore, is an endogenous and ancient defense system against retroviruses that retroviral pathogens have been able to bypass. We can use this discovery in two ways. We can either generate a SERINC5-like molecule which acts in a similar way but is not counteracted by Nef, or we can develop a drug to prevent Nef from recognizing SERINC5, therefore unleashing its full antiviral activity. Further research will hold the answer.</p>\r\n', 3.5, 'Rosa A, Chande A, Ziglio S, De Sanctis V, Bertorelli R, Goh S, McCauley S, Nowosielska A, Antonarakis S, Luban J, Santoni F, Pizzato M.<a href="http://www.nature.com/nature/journal/v526/n7572/full/nature15399.html" target="_blank">HIV-1 Nef promotes infection by excluding SERINC5 from virion incorporation.</a> <em>Nature</em>. 2015;526(7572):212-217. doi:10.1038/nature15399.', 1, '2016-05-13 16:08:44', 38, NULL, 2, 'https://doi.org/10.25250/thescbr.brk015'),
(51, 'How early-life adversity gets under the skin', '<p>Many women experience mental health problems during pregnancy and this can have health consequences for the unborn child. Indeed, a wealth of research findings have now shown that women with depression and anxiety in pregnancy are more likely to give birth prematurely, to have a smaller birthweight baby and their child is more likely to experience developmental delays. It is not yet clear exactly why this occurs, but determining the biological mechanisms involved is essential if we want to prevent or reduce the harmful effects being passed from mother to child.</p>\r\n\r\n<p>All individuals have a specific genetic code, or DNA, which remains relatively fixed from conception until death. However additional information in our cells helps control how that genetic code is used and this can change over time. This is called epigenetics and methylation of DNA ("<a data-mce-href="https://en.wikipedia.org/wiki/DNA_methylation" href="https://en.wikipedia.org/wiki/DNA_methylation" target="_blank">DNA methylation</a>") is one of the main epigenetic mechanisms. Epigenetics enables us to adapt and respond to the external environment and is influenced by things like the foods we eat, exercise and stress. Therefore, in addition to our genetic code, our epigenetic patterns also influence our individual traits, behaviours and risk for disease.</p>\r\n\r\n<p>When an expecting mother experiences depression or anxiety during pregnancy, the accompanying chemical and hormone changes alter the environment of the foetus inside the womb. This might influence the infant''s epigenetic patterns and could be an important biological mechanisms explaining the link between poor maternal mental health and infant health and development problems.</p>\r\n\r\n<p>In this study, we determined whether maternal anxiety and depression during pregnancy could impact on the epigenetic patterns in the offspring, focusing on an important gene which helps control foetal growth (insulin-like growth factor 2 (IGF2) and the closely associated gene H19).</p>\r\n\r\n<p>To do so, we recruited a large group of pregnant women when they attended routine hospital appointments. At 28 weeks of pregnancy we used detailed questionnaires to determine whether women had high levels of depression or anxiety. We also obtained information on the women''s health and lifestyle and that of her offspring once born. A blood sample taken from the newborn baby''s umbilical cord was used to measure the levels of DNA methylation of the IGF2/H19 gene in 576 offspring. We investigated a region of the IGF2/H19 gene which has been shown to be particularly important in regulating growth and the levels of DNA methylation in this region have been directly correlated with birth weight previously.</p>\r\n\r\n<p>We then compared the levels of IGF2/H19 DNA methylation between groups of infants, depending on whether or not their mother experienced depression and/or anxiety during pregnancy. Our results showed that mothers who had higher levels of anxiety during pregnancy gave birth to infants with decreased IGF2/H19 methylation levels. Furthermore, when we looked more closely at this association, we found that it was stronger in lower birthweight infants and surprisingly in female babies, with no association observed in male infants.</p>\r\n\r\n<p>Given what is already known about IGF2/H19, decreased DNA methylation levels in the region of the gene investigated, would lead to decreased production of the gene, and this in turn could suppress foetal growth. In the context of our study findings this is important because it provides a possible explanation for the known link between maternal mental health problems and adverse birth outcomes in the child, which could occur through changes in the extent of IGF2/H19 methylation. As of yet we have no clear explanation for why girls may be most affected, but this is the subject of additional investigation. It could relate to differences in foetal growth between the sexes; female infants typically grow more slowly and are smaller at birth on average, than males.</p>\r\n\r\n<p>Further work is also underway in our laboratory to determine whether IGF2/H19 methylation in infants at birth is predictive of later health and development outcomes. And beyond this, the next important question is how can we reverse these effects and ensure that the mental health problems experienced by one generation are not transmitted to the next.</p>\r\n', 3.5, 'Mansell T, Novakovic B, Meyer B, Rzehak P, Vuillermin P, Ponsonby A, Collier F, Burgner D, Saffery R, Ryan J, Vuillermin P, Ponsonby A, Carlin J, Allen K, Tang M, Saffery R, Ranganathan S, Burgner D, Dwyer T, Jachno K, Sly P.<a href="http://www.nature.com/tp/journal/v6/n3/full/tp201632a.html" target="_blank">The effects of maternal anxiety during pregnancy on IGF2/H19 methylation in cord blood.</a> <em>Translational Psychiatry</em>. 2016;6(3):e765. doi:10.1038/tp.2016.32.', 1, '2016-05-19 16:09:55', 39, NULL, 1, 'https://doi.org/10.25250/thescbr.brk016'),
(52, 'A new code for a new life', '<p>Life on our planet is based on DNA. DNA is a big molecule present in all organisms, and it contains the instructions to build each of life''s components. It is the biological way to store and spread information. It is the universal language of life on Earth.</p>\r\n\r\n<p>DNA''s language is composed of four different letters called nucleic acids or bases: adenine (A), thymine (T), cytosine (C) and guanine (G). All together they form a sequence that can be read and translated by living cells into 20 different amino acids (the building blocks of proteins), which, in turn, can be assembled into functional and structural proteins. This is the basis of all the living things we know.</p>\r\n\r\n<p>Since the beginning of life on Earth, DNA is multiplied within cells and transmitted from generation to generation, but the code has always been composed of the four bases A, T, C and G. In the 3-dimensional structure of the DNA molecule, these bases are always associated into "base pairs": an "A" on one strand always pairs with a "T" on the other strand, and a "C" with a "G".</p>\r\n\r\n<p>For a long time, scientists have been trying to extend this alphabet in order to store more information, to create new biological forms, or to understand if other kinds of life that use another DNA alphabet can exist (whether on Earth or elsewhere).</p>\r\n\r\n<p>In 2008, a research team published the discovery and the description of new potential base pairs that had been chemically synthesized. One of these unnatural base pairs (UBPs) was selected for further study: d5SICS and dNaM. This pair was chosen because the researchers observed that d5SICS and dNaM were compatible with the natural mechanisms of DNA replication (a crucial step for the reproduction of living beings) and can also be transformed into an intermediate between DNA and proteins, the RNA.</p>\r\n\r\n<p>But all those reactions were made in the laboratory, in glass tubes. The real challenge would now be to bring these new letters of the DNA alphabet into a living organism, and see if the organism can replicate them.</p>\r\n\r\n<p>The first problem is that these UBPs are not readily available within cells. For this study, Romesberg''s team genetically modified a strain of <em>Escherichia coli</em>, a bacteria commonly found in the human gut, to enable it to transport the new bases from the growth medium into the cell. They did this by engineering these bacteria to have a gene from a single cell algae (<em>Phaedactylum tricornutum</em>). The inserted gene enabled the modified <em>Escherichia coli</em> to transport the new bases from the medium into cells. Using this mutant organism and <a data-mce-href="http://study.com/academy/lesson/how-radioactive-isotopes-track-biological-molecules.html" href="http://study.com/academy/lesson/how-radioactive-isotopes-track-biological-molecules.html">radioactive labelling</a> of the UBPs (a common technique for visualizing molecules), the team observed that the new nucleotides were successfully transported from the growth media into the cells.</p>\r\n\r\n<p>Romesberg''s team then produced a small circular piece of DNA, called a plasmid, and inserted the new base pair in a known position of its sequence. After that, this new plasmid, called pINF, was inserted into the modified <em>Escherichia coli</em> strain engineered with the base-uptake gene. They let the bacteria grow during 15h hours on a special medium enriched with UBPs and then they checked if the bacteria had produced copies of pING. They did this by verifying whether the number of copies of pINF had increased, and the plasmid had been multiplied 20,000,000-fold! Finally, they used a technique called <a data-mce-href="http://www.waters.com/waters/fr_CH/How-Does-High-Performance-Liquid-Chromatography-Work?/nav.htm?cid=10049055&amp;locale=fr_CH" href="http://www.waters.com/waters/fr_CH/How-Does-High-Performance-Liquid-Chromatography-Work?/nav.htm?cid=10049055&amp;locale=fr_CH">liquid chromatography</a> to see if the copies of pINF plasmids contained the UBPs, and they detected a lot of them!</p>\r\n\r\n<p>These results mean that Romesberg''s team successfully multiplied a new type of life''s language in cells, marking the beginning of a new field in biological research.</p>\r\n\r\n<p>This technique will now be developed to tag and follow some DNA replication processes. However, in the future we may be able to create a completely new language, encoding for new amino acids and, who knows, build new proteins. It is a huge challenge, but this could be the starting point of a new life form made by humans. This could allow us to create products/molecules we need, using organisms as direct producers, and help us to imagine some other life systems that could exist somewhere else in our universe.</p>\r\n', 3.5, 'Malyshev D, Dhami K, Lavergne T et al. <a href="http://www.nature.com/nature/journal/v509/n7500/full/nature13314.html" target="_blank">A semi-synthetic organism with an expanded genetic alphabet</a>. <i>Nature</i>. 2014;509(7500):385-388. doi:10.1038/nature13314.', 7, '2016-05-26 16:12:24', 40, NULL, 2, 'https://doi.org/10.25250/thescbr.brk017');
INSERT INTO `breaks` (`id`, `title`, `content`, `reading_time`, `original_article`, `subject_id`, `date_created`, `author1_id`, `author2_id`, `editor_id`, `imzy_link`) VALUES
(56, 'Collateral damage: antibiotics disrupt the balance in the gut', '<p>Bacteria are present everywhere, also on our body surfaces. The intestine provides optimal living conditions to a diverse microbial ecosystem, termed the gut microbiota. In the intestine, the microbes live in very close connection and constant interaction with the host, us. The connection between bacteria and humans has a long evolutionary history and the microbes and their activities have become an integral part of how our bodies work. Microbial products act directly on the cells in the intestine, for example by providing energy to the cells lining the intestine and by stimulating the intestinal immune system. Many microbial products enter the bloodstream and influence different organ systems and tissues. It is becoming increasingly recognised that these microbes are part of us, and have a role in the maintenance of human health.</p>\r\n\r\n<p>As we begin to understand that microbes living in our intestines can affect how our bodies work, it is becoming evident that disturbing the natural balance in their microbial community may have negative consequences on us. Antibiotics taken orally kill not only the pathogens that they are supposed to target, but also innocent bystanders in the intestine. Antibiotic use is known to change the composition of the gut bacterial community, killing susceptible organisms, and thus allowing resistant ones to bloom. Bacterial species differ in their ability to tolerate change, and often it is the potentially pathogenic species that have the ability to opportunistically repopulate a new ecological space when it becomes available; for example when antibiotics have reduced the number of normally dominant species. Antibiotics are among the most common prescription drugs given to children. It has been assumed that gut microbiota recovers within weeks or a few months after an antibiotic treatment. However, studies which used mice as model organism, suggest that early-life antibiotic treatments could have long-term effects on the host, showing in particular that antibiotic treatments in early life alter the microbiota in a way which predisposes the host to be overweight. Indeed, many studies in human children have shown that antibiotic use in early life is associated with increased weight (measured by the <a data-mce-href="https://en.wikipedia.org/wiki/Body_mass_index" href="https://en.wikipedia.org/wiki/Body_mass_index" target="_blank">BMI</a>) later in childhood. These observations led to the question of how the microbiota change in healthy children after antibiotic treatments.</p>\r\n\r\n<p>To answer this question, we analysed the fecal bacterial composition in a group of 142 healthy daycare-attending 2-7 year olds in Finland. The bacterial composition was analysed using <a data-mce-href="https://en.wikipedia.org/wiki/DNA_sequencing" href="https://en.wikipedia.org/wiki/DNA_sequencing" target="_blank">DNA-sequencing</a>: we extracted DNA from the frozen fecal samples, and read the DNA sequence of a gene region that differs between bacterial species. The DNA sequences allowed us to identify which bacteria were in the fecal samples and how abundant they were. We collected antibiotic purchase information for the children from the national drug purchase registry, which allowed us link the composition of the microbiota to the children''s antibiotic use history.</p>\r\n\r\n<p>We found that antibiotic use was the main factor affecting the children''s microbiota composition. Specifically, the time since the child had their last <a data-mce-href="https://en.wikipedia.org/wiki/Macrolide" href="https://en.wikipedia.org/wiki/Macrolide" target="_blank">macrolide</a>-antibiotic treatment was strongly associated with the composition and diversity of the microbiota: macrolide treatments appeared to cause a decline in gut bacterial diversity, and particularly a dramatic reduction in the abundance of one gorup, the bifidobacteria. Bifidobacteria are generally considered beneficial to the health of children and they may even protect against obesity. In general, the microbial changes that we observed after a macrolide treatment have all been previously associated with obesity or related disorders. The full recovery of the children''s gut microbiota was estimated to take up to two years after their last macrolide treatment. The associations between gut microbial diversity and the other common antibiotic types, penicillin or amoxicillin, were somewhat weaker and the recovery of the bacterial diversity slightly faster. All of these antibiotic types are often prescribed to treat the same infections, so the differences were not caused by different types of infections. These results suggest that antibiotics, and macrolides in particular, cause potentially obesity-promoting changes in children''s gut microbiota, which could explain why the use of these antibiotics in childhood is associated with an increased risk of being overweight.</p>\r\n', 3.5, 'Korpela K, Salonen A, Virta L et al. <strong><a href="http://www.nature.com/ncomms/2016/160126/ncomms10410/full/ncomms10410.html" target="_blank">Intestinal microbiome is related to lifetime antibiotic use in Finnish pre-school children</a></strong>. <i>Nature Communications</i>. 2016;7:10410. doi:10.1038/ncomms10410.', 6, '2016-06-02 09:27:31', 42, NULL, 2, 'https://doi.org/10.25250/thescbr.brk018'),
(58, 'The lifetime of memories', '<p><em>"Without memory, there is no culture. Without memory, there would be no civiliz</em><em>a</em><em>tion, no society, no future." </em><a data-mce-href="https://en.wikipedia.org/wiki/Elie_Wiesel" href="https://en.wikipedia.org/wiki/Elie_Wiesel" target="_blank"> Elie Wiesel</a><br><br>\r\n\r\nMemory is essential for animal survival and gives human beings the extraordinary ability to develop consciousness and ultimately the self. The brain mechanisms that allow us to store, elaborate and recall information from the outside world have been continuously enriched and refined throughout animal evolution. From classical greek philosophers to contemporary neuroscientists, memory has attracted attention because of its profound implications in all aspects of our life. How our lives would be if we could remember every little detail of them? Why do we forget people, places, things and events? Even if answering these questions still needs extensive research, scientists all over the world are continuously and restlessly adding pieces to the long standing scientific question of how memory functions and why it fails in pathological conditions.</p>\r\n\r\n<p>Most of our knowledge about human memory arises from clinical studies reporting memory deficits due to brain injuries or developmental and degenerative diseases. A brain surgery aimed to relieve <a data-mce-href="https://en.wikipedia.org/wiki/Henry_Molaison" href="https://en.wikipedia.org/wiki/Henry_Molaison" target="_blank">Henry Molaison</a> (1926-2008) from epilepsy, left him unable to form new memories. Congenital abnormalities in the brain of <a data-mce-href="https://en.wikipedia.org/wiki/Kim_Peek" href="https://en.wikipedia.org/wiki/Kim_Peek" target="_blank">Kim Peek</a> (1951-2009) conferred him an extraordinary memory. These two examples uncover an important fact: brain is able to precisely determine how long particular events of our life will be stored and thus available for recall. But, how does the brain determine the lifetime of memories?</p>\r\n\r\n<p>During the formation of memories that we can consciously recall, a small population of neurons in a brain area called <a data-mce-href="https://en.wikipedia.org/wiki/Dentate_gyrus" href="https://en.wikipedia.org/wiki/Dentate_gyrus" target="_blank">dentate gyrus </a>of the hippocampus becomes active. This population is different for each event of our life and, with time, becomes essential to recall that particular memory: when these neurons are silent we are unable to remember but their activation will prompt the recall of that particular experience. This group of neurons is part of the <a data-mce-href="https://en.wikipedia.org/wiki/Engram_%2528neuropsychology%2529" href="https://en.wikipedia.org/wiki/Engram_%2528neuropsychology%2529" target="_blank">engram</a> (read also the Break: <strong><a data-mce-href="http://thesciencebreaker.com/2015/12/14/break006/" href="http://thesciencebreaker.com/2015/12/14/break006/" target="_blank"><em>Where is the Engram?</em></a></strong>), the physical trace of memory in the brain. Modern techniques allow us to identify and count the neurons involved in the encoding of a particular experience. Moreover, using a technique that combines light and genes, called <a data-mce-href="https://en.wikipedia.org/wiki/Optogenetics" href="https://en.wikipedia.org/wiki/Optogenetics" target="_blank">optogenetics</a>, we can activate neurons with light to interfere or recreate their activity during memory formation and recall. In the first part of our study, we used this fascinating technique to artificially add or subtract neurons to the memory engram. The results of these experiments showed that adding neurons to the engram lengthens memory duration while removing neurons decreases memory persistence.</p>\r\n\r\n<p>What is then the mechanism that the brain uses to determine the number of neurons in the engram and thus to determine the persistence of a particular memory? In our experiments we observed that engram neurons control the activity of a different type of neurons, named Somatostatin (SST) neurons. Unlike engram neurons that excite target cells, SST neurons are inhibitory cells, and thus silence the neurons they contact. Engram neurons use SST neurons to silence the activity of neighboring neurons in the dentate gyrus, preventing their integration in the memory trace and in this way limiting the persistence of memories.</p>\r\n\r\n<p>In our study, we have demonstrated that memories encoded by small number of neurons will be remembered only for a short period of time while a large number of neurons will give raise to memories that will persist longer. SST inhibitory neurons are part of the system that our brain uses to control the size of the engram and thus the lifetime of memories. Defining the mechanisms that limits memory stability bring us closer to understand where are the boundaries of the storage capacity of our brain. Brain pathologies may lead to memory deficits such as in Alzheimer disease but also to "excessive" memories (frequently with associated with bad experiences) such in <a data-mce-href="https://en.wikipedia.org/wiki/Posttraumatic_stress_disorder" href="https://en.wikipedia.org/wiki/Posttraumatic_stress_disorder" target="_blank">Posttraumatic stress disorder</a> (read also the Break: <strong><em><a data-mce-href="http://thesciencebreaker.com/2016/04/08/break011/" href="http://thesciencebreaker.com/2016/04/08/break011/" target="_blank">Attempts to Forget the Past Make It Harder to Remember the Present</a></em></strong>). It would be relevant thus to study whether memory storage mechanisms that we describe in our work are compromised in these mental diseases.</p>\r\n', 3.5, 'Stefanelli T, Bertollini C, Lüscher C, Muller D, Mendez P. <strong><a href="http://www.cell.com/neuron/abstract/S0896-6273(16)00049-0" target="_blank">Hippocampal Somatostatin Interneurons Control the Size of Neuronal Memory Ensembles</a>.</strong> <i>Neuron</i>. 2016;89(5):1074-1085. doi:10.1016/j.neuron.2016.01.024.', 2, '2016-06-22 09:51:14', 44, NULL, 1, 'https://doi.org/10.25250/thescbr.brk020'),
(57, 'The use of neonicotinoid pesticides affects wild bee populations', '<p>Bees are more than honey-makers. They pollinate crops and hence are key elements in our food production. Honeybees, wild bee species such as bumblebees and solitary bees, butterflies, wasps, and flies, all provide an invaluable work of pollination. In fact, a third of the food that we eat depends on them! Nowadays beekeepers around the world continue to observe the mysterious disappearance of bees as well as declines in honeybee colony sizes. The use of insecticides in agriculture could be one of the main factors responsible for it.</p>\r\n\r\n<p><a data-mce-href="https://en.wikipedia.org/wiki/Neonicotinoid" href="https://en.wikipedia.org/wiki/Neonicotinoid" target="_blank">Neonicotinoids</a> are pesticides widely used to coat the seed and make it resistant to attack from insects in the soil. However, these pesticides may represent a consistent risk to bees because, after seed germination, the compound accumulates in the stem, flowers, and leaves of the plant, from where it can be transferred to bees. To protect bee populations, the <a data-mce-href="http://ec.europa.eu/food/animals/live_animals/bees/pesticides/index_en.htm" href="http://ec.europa.eu/food/animals/live_animals/bees/pesticides/index_en.htm" target="_blank">European Union prohibited the use of three neonicotinoids</a> (read also the Break: <b><a href="https://www.thesciencebreaker.com/break.php?breakId=43" target=_blank">A dangerous habit: bees prefer pesticide-contaminated nectar</b></a>). Pesticide-users (<em>e.g.</em>farmers and agricultural stakeholders) criticized the measure, claiming that the effects of the insecticide have mostly been measured on bees artificially fed with it and that, the evidence for neonicotinoids representing a menace for bees in real fields, is weak.</p>\r\n\r\n<p>Thus, the key question is: do neonicotinoids influence honeybees and wild bees in real-world agricultural landscapes?</p>\r\n\r\n<p>A Swedish research team decided to take on the challenge and investigate the effects of neonicotinoids under natural field conditions. The study was done in 16 independent landscapes, surrounding 16 oilseed rape fields. The 16 landscapes were grouped by ecological similarity, resulting in 8 groups of two. For each pair, one of the fields was treated with a fungicide that is harmless to bees (as a <a data-mce-href="https://en.wikipedia.org/wiki/Scientific_control" href="https://en.wikipedia.org/wiki/Scientific_control" target="_blank">control</a>), and the other with the same fungicide plus a product containing neonicotinoids.</p>\r\n\r\n<p>Researchers measured: a) The density of wild bees (amount of wild bees per unit of area), b) the nest building activity of a solitary wild bee species, c) The colony development capacity of the wild but social bumblebee, and finally d) the colony strength (number of adult bees) of the European domesticated honeybee.</p>\r\n\r\n<p>The team found that wild bee density was lower in fields sown with neonicotinoid-coated seeds, demonstrating that the presence of neonicotinoids compounds has a negative effect in the number of these bee species.</p>\r\n\r\n<p>To analyze the nesting capacity of solitary bees, scientists distributed one trap nest with 27 bees in each field and monitored their nest building activity. Solitary bees in normal conditions each build their own nest. Consistently, bees living in insecticide-free fields did so, whereas bees did not build new nests in any of the eight fields treated with the pesticide, thus evidencing again the harmful effect of neonicotinoids.</p>\r\n\r\n<p>The colony development capacity of bumblebees was also negatively affected by neonicotinoids. Bumblebees are social bees and form colonies of one queen and tens or hundreds of workers. During their development, bumblebee colonies grow in weight and worker force to then switch to producing new queens and males. This change in working pattern results in a decline in colony weight. For this study, six commercially reared colonies were deposited in each field. Bumblebee colonies at control fields grew in weight and worker force as it was expected while the rate of weight gain of colonies at fields with insecticide-coated seeds was lower.</p>\r\n\r\n<p>In addition, it is known that bumblebees have an annual life cycle where the new queens born at the end of the season hibernate and form new colonies the following spring. At the end of this experiment, fewer queen and worker/male cocoons were produced at fields treated with neonicotinoids compared to control ones, which will result in less new colonies the following spring.</p>\r\n\r\n<p>The last finding was perhaps the most surprising. Neonicotinoid seed treatment did not affect the colony strength of the domesticated honeybees. These results are in line with a previous study reporting that honeybees seem to be better at detoxifying after neonicotinoid exposure compared to bumblebees. Yet, the possibility of neonicotinoids having negative long-term effects in honeybees was not studied here and hence cannot be excluded.</p>\r\n\r\n<p>In summary, the Swedish researchers succeeded in demonstrating that the use of neonicotinoids as a seed coating treatment in oilseed rape farming represents indeed a risk for wild bees in real world landscapes. Concomitantly, they question the use of domesticated honeybees as representatives for wild bees in pesticide toxicity experiments. And lastly, the authors also underline how studies performed in laboratory conditions could not be precise enough in realistically predicting the effects of pesticides. They propose studies of communities and ecosystems as an alternative to predict the real-world consequences of pesticides.</p>\r\n', 4, 'Rundlöf M, Andersson G, Bommarco R et al. <strong><a href="http://www.nature.com/nature/journal/v521/n7550/full/nature14420.html" target="_blank">Seed coating with a neonicotinoid insecticide negatively affects wild bees</a></strong>. <i>Nature</i>. 2015;521(7550):77-80. doi:10.1038/nature14420.', 3, '2016-06-10 09:33:12', 43, NULL, 2, 'https://doi.org/10.25250/thescbr.brk019'),
(59, 'Distractions and emotions: key factors in vehicle collisions', '<p>In general, fatal crash rates have been declining in the U.S. for several decades.<sup>1</sup> This improvement is due to a variety of factors, including the reliability and overall safety of vehicles and the design and conditions of U.S. roadways. However, the U.S. has not kept pace with other developed countries in terms of overall traffic safety.<sup>2</sup> There are many potential reasons for this discrepancy, including driver behavior and performance.</p>\r\n\r\n<p>Until relatively recently, we have not been able to fully understand the role of driver behavior and performance leading up to a crash. To fill that knowledge gap, the Virginia Tech Transportation Institute (VTTI) pioneered - and continues to evolve - the naturalistic driving study (NDS) method. The NDS method involves instrumenting volunteers'' vehicles with an unobtrusive suite of cameras, sensors, and radars that continuously and automatically collect real-world driving behavior and performance. This lets us observe actual driving during the minutes and seconds leading up to a crash - analyzed frame-by-frame when necessary - from multiple channels of video.<sup>3,4</sup></p>\r\n\r\n<p>However, previous naturalistic driving studies were scoped such that they captured a relatively small number of crashes.<sup>5</sup> With the completion of the Second Strategic Highway Research Program (SHRP 2) NDS, we now have access to the largest light-vehicle, crash-only sample size to date, with more than 1,600 total crashes identified<a data-mce-href="#_ftn6" href="https://thesciencebreaker.wordpress.com/wp-admin/post.php?post=963&amp;action=edit#_ftn6" name="_ftnref6"></a>.<sup>6</sup></p>\r\n\r\n<p>Our goal with the current analysis was to use the SHRP 2 NDS data set to more definitively determine the risk of several key crash causation factors - including impairment, driver error, and distraction - along with their respective prevalence. Overall, we found that driver error, impairment, and distraction were found to contribute greatly to crash risk, with at least one such factor present in nearly 90 percent of the 905 injurious and property-damage (i.e., higher-severity) crashes selected for analysis. We formed a very clear picture that distraction continues to be a great detriment to driver safety, with distracted drivers at double the risk of a crash for more than half of their trips. We saw that some factors absent in previous research findings increased driver risk, including driving under certain observable emotional states, which increased driver risk nearly tenfold. The use of a handheld cell phone (talking on, texting, reaching for) increased risk by nearly four times that of model (alert, attentive, and sober) driving. We also saw that other factors largely reported before to increase driver risk - such as applying makeup or interacting with a child in the rear seat - were relatively benign or even protective in terms of risk.</p>\r\n\r\n<p>These results are sobering. For one, while we as a population <em>know</em> that engaging in distracting activities while driving increases crash risk, this analysis shows we are still <em>choosing</em> to engage in such activities a huge percentage of the time. Such findings are important because we see a younger population of drivers, particularly teens, who are more prone to engaging in distracting activities while driving. If we take no steps in the near future to limit the number of distracting activities in a vehicle, those who represent the next generation of drivers will only continue to be at greater risk of a crash.</p>\r\n\r\n<p>It was also interesting to see in this analysis the role that being overtly emotional can play in driver crash risk. Overall, we hope that these conclusions will better inform policymakers, driver educators, law enforcement agencies, vehicle designers, and the general public about the risks of various sources of impairment, error, and distraction so that appropriate actions can be taken to help reduce their occurrence.</p>\r\n', 3, 'Dingus T, Guo F, Lee S, Antin J, Perez M, Buchanan-King M, Hankey J.&nbsp;<a href="http://www.pnas.org/content/113/10/2636.abstract" target="_blank">Driver crash risk factors and prevalence evaluation using naturalistic driving data.</a> <em>Proceedings of the National Academy of Sciences</em>. 2016;113(10):2636-2641. doi:10.1073/pnas.1513271113.', 1, '2016-06-30 09:57:09', 46, NULL, 1, 'https://doi.org/10.25250/thescbr.brk021'),
(60, 'Ocean acidification and its effects on coral reef growth', '<p>Aptly named "Rainforests of the Sea", tropical coral reefs are one of the most biodiverse ecosystems on the planet. Much like how trees of a forest provide shelter for plants and animals living within, corals reefs provide food and shelter for millions of species ranging from microscopic bacteria to predatory sharks. Not only do coral reefs harbor vast ecological wealth, but they also provide us with ecosystem services valued at over $300 billion each year by supporting tourism, fishing, and protecting our coastlines by absorbing wave energy.</p>\r\n\r\n<p>Unfortunately, coral reef health is in decline worldwide (read also the Break: <b><a href="https://www.thesciencebreaker.com/break.php?breakId=67">The silent battle of young corals against ocean acidification</b></a>). Reefs are currently challenged by a variety of threats ranging from local factors like pollution and overfishing to global-scale issues like climate change and <a data-mce-href="http://www.pmel.noaa.gov/co2/story/What+is+Ocean+Acidification%3F" href="http://www.pmel.noaa.gov/co2/story/What+is+Ocean+Acidification%3F" target="_blank">ocean acidification</a>. Ocean acidification threatens many marine organisms, particularly calcifiers like oysters, crabs, and corals - this is because changing seawater chemistry interferes with calcification, the process by which organisms build protective shells or skeletons. Coral reefs are widely regarded as one of the most vulnerable ecosystems to ocean acidification, in part because the very architecture of the ecosystem relies on calcifying organisms. Through calcification many corals build the spectacular, beautiful, and protective <a data-mce-href="http://oceanservice.noaa.gov/facts/coralmadeof.html" href="http://oceanservice.noaa.gov/facts/coralmadeof.html" target="_blank">calcium carbonate</a> skeletons that form reefs. Without coral calcification, there are no reefs, and without reefs, there is no food or shelter for the millions of associated species.</p>\r\n\r\n<p>Much of the attention surrounding coral reefs and ocean acidification has focused on future implications - what does ocean acidification mean for the future of coral reefs? Will changing seawater chemistry cause reefs to stop growing altogether? These are important questions that are being addressed through ongoing laboratory and field studies. However, the chemistry of the oceans has already changed. Since the pre-industrial revolution (circa 1880), the oceans have absorbed ~25% of the extra carbon dioxide humans injected into the atmosphere, causing the acidity of the oceans to increase by ~30%. The motivating question for our study was to understand whether these changes in seawater chemistry that have already occurred are having measurable impacts on coral reefs - in other words, is ocean acidification <em>already</em> impairing coral reef growth?</p>\r\n\r\n<p>To understand whether ocean acidification is already impairing coral reef growth, we restored the seawater chemistry of a natural coral reef to pre-industrial conditions. After effectively reversing ocean acidification we measured the effects this had on coral reef calcification. While studies have addressed this question in the laboratory using individual species or small groups of organisms, this was the first seawater chemistry manipulation experiment of a coral reef in the natural environment. As you might imagine, manipulating the chemistry of seawater flowing over a coral reef is not an easy task. This was an engineering scale problem that involved altering the chemistry of over 200 tons of seawater while accounting for dilution, hydrodynamics (flow and circulation), and many other factors. Understanding how to do this correctly took years! Using a 15,000 liter reservoir, we added <a data-mce-href="https://en.wikipedia.org/wiki/Base_(chemistry)" href="https://en.wikipedia.org/wiki/Base_%28chemistry%29" target="_blank">sodium hydroxide</a>, which lowers water acidity, to the seawater flowing over a coral reef, to restore seawater chemistry closer to pre-industrial conditions. After allowing this experimental seawater to flow over the reef for approximately one hour, we checked if temporarily reversing ocean acidification increased coral reef calcification. We repeated our experiment fifteen times over the course of several weeks.</p>\r\n\r\n<p>We found that when seawater chemistry was restored closer to pre-industrial conditions, corals grew more quickly. These results provided the first strong evidence that ocean acidification is already slowing coral reef growth. This approach of manipulating seawater chemistry to mitigate ocean acidification has been indicated as a geoengineering technique to save coral reefs from global change. Our results suggest that, when implemented on small scales, this approach might temporarily increase reef calcification, thereby mitigating ocean acidification in the short-term. However, scaling this approach over long periods of time and over the geographic scales necessary is technically infeasible. The Great Barrier Reef, for example, extends over 2,000 km! The only long-term solution to mitigating ocean acidification and securing the health of coral reef ecosystems is to drastically cut carbon emissions. We hope our findings will draw attention to the issue that ocean acidification is not just a concern for the future, but that it''s already impairing valuable marine ecosystems, emphasizing the need to act quickly and responsibly to preserve the future of coral reefs.</p>\r\n', 3.5, 'Albright R, Caldeira L, Hosfelt J et al. <strong><a href="http://www.nature.com/articles/nature17155" target="_blank">Reversal of ocean acidification enhances net coral reef calcification</a>.</strong> <i>Nature</i>. 2016;531(7594):362-365. doi:10.1038/nature17155.', 3, '2016-07-08 12:41:27', 47, NULL, 2, 'https://doi.org/10.25250/thescbr.brk022'),
(61, 'Lab-life: the afternoon siesta of the fruit fly', '<p>"The early bird catches the worm" it''s a simple enough idiom that we''ve all heard. But to a circadian biologist - a scientist studying 24 hour rhythms - that idiom leads to all manner of further questions: how does the bird know when to get up, are there early worms that escape the bird, and why aren''t all birds early? It''s a question we can all relate to: why are some people early risers, others night owls, and teenagers stuck in bed till midday?</p>\r\n\r\n<p>It turns out that our bodies have a "clock" that accurately tells the time, and which organises all sorts of physiological processes to keep them in synchrony with the day/night cycle. Importantly, the body clock can be reset to accommodate changes in seasonal day length or transcontinental travel, although the latter takes several days, during which our body clocks are out of synchrony with our environment and we experience the nauseous symptoms commonly known as jetlag.</p>\r\n\r\n<p>The field of circadian biology took off in the late 1960s when Ron Konopka and Seymour Benzer studied the body clock of the fruitfly <em>Drosophila melanogaster</em>, a species beloved by lab-based biologists for its well-understood genetics, ease of culture, and diverse range of behaviours<sup>1</sup>. Like humans, animals do the same thing at more or less the same time of day (or night in the case of nocturnal animals), and this holds true even for the tiny fruitfly. In a simple laboratory setup of constant temperature interspersed with regular periods of 12 hours of light and dark, fruitflies are active in the morning and evening, but have a rest or "siesta" in the middle of the day.</p>\r\n\r\n<p>This rather artificial laboratory environment has underpinned forty years of experiments that have identified the genes that act as the mechanical gears of the clock, as well as the neurons dedicated to controlling these behavioural rhythms. Perhaps unsurprisingly it was shown that natural selection favours animals or bacteria (which surprisingly also have circadian clocks) with functioning clocks over animals with non-functioning clocks.</p>\r\n\r\n<p>So it was a something of a surprise when Kyriacou and colleagues<sup>2 </sup>reported that when recording the activity of flies in the wild during an Italian summer, fruitflies are actually most active during the period in which in the laboratory they would normally be having a siesta, calling into question much of the previous research. Interestingly, around the same time results from field experiments using mice showed that in the wild mice were diurnal, not nocturnal as they are under artificial laboratory regimes<sup>3</sup>, again contradictory to years of laboratory work.</p>\r\n\r\n<p>We decided to address these apparently conflicting results by attempting to simulate the natural environment in the laboratory much more accurately than had previously been attempted. This involved the use of sophisticated temperature controlled incubators and custom built arrays of LEDs that could accurately match the slow changes in light intensity, colour and temperature of an Italian summer''s day. Furthermore we took advantage of technological advances in video tracking to record the behaviour of the flies moving around in mixed-sex groups within large 15 cm dishes, allowing much freer movement more similar to the flies'' natural habitat where they swarm over rotting fruit.</p>\r\n\r\n<p>The results<sup>4</sup> unequivocally confirmed that under these "natural-like" conditions, flies were indeed active during the warm summer afternoons. But as so often in science, what appeared to be a spanner in the works proved to be the tool to solve the problem: using our new incubator setup we screened a range of mutants to find out which genes were required for this afternoon activity. It turned out that flies lacking the <a data-mce-href="https://en.wikipedia.org/wiki/Nociceptor" href="https://en.wikipedia.org/wiki/Nociceptor" target="_blank">nociceptor</a> gene <em>TrpA1 </em>never showed an afternoon peak of activity. Nociceptors such as TrpA1 are the body''s pain receptors responding to intense light or heat, suggesting that the 35 °C peak afternoon temperature was sufficiently stressful to override the flies'' body clock preference for a siesta.</p>\r\n\r\n<p>So, if you''re a scientist interested in early birds, experiments that look like a can of worms can turn out to be the tasty result you''re looking for!</p>\r\n', 3.5, 'Green E, O’Callaghan E, Hansen C et al. <strong><a href="http://www.pnas.org/content/112/28/8702.long" target="_blank">Drosophila circadian rhythms in seminatural environments: Summer afternoon component is not an artifact and requires TrpA1 channels.</a></strong> <i>Proceedings of the National Academy of Sciences</i>. 2015;112(28):8702-8707. doi:10.1073/pnas.1506093112.', 4, '2016-07-13 12:49:48', 48, NULL, 2, 'https://doi.org/10.25250/thescbr.brk023'),
(62, 'Cold adaptation: gut bacteria can make the difference', '<p>If someone told you that in our body we harbour billions of bacteria, surely you would feel mocked, but it''s true! There is evidence showing that microbes colonize all the part of our body that are exposed to the external environment (like mouth and skin), with the greater portion of them residing in the gut. All the microorganisms, which are resident in our intestinal tract, are defined by scientists as gut microbiota. Importantly, gut microbiota is not a fixed group of microorganisms, but it may vary in its composition according to several factors <em>e.g. </em>food, age, health, antibiotic use (read also the Break: <strong><a data-mce-href="http://thesciencebreaker.com/2016/06/02/break018/" href="http://thesciencebreaker.com/2016/06/02/break018/" target="_blank">Collateral damage: antibiotics disrupt the balance in the gut</a></strong>).</p>\r\n\r\n<p>In the past twenty years, interest has increased in science for the study of the interplay between the gut microbiota and our bodies. It is now clear that microbiota, also called "the forgotten organ", plays a major role both in physiological and pathological conditions. Gut microbiota has a variety of different functions, among which harvesting energy and providing information for the correct maturation of the <a data-mce-href="https://en.wikipedia.org/wiki/Immune_system" href="https://en.wikipedia.org/wiki/Immune_system" target="_blank">immune system</a>. Imbalance in the microbiota community, called <a data-mce-href="https://en.wikipedia.org/wiki/Dysbiosis" href="https://en.wikipedia.org/wiki/Dysbiosis" target="_blank">dysbiosis</a>, has been correlated with the development of different diseases like autoimmune and allergic diseases, obesity and diabetes<sup>1,2</sup>. However, the biological mechanisms and the factors that guide the effects of microbiota on the human metabolism are still poorly understood.</p>\r\n\r\n<p>Recently, researchers from Dr. Trajkovski''s lab described how cold exposure leads to a specific change in the composition of gut microbiota that favours energy absorption. After exposing lab-mice to cold for 10 days, they detected an increase in energy consumption and a decrease in body weight. Unexpectedly, despite the stable energy expenditure and food intake, the loss of fat decreased over time, leading the researchers to investigate the effects of prolonged cold exposure. In particular, focusing on the gut microbiota composition, they were able to observe a shift from "warm microbiota" to "cold microbiota" after 30 days of cold exposure. Interestingly, the transplantation of the "cold microbiota" into mice born without any microorganisms living in or on them (called germ-free mice), was sufficient to induce the metabolic changes needed to make these mice more resistant to cold. The researchers have shown that the above mentioned effects were promoted by the browning of the white fat depots (read also the Break: <strong><a data-mce-href="http://thesciencebreaker.com/2016/04/01/break010/" href="http://thesciencebreaker.com/2016/04/01/break010/" target="_blank">The colour beige: heating up the fat</a></strong>), in part mediated by the shift in the microbiota composition.</p>\r\n\r\n<p>Furthermore, they investigated in detail the effect of cold exposure on the intestine. After prolonged cold exposure, the researchers observed a dramatic increase in intestinal surface area, which improves the mice''s ability to get energy from food. This improvement in energy absorption explains the stabilization of the mice''s body weight despite the increased energy consumption when exposed to cold. Remarkably, they also observed that one specific species of bacteria, <em>Akkermansia muciniphila</em>, is almost completely absent in the cold microbiota, and its transplantation into mice leads to a full prevention of their ability to enhance energy absorption and increase <a data-mce-href="http://www.chp.edu/our-services/transplant/intestine/education/about-small-large-intestines" href="http://www.chp.edu/our-services/transplant/intestine/education/about-small-large-intestines" target="_blank">intestinal surface</a>.</p>\r\n\r\n<p>It is becoming clear that the relationship between host and gut microbiota plays a pivotal role in the regulation of both physiological and pathological conditions. In their publication, researchers propose that the co-evolution between host and microbiome may have had a role during periods of increased energy demand, like winter, where changes in the gut microbiota composition were needed to increase the calorie absorption from food. The group of Trajkovski was able to show that the above mentioned increment in calorie absorption is in part due to the expansion of the intestine. Furthermore, one strain from the gut microbiota, <em>Akkermansia muciniphila,</em> was reported to revert some of the benefits of cold exposure, representing the first example of symbiosis able to affect body''s energy utilization.</p>\r\n', 3, 'Chevalier C, Stojanović O, Colin D et al. <strong><a href="http://www.sciencedirect.com/science/article/pii/S0092867415014841" target="_blank">Gut Microbiota Orchestrates Energy Homeostasis during Cold</a></strong>. Cell. 2015;163(6):1360-1374. doi:10.1016/j.cell.2015.11.004', 6, '2016-09-07 12:55:33', 49, NULL, 2, 'https://doi.org/10.25250/thescbr.brk024'),
(63, 'Resetting nature’s clock: shifting seasons and species relationships', '<p>Every year, many of us gaze in fascination at the movement of nature''s clock, looking forward to seeing wild plants coming into bloom or the arrival of the first migrant birds. Less obvious, but just as dramatic, <a data-mce-href="https://en.wikipedia.org/wiki/Phytoplankton" href="https://en.wikipedia.org/wiki/Phytoplankton" target="_blank">populations of phytoplankton</a> bloom below the surface of lakes and oceans, while fish migrate to their spawning grounds. Over time, the dedicated observer may even come to anticipate the dates that their favourite natural events will occur on. However, the timing of these seasonal events is changing. Many familiar "signs of spring" are occurring earlier in the year than they used to. Often these changes have come about because seasonal biological events are sensitive to climatic conditions. For this very reason, they continue to be used by the <a data-mce-href="http://www.ipcc.ch/" href="http://www.ipcc.ch/" target="_blank">Intergovernmental Panel on Climate Change (IPCC)</a> as tangible indicators of global warming.</p>\r\n\r\n<p>Shifts in the biological seasons are likely to have significant effects upon ecosystems, and the goods and services that they provide to society, now and in the future. Importantly, species have been changing their "seasonal" behaviour to different extents over time, and this could disrupt the seasonal relationships between plants and animals. To understand what the consequences of these changes are likely to be, it would be useful to know which species groups are most, and least, affected by climate change. This would allow us to identify species that are particularly effective climate change indicators, and to make some tentative projections of what the future might have in store.</p>\r\n\r\n<p>In a recent study we addressed this important knowledge gap. We combined a wealth of biological records from the UK, for as many species as possible, and information on the climatic conditions that might be affecting their seasonal activities. These incredible datasets were gathered by <a data-mce-href="https://en.wikipedia.org/wiki/Citizen_science" href="https://en.wikipedia.org/wiki/Citizen_science" target="_blank">voluntary citizen scientists</a> and also by professional scientists based at research institutes and universities. Altogether, we gathered more than 370.000 records for over 800 aquatic and dry-land plant and animal species, collected over many years.</p>\r\n\r\n<p>We then related the seasonal behaviour of plants and animals to local temperature and rainfall. Furthermore, we developed a new statistical method to evaluate how much and in which cases a given species may be influenced by the annual temperature or rainfall cycle. Importantly, we examined and compared how much the seasonal activities of different species groups, such as plants and herbivores, predators and prey, change with temperature and precipitation.</p>\r\n\r\n<p>Using our novel approach, we were able to show that the seasonal activities of UK plant and animal species are less sensitive to rainfall than to temperature. Generally, many natural seasonal events occur earlier under warmer conditions. It was also clear that species are usually influenced by temperature changes only in certain periods of the year. A clear pattern was that this temperature sensitivity varied among species at different levels in the food web.</p>\r\n\r\n<p>In simple terms, for any given amount of temperature change, predators and prey are likely to change their seasonal "behaviour" by different levels. In particular, we found that predatory species were not as much influenced by climate change as prey species. This was the first time that such a pattern had been observed among so many species and habitats and at a national scale. Changes in the relative seasonal timings of predators and their prey may alter food webs, with implications for UK biodiversity and ecosystem health. If, as predicted, climate warming continues into the future, then the disparity in responses of predators and prey is also likely to increase. Interactions between species are complex, with changes in one species likely to have knock-on effects for others, which may result in effects that cascade throughout ecosystems.</p>\r\n\r\n<p>We must now make every effort to understand the extent to which these effects may be realised, and the extent to which ecosystems might be resilient to change. To do this, we must recognise the immense value of long-term ecological research, and continue to invest in this activity. Crucially, environmental change often acts through complex networks of species interactions and relationships, and so we must also take care to monitor at the whole-ecosystem scale. Without this long-term view, we will be unable to detect emerging threats or to learn from our successes and failures in local environmental management and mitigation.</p>\r\n', 3.5, 'Thackeray, S., Henrys, P., Hemming, D., Bell, J., Botham, M., Burthe, S., Helaouet, P., Johns, D., Jones, I., Leech, D., Mackay, E., Massimino, D., Atkinson, S., Bacon, P., Brereton, T., Carvalho, L., Clutton-Brock, T., Duck, C., Edwards, M., Elliott, J., Hall, S., Harrington, R., Pearce-Higgins, J., Hye, T., Kruuk, L., Pemberton, J., Sparks, T., Thompson, P., White, I., Winfield, I. and Wanless, S. (2016).<a href="http://www.nature.com/nature/journal/v535/n7611/full/nature18608.html" target="_blank">Phenological sensitivity to climate across taxa and trophic levels.</a> <em>Nature</em>, 535(7611), pp.241-245.', 3, '2016-09-14 13:01:41', 50, NULL, 1, 'https://doi.org/10.25250/thescbr.brk025'),
(107, 'Hacking the tryptophan metabolic process to reduce neurodegeneration', '<p>Oats, dried prunes, tuna fish, milk, chicken, bread, peanuts, and chocolate are fabulous foods that enrich our everyday meals. But apart from their culinary properties, they are also great sources of tryptophan. <a href="https://en.wikipedia.org/wiki/Tryptophan" target="_blank">Tryptophan</a> is an amino acid which is used by cells either as a building block for the synthesis of proteins or as a precursor in several metabolic pathways. Serotonin - our natural mood stabiliser - is an example of a product generated from tryptophan absorbed in our diet. However, the great majority of ingested tryptophan is used by a metabolic process called the kynurenine pathway. The final products of this process provide an ultra violet (UV) filter in our eyes which protects the retina from UV damage and are also used by cells in a diversity of other fundamental reactions. More interestingly, several intermediate products (metabolites) of the kynurenine pathway cause either damaging or protective effects to nerve cells (neurons). Indeed, the pathway can be divided into a "bad" or toxic section in which the metabolites 3-hydoxykynurenine and quinolinic acid are produced and a "good" or protective section which produces kynurenic acid (KYNA).</p>\r\n\r\n<p>Neurodegenerative disorders is an umbrella term used to identify several human pathologies characterised by the progressive degeneration of neurons which ultimately culminate with their death in selective regions of the brain. Some examples of these pathologies are <a href="https://simple.wikipedia.org/wiki/Alzheimer%27s_disease" target="_blank">Alzheimer''s</a>, <a href="https://simple.wikipedia.org/wiki/Parkinson%27s_disease" target="_blank">Parkinson''s</a> and <a href="https://simple.wikipedia.org/wiki/Huntington%27s_disease" target="_blank">Huntington''s</a> diseases - for which there are no cures. Although disease-specific clinical traits are shown by patients, all of these disorders are characterised by aberrant changes in levels of kynurenine metabolites. Indeed, while in normal health a fine balance between the "bad" and "good" metabolites is maintained, a shift towards the production of "bad" metabolites is observed in the brain of patients. Thus, exciting questions can be raised: Can we intervene to decrease the production of "bad" metabolites and increase levels of the "good" metabolite? Would this intervention be a valuable therapeutic avenue for treating these devastating diseases?</p>\r\n\r\n<p>In order to address this question, we needed a simple yet reliable model to study. We found this in the common fruit fly <em><a href="https://simple.wikipedia.org/wiki/Drosophila" target="_blank">Drosophila melanogaster</a></em>, a robust system used for many decades to study several cellular processes relevant to human disease. Interestingly, fruit flies do not develop neurodegenerative diseases, but these can often be mimicked by inserting the defective human genes into flies and using appropriate genetic and molecular tools.</p>\r\n\r\n<p>We focused on how diet can improve the condition of flies displaying symptoms of Huntington''s disease. We found that an enriched tryptophan diet protects these flies from neuron loss by shifting the kynurenine metabolism towards production of the "good" metabolite KYNA. Similarly, by raising the dosage of the enzyme responsible for producing KYNA, an improvement in Huntington''s disease-related defects was observed. Thus, the "good" metabolite can counteract the effects of the "bad" metabolites, prevent cells from dying, and thereby improve health in the flies.</p>\r\n\r\n<p>Are there any other ways to raise levels of the protective metabolite in the flies? We demonstrated that "switching off" two critical enzymes - called TDO and KMO - of this pathway led to increased levels of the protective metabolite. This change causes reversion of several disease-related defects not only in Huntington''s disease model flies, but also in fly models of Alzheimer''s and Parkinson''s diseases. Furthermore, drug interventions which arrest TDO and KMO activities reduce neuronal death and other symptoms displayed by these neurodegenerative fly models.</p>\r\n\r\n<p>As the current worldwide population lives longer, the number of people suffering from these neurodegenerative disorders is sadly increasing. These exciting results demonstrate that modulation of the kynurenine pathway constitutes a promising therapeutic avenue for design and development of drugs which could delay the onset and reduce the symptoms in a myriad of neurodegenerative disorders.</p>', 3, 'Breda, C., Sathyasaikumar, K., Sograte Idrissi, S., Notarangelo, F., Estranero, J., Moore, G., Green, E., Kyriacou, C., Schwarcz, R. and Giorgini, F. (2016). <a href="http://www.pnas.org/content/113/19/5435.long" target="_blank">Tryptophan-2,3-dioxygenase (TDO) inhibition ameliorates neurodegeneration by modulation of kynurenine pathway metabolites</a>. <em>Proceedings of the National Academy of Sciences</em>, 113(19), pp.5435-5440.', 1, '2017-04-25 14:49:02', NULL, NULL, 2, 'https://doi.org/10.25250/thescbr.brk045');
INSERT INTO `breaks` (`id`, `title`, `content`, `reading_time`, `original_article`, `subject_id`, `date_created`, `author1_id`, `author2_id`, `editor_id`, `imzy_link`) VALUES
(64, 'Nitrogen pollution from lowlands reaches distant mountain lakes', '<p><a data-mce-href="https://en.wikipedia.org/wiki/Nitrogen" href="https://en.wikipedia.org/wiki/Nitrogen" target="_blank">Nitrogen</a> is required by all living things, but too much of it can be a problem for aquatic ecosystems. Excess reactive nitrogen (nitrate and ammonium, forms that can be used directly by plants and algae) can cause lake<a data-mce-href="https://en.wikipedia.org/wiki/Eutrophication" href="https://en.wikipedia.org/wiki/Eutrophication" target="_blank">eutrophication</a> which can include harmful <a data-mce-href="https://en.wikipedia.org/wiki/Algal_bloom" href="https://en.wikipedia.org/wiki/Algal_bloom" target="_blank">algal blooms</a>, declining water quality, loss of biodiversity and other changes in lake ecosystems.</p>\r\n\r\n<p>Humans have more than doubled the amount of reactive nitrogen in the natural system.<sup>1</sup> The main <a data-mce-href="https://en.wikipedia.org/wiki/Nitrogen_cycle#Human_influences_on_the_nitrogen_cycle" href="https://en.wikipedia.org/wiki/Nitrogen_cycle#Human_influences_on_the_nitrogen_cycle" target="_blank">sources of reactive nitrogen</a> are agricultural and industrial runoff, sewage, fossil fuels, and legume crop production. These sources are found in populated areas, but evidence is mounting that reactive nitrogen pollution is more widespread. For example, several researchers have suggested that the amount of reactive nitrogen is increasing in remote mountains of the Western United States and affecting the ecology of lakes within<sup>2,3,4</sup>. However, researchers had not yet identified the sources of reactive nitrogen to these remote areas - that information is necessary to create a game plan for nutrient management and mitigation.</p>\r\n\r\n<p>We set out to determine the sources of nitrate to high elevation lakes for the very first time. We focused our research on lakes in the Uinta Mountains in Utah (U.S.A), which are 150 km from the nearest metropolitan area (Salt Lake City). Human activity around the study lakes is limited to recreation and, in years past, free range sheep and cattle grazing.</p>\r\n\r\n<p>We <a data-mce-href="https://en.wikipedia.org/wiki/Isotope_analysis" href="https://en.wikipedia.org/wiki/Isotope_analysis" target="_blank">analyzed stable isotopes</a> (oxygen and nitrogen) of nitrates within lake water, stream water, and snow, and in particular, the oxygen-17 contents, which are typically not measured in similar studies. Stable isotopes are atoms of the same element with different masses, making them behave slightly differently in nature. By looking at all three stable isotopes of oxygen, as well as the two stable isotopes of nitrogen, we obtained signatures (like fingerprints) of nitrate that help us differentiate between various nitrate sources, including fertilizer, manure, soil, and atmosphere.</p>\r\n\r\n<p>Using this fingerprinting technique, we determined that 60% of lake and inflow nitrates in Uinta Mountain watersheds originate from fertilizers. These fertilizers are transported through the air and may come from as far away as the <a data-mce-href="https://en.wikipedia.org/wiki/Wasatch_Front" href="https://en.wikipedia.org/wiki/Wasatch_Front" target="_blank">Wasatch Front</a> and California. Another 10% of nitrates in the lakes and streams are derived from fossil fuels. That brings the total nitrate originating from human activities to 70% with fertilizers having, by far, the greatest impact. That''s a surprisingly large contribution of human-derived nitrates for lakes that have little direct human influence. To add insult to injury, remote mountain lakes tend to be extra sensitive to changes in nutrients.</p>\r\n\r\n<p>So, is atmospheric nitrogen pollution limited to the Uinta Mountains? As you may suspect, the effects of atmospheric pollution are likely to be more widespread. Our comparisons to available more conventional stable isotope data for nitrates in Rocky Mountain lakes in the Western United States<sup>5</sup> suggest that other mountain lakes are also receiving human-derived sources of reactive nitrogen.</p>\r\n\r\n<p>This study adds to evidence that human activities have a strong influence on remote regions, and calls into question our current practices. Identifying atmospherically delivered fertilizers as the largest contributor to nitrate in these remote lakes is important for developing effective mitigation strategies, particularly given the anticipated expansion of human populations and fertilizer use.</p>\r\n', 3, 'Hundey E, Russell S, Longstaffe F, Moser K. <strong><a href="http://www.nature.com/articles/ncomms10571" target="_blank">Agriculture causes nitrate fertilization of remote alpine lakes.</a></strong> <i>Nature Communications</i>. 2016;7:10571. doi:10.1038/ncomms10571.', 3, '2016-09-21 13:10:08', 52, NULL, 1, 'https://doi.org/10.25250/thescbr.brk026'),
(65, 'Living without mitochondria: the downfall of one textbook truth', '<p>It was the greatest leap in evolution since the emergence of life on Earth. So-called eukaryotic cells, the building blocks of all multicellular organisms like you and me, animals, plants, fungi, and also a whole zoo of single-celled <a data-mce-href="https://en.wikipedia.org/wiki/Protist" href="https://en.wikipedia.org/wiki/Protist" target="_blank">protists</a>, evolved from a common ancestor more than a billion years ago. This ancestor resembled current-day prokaryotes, i.e. bacteria and archaea. These organisms populated all imaginable habitats and developed a plethora of wildly diverse means of obtaining their energy. However, the simple structure of their cells limited their size and complexity of interactions. With time, our ancestor developed a series of innovations that are nowadays defining characteristics of the eukaryotic cell. How this process went through, what was its driving force, and in what order those particular steps took place still remains a question provoking lively discussions among scientists. However, regardless of the details, among the most crucial steps in forming a eukaryote was undoubtedly the acquisition of a mitochondrion.</p>\r\n\r\n<p>Today, mitochondria are small <a data-mce-href="https://en.wikipedia.org/wiki/Organelle" href="https://en.wikipedia.org/wiki/Organelle" target="_blank">organelles</a> within our cells that form complex and ever-changing networks. The sausage-shaped mitochondria constantly split, merge, and move around in order to provide all cellular regions with a life-giving chemical energy. This energy, produced on the inner surface of mitochondria, is produced by reactions between the nutrients we get from food and the oxygen we breathe in. Cells can produce energy also without the help of mitochondria and many organisms which live in oxygen-poor environment indeed do so, but the mitochondrial energy production is more than ten times more efficient. Mitochondria have also many other roles including production of various compounds, which will be important later in our story.</p>\r\n\r\n<p>Once an independent bacterium, the ancestor of mitochondria somehow found itself inside another cell, survived against all odds, and developed a relationship of interdependence with its new host. Importantly, this interaction added a new trick to the repertoire of our ancestors: the above-mentioned unprecedentedly efficient way of producing energy which enabled an explosion of new, more complex and diverse life forms. To better understand this evolutionary process, scientists were searching for a missing link: a eukaryote lacking mitochondria. The search came up empty. All hot candidates, mostly parasitic protists, were shown to possess a simplified mitochondrion. By studying their mitochondria, researchers found out that no matter how small and simplified the organelle is, it always retains one function: the synthesis of <a data-mce-href="https://en.wikipedia.org/wiki/Iron%D0sulfur_protein" href="https://en.wikipedia.org/wiki/Iron%D0sulfur_protein" target="_blank">iron-sulfur clusters</a>. These inorganic compounds are essential components of many vital proteins and no cell can live without them. More and more examples accumulated over time supporting two textbook truths: first, there are no ancient amitochondriate eukaryotes living today and second, eukaryotes can never completely lose mitochondria, particularly because they need the iron-sulfur clusters that mitochondria produce.</p>\r\n\r\n<p>This is where our work comes in. Among all those false alarms of seemingly amitochondriate protists, one candidate group remained unexplored: Oxymonads. These anaerobic protists are largely neglected as they cause no health issues and have no economic potential whatsoever. Most oxymonads live in the gut of termites, while others, like our research subject <em>Monocercomonoides</em>, inhabit a wider range of hosts - we isolated our specimen from feces of a <a data-mce-href="https://en.wikipedia.org/wiki/Chinchilla" href="https://en.wikipedia.org/wiki/Chinchilla" target="_blank">chinchilla</a>. To date, nothing in the oxymonad cell has been found that resembles a mitochondrion although it is clear that their ancestors must have had one because it is present in their closest free-living <a data-mce-href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2148110/" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2148110/" target="_blank">relatives</a>. We decided to investigate this case further. By using various experimental methods, we searched for traces of mitochondria and failed. Not only the physical organelle seemed to be missing but also all the proteins that are usually associated with it.</p>\r\n\r\n<p>To be absolutely sure, we sequenced all the genetic material of <em>Monocercomonoides</em> in a futile search for a trace of mitochondria. We found everything we would expect from such an organism, but all the mitochondrion-associated genes were missing, including the genes for producing the iron-sulfur clusters. Could <em>Monocercomonoides</em> survive without these molecules? Not so fast! This creature had a surprise for us: a completely different kind of iron-sulfur cluster production system, apparently recently borrowed from bacteria by a process called <a data-mce-href="https://en.wikipedia.org/wiki/Horizontal_gene_transfer" href="https://en.wikipedia.org/wiki/Horizontal_gene_transfer" target="_blank">horizontal gene transfer</a>. So <em>Monocercomonoides</em> freed themselves from their dependence on mitochondria by finding a new way to produce the essential iron sulfur clusters on their own. This is the first report of a eukaryote that completely lost its mitochondria.</p>\r\n', 3.5, 'Karnkowska, A., Vacek, V., Zubčov, Z., Treitli, S., Petrželkov, R., Eme, L., Novák, L., Žrsk, V., Barlow, L., Herman, E., Soukal, P., Hroudov, M., Doležal, P., Stairs, C., Roger, A., Eli, M., Dacks, J., Vlček, Č. and Hampl, V. (2016). <a href="http://www.cell.com/current-biology/abstract/S0960-9822(16)30263-9" target="_blank">A Eukaryote without a Mitochondrial Organelle</a>. <em>Current Biology</em>, 26(10), pp.1274-1284.', 4, '2016-10-03 13:18:21', 54, NULL, 2, 'https://doi.org/10.25250/thescbr.brk027'),
(66, 'Invisible allies for healthy juvenile growth', '<p>Maybe you remember it from your childhood. There was a doorframe in your parents'' house, marked by a ladder of small horizontal lines, with dates and your name written next to each line: the more recent the date, the higher the position of the line. This simple growth chart recorded your height at a given age, and how much you grew from the last time. And probably you took it completely for granted that as the time passed, each line was drawn above the last, never wondering how this all works.</p>\r\n\r\n<p>Nowadays we know that the gain in body size during the infant growth period is a result of the interactions between nutrition and the organism''s hormonal cues. In mammals, post-natal growth is controlled by the <a data-mce-href="https://simple.wikipedia.org/wiki/Growth_hormone" href="https://simple.wikipedia.org/wiki/Growth_hormone" target="_blank">Growth Hormone</a>, which instructs the tissues to produce <a data-mce-href="https://en.wikipedia.org/wiki/Insulin-like_growth_factor_1" href="https://en.wikipedia.org/wiki/Insulin-like_growth_factor_1" target="_blank">Insulin-like Growth Factor-1 (IGF-1)</a>, which then promotes organ and systemic growth. When food is scarce for a long time or the diet doesn''t contain enough nutrients (situations defined as chronic undernutrition), juveniles stop growing and in the long term become small and thin (1). If we return to our doorframe growth chart parallel, it would mean that there would be more dates at the same line. In certain parts of the world, such as Africa and India, <a data-mce-href="http://www.who.int/maternal_child_adolescent/topics/child/malnutrition/en/" href="http://www.who.int/maternal_child_adolescent/topics/child/malnutrition/en/" target="_blank">childhood undernutrition</a> is a major health threat, especially for the long term consequences which besides stunting include also neurocognitive deficits. However, it seems that it is not only the amount and quality of food that matters for the growth. Recently, a consistent body of research (2) suggested that the bacteria in our intestines, the so called gut microbiota (read also the Break: <strong><a data-mce-href="http://thesciencebreaker.com/2016/09/07/break024/" href="http://thesciencebreaker.com/2016/09/07/break024/" target="_blank">Cold adaptation: gut bacteria can make the difference</a></strong>), might play an important role. Therefore, we wanted to find answer to the question: What is the contribution of this gut microbiota to postnatal growth during feeding on rich diet and during chronic undernutrition?</p>\r\n\r\n<p>To explore this we studied two groups of mice: conventional mice harboring a normal intestinal microbiota and germ-free mice, which are devoid of any detectable living bacteria. Juvenile male mice received rich diet, which had all the necessary nutrients in sufficient amounts, and we followed their growth until young adulthood. Conventional mice showed better growth rate compared to their germ-free counterparts and such difference was not caused by bigger fat stores but, instead, by overall organ and bone growth. In accordance with this physical difference, microbiota also increased the titers of growth promoting IGF-1 in the sera of conventional mice.</p>\r\n\r\n<p>Having established the crucial role of microbiota for optimal growth in nutrient rich conditions, we wanted to know if it plays the same role in chronic undernutrition. To test this, we designed a diet, which was low in proteins and fats, but provided the mice with the same amount of calories thanks to addition of carbohydrates. When the juvenile male mice received this depleted diet, we observed that the germ-free mice failed to grow - they were completely stunted. On the other hand, their conventional counterparts harboring the bacteria continued to grow, although the growth rate was slower than that of the conventional mice fed the nutritionally rich diet.</p>\r\n\r\n<p>Concerning the host growth and bacteria colonization, our lab has previously shown that certain specific bacteria from the species <a data-mce-href="https://en.wikipedia.org/wiki/Lactobacillus_plantarum" href="https://en.wikipedia.org/wiki/Lactobacillus_plantarum" target="_blank"><em>Lactobacillus (L.) plantarum</em> </a>are able to promote growth of the <em>Drosophila</em> larvae upon feeding on diet low in proteins (3). <em>L. plantarum</em> belongs to the lactic acid-producing bacteria and these are nomadic bacteria that can be found in different habitats, such as soil and fermented foods, but also in the intestinal tracts of both invertebrates and vertebrates, including mice and humans. We were therefore wondering: Will the single bacterium which promotes growth in <a data-mce-href="https://en.wikipedia.org/wiki/Drosophila_melanogaster" href="https://en.wikipedia.org/wiki/Drosophila_melanogaster" target="_blank"><em>Drosophila</em></a> model, be able to promote the growth of the stunted germ-free mice? And to what extent? To this end we provided the germ-free mice with the growth-promoting strain <em>L. plantarum</em><sup>WJL</sup> and submitted them to the depleted diet. We observed that the mono-bacterium-associated mice grew as well as the conventional mice on the depleted diet and that this one <em>L. plantarum</em> strain increased the levels of growth promoting IGF-1 in sera to the same extent as we observed in the conventional mice.</p>\r\n\r\n<p>Thus our work established that microbiota is necessary and that specific <em>L. plantarum</em> bacterial strain is sufficient to boost juvenile growth rate. Based on these findings we envision that, together with supportive nutrition strategies, microbial interventions using specific bacterial strains may represent a novel and useful strategy to limit the adverse effects of chronic undernutrition on post-natal growth - a threat that still affects more than 160 million of children below 5-years of age in low- and middle-income countries.</p>\r\n', 4, 'Schwarzer M, Makki K, Storelli G et al. <strong><a href="http://science.sciencemag.org/content/351/6275/854.full" target="_blank">Lactobacillus plantarum strain maintains growth of infant mice during chronic undernutrition</a></strong>. <i>Science</i>. 2016;351(6275):854-857. doi:10.1126/science.aad8588.', 6, '2016-10-12 13:25:03', 55, NULL, 1, 'https://doi.org/10.25250/thescbr.brk028'),
(67, 'The silent battle of young corals against ocean acidification', '<p><a data-mce-href="https://en.wikipedia.org/wiki/Coral_reef" href="https://en.wikipedia.org/wiki/Coral_reef" target="_blank">Coral reefs</a> are ecosystems of extraordinary diversity. Considered "the rainforests of the sea", they contain ~35% of described marine species despite only occupying 0.2% of the world''s ocean. Although they are extremely important habitat providers and form large living structures (some reefs can be seen from space!) the coral animals themselves are small and very sensitive to changes in their environmental conditions. A stressed coral can indicate changes in the physical and chemical properties of the seawater and pressures on the whole ecosystem. Coral reefs are currently faced with an unprecedented mix of human-induced stressors, ranging from overfishing of herbivorous fish (which allows turf algae to overgrow the coral), to changes in the water quality from coastal development, dredging and agricultural runoff. However, two of the major threats are global issues resulting from the billions of tonnes of CO<sub>2</sub> we pump into our atmosphere each year. Firstly, global warming is heating the ocean at a record rate; in fact the world''s reefs have recently (2015-2016) experienced the third global coral bleaching event ever recorded. And secondly, as well as acting as a sink for heat, the ocean also absorbs CO<sub>2</sub> gas, which is changing the chemistry of the seawater and driving ocean acidification. A warming and acidifying ocean are pushing corals to their limits and it is important now more than ever to communicate the plight of the world''s coral reefs.</p>\r\n\r\n<p>In recent years it has become essential to understand how multiple stressors will affect corals (read also the Break: <strong><a data-mce-href="http://thesciencebreaker.com/2016/07/08/break022/" href="https://www.thesciencebreaker.com/break.php?breakId=60" target="_blank">Ocean acidification and its effects on coral reef growth</a></strong>). Of course there are always winners and losers when conditions change, therefore it is also important to investigate the variability in responses to stress among different species, corals from different environments and in different stages of their lifecycle.</p>\r\n\r\n<p>Our research focussed on juvenile corals from the sub-tropical <a data-mce-href="https://en.wikipedia.org/wiki/Houtman_Abrolhos" href="https://en.wikipedia.org/wiki/Houtman_Abrolhos" target="_blank">Houtman Abrolhos Islands</a> in Western Australia. Juvenile corals are small (one-month-old recruits in this study were around 1 mm in diameter) and yet are immediately faced with the same natural pressures faced by adult corals on the reef. These include; overgrowth by other benthic plants and animals, predation, and damage from storms. It is therefore important for young corals to rapidly grow into more robust size classes. However, changes to their physical and chemical environment have been shown to reduce their growth rate. In this study we were interested in how high temperature and CO<sub>2</sub> affect not only the overall growth rate of young corals, but also the structure of the juvenile skeleton.</p>\r\n\r\n<p>We cultured and grew corals from larvae to one-month-old juveniles under elevated temperature and CO<sub>2</sub> regimes simulating the conditions that will likely be experienced in our oceans by the year 2100. We then analysed the skeletons to both view and quantify the changes to the structure. Since coral recruits are so tiny, it has been difficult in the past to show how the skeletal structure changes under these stressful conditions. However advancements in <a data-mce-href="https://en.wikipedia.org/wiki/X-ray_microtomography" href="https://en.wikipedia.org/wiki/X-ray_microtomography" target="_blank">MicroCT technology</a> made it possible for us to reconstruct high resolution 3D images using a 3D X-ray microscope. This 3D approach allowed us to view the skeleton from any angle, look inside at internal structures, quantify changes in the volume of the skeleton and make cross sectional measurements.</p>\r\n\r\n<p>We found that acidification caused reduced overall mineral deposition and structural deformities in the skeleton. The deformities included reduced vertical growth, reduced thickness of individual skeletal structures, over or undersized structures, as well as large gaps in the lattice structure of the skeleton. We also observed a more porous and fragile skeleton with a higher frequency of fractures in the corals grown under high CO<sub>2</sub>.</p>\r\n\r\n<p>Interestingly, temperature did not have a negative effect on skeletal growth. We expected temperature and CO<sub>2</sub> to have a cumulative effect; however temperature either had little effect or a mitigative effect against CO<sub>2</sub>. We think this response could be unique to sub-tropical juveniles. It may be related to both the large dispersal distances possible in the larval phase (and therefore a potentially wide range of thermal environments experienced) and because the experiment was conducted in the sub-tropics, where there is a broader annual range in temperatures.</p>\r\n\r\n<p>This study highlights how susceptible young corals are to ocean acidification. Reduced growth and structural robustness could reduce early survival rates, as young corals need to grow rapidly to maintain a position on the substrate. The reef-wide implication for reduced growth and survival of juveniles is reduced overall recruitment success on a reef. Successful recruitment is particularly important when a disturbance such as the recent global bleaching event occurs, because the yearly influx of new young corals are a vital part of reef recovery.</p>\r\n', 4, 'Foster T, Falter J, McCulloch M, Clode P. <a href="http://advances.sciencemag.org/content/2/2/e1501130" target="_blank"><strong>Ocean acidification causes structural deformities in juvenile coral skeletons</strong>.</a> <i>Science Advances</i>. 2016;2(2):e1501130-e1501130. doi:10.1126/sciadv.1501130.', 3, '2016-10-19 13:35:19', 56, NULL, 2, 'https://doi.org/10.25250/thescbr.brk029'),
(68, 'How to print a brain - the initial steps', '<p>To understand the biological processes that take part within living organisms, and also as an alternative method to animal experimentation, researchers are trying to develop new models that mimic real tissues and organs, and that behave as the real ones. This field of research, <a data-mce-href="https://www.nibib.nih.gov/science-education/science-topics/tissue-engineering-and-regenerative-medicine" href="https://www.nibib.nih.gov/science-education/science-topics/tissue-engineering-and-regenerative-medicine" target="_blank">tissue engineering</a>, is a multidisciplinary field where medicine and engineering work together in order to replace, regenerate or improve a part or function of the body.</p>\r\n\r\n<p>However, first approaches <em><a data-mce-href="https://en.wikipedia.org/wiki/In_vitro" href="https://en.wikipedia.org/wiki/In_vitro" target="_blank">in vitro</a></em> (in the lab, outside the normal biological context of the cells), were performed in 2D (two dimension, into a plane). These configurations did not allow cells to behave properly, as they do not recognize the environment where they are placed, which is far from being similar to their natural one.</p>\r\n\r\n<p>3D printing is a technique that has been recently developed to produce three-dimensional objects. It works in the same way that a paper printer, but instead adding layers of ink to make words or images, 3D printers adds layers of a specific material to make an object. This technique has spread very much in recent years, reaching different fields such as fashion, engineering and medicine. Thanks to this spread and because of its layer-by-layer process, it has become a critical tool in tissue engineering. Due to the limitations of 2D tissue models, many research groups are focused on the development of new 3D tissue models using these machines. 3D printing tissue allows for the accurate positioning of cells, and also offers the opportunity of simultaneously printing several biomaterials. These materials will form the <a data-mce-href="http://pharmrev.aspetjournals.org/content/61/2/198?ijkey=514bcdacaa781c38a88d382e1df06827ed146591&amp;keytype2=tf_ipsecsha" href="http://pharmrev.aspetjournals.org/content/61/2/198?ijkey=514bcdacaa781c38a88d382e1df06827ed146591&amp;keytype2=tf_ipsecsha" target="_blank">extracellular matrix (ECM)</a> of the cells, which is the physical structure (scaffold) in which they will attach, proliferate and evolve until they organize in a tissue-like manner.</p>\r\n\r\n<p>To understand this work, we have to know that the brain is an enormously complex organ structured into various regions of layered tissue, and has a lot of interconnected networks, which makes it especially difficult to replicate.</p>\r\n\r\n<p>One of the research groups that is developing these 3D models was able to produce a new 3D layered brain-like structure using a hydrogel called "Gellan Gum" (GG) as the material for the scaffold. A modified peptide (short chain of amino acid) was added to the GG to improve its properties, and then they also added neural cells obtained from the brain of mice. These cells were covered by a thin layer of hydrogel before the 3D printing process, in order to protect them. Then, they used this mix (GG, peptide, and cells), to print a 3D model of a layered neural tissue, much like the brain!</p>\r\n\r\n<p><a data-mce-href="http://www.sciencedirect.com/science/article/pii/S2090123213000969" href="http://www.sciencedirect.com/science/article/pii/S2090123213000969" target="_blank">Hydrogels</a>, which are polymeric soft materials composed by a huge amount of water, have been adopted as standard extracellular matrix substitutes because cells can live in them. These materials are also very interesting because they can degrade as cells generate new tissue.</p>\r\n\r\n<p>This group developed a material in which cells are able to live and thrive; and in addition, the modified peptide they added helped to promote cell adhesion and proliferation compared to the same materials without this specially modified peptide. They also studied other properties of the new material, such as the porosity and the <a data-mce-href="https://en.wikipedia.org/wiki/Diffusion" href="https://en.wikipedia.org/wiki/Diffusion" target="_blank">diffusion</a> of the material, which are important properties for facilitating nutrients, oxygen and waste exchange between cells and their culture medium. The results were also positive and they revealed the importance of different parameters carried out along the study that affects network size and its porosity. For example, if the pore is very small, cells cannot travel along the material, and the nutrients cannot be delivered. If they are very big, cells will not be comfortable and they could not create a new tissue</p>\r\n\r\n<p>This first study was meant to validate a low-cost, simple extrusion technique and a bio-ink with the potential to organize neuron subtypes in layered structures, using a process that could be performed in any cell culture laboratory or bioprinter.</p>\r\n\r\n<p>These brain-like structures offer the opportunity to provide a more accurate representation of 3D <em><a data-mce-href="https://en.wikipedia.org/wiki/In_vivo" href="https://en.wikipedia.org/wiki/In_vivo" target="_blank">in vivo</a></em> environments with applications ranging from cell behavior studies, understanding brain injuries and neurodegenerative diseases to drug testing. Finally, we cannot forget, that all these three-dimensional models offer an alternative to animal testing.</p>\r\n', 3.5, 'Lozano R, Stevens L, Thompson B et al. <strong><a href="http://www.sciencedirect.com/science/article/pii/S0142961215006043" target="_blank">3D printing of layered brain-like structures using peptide modified gellan gum substrates.</a></strong> <i>Biomaterials</i>. 2015;67:264-273. doi:10.1016/j.biomaterials.2015.07.022.', 7, '2016-11-08 14:42:45', 57, NULL, 2, 'https://doi.org/10.25250/thescbr.brk030'),
(69, 'Exploring nature to treat multiple sclerosis', '<p>The chemical diversity in nature is unprecedented and exploring natural products for medical applications has a long tradition with many successful stories to be told. As a matter of fact, natural products or natural product-derived pharmaceuticals, constitute about half of all medical drugs on the market. One of the most well-known examples is the acetylsalicylic acid (Aspirin&trade;), an anti-inflammatory compound originally derived (for the first time in 1828) from the bark of <em>Salix</em> trees. Another example is cyclosporine (Sandimmune&trade;) an <a data-mce-href="https://en.wikipedia.org/wiki/Immunosuppressive_drug" href="https://en.wikipedia.org/wiki/Immunosuppressive_drug" target="_blank">immunosuppressive</a> peptide discovered at the beginnings of the 1970s and isolated from Slime moulds (<em><a data-mce-href="https://en.wikipedia.org/wiki/Slime_mold" href="https://en.wikipedia.org/wiki/Slime_mold" target="_blank">Ascomycotas</a></em>). The discovery of cyclosporine was revolutionary and is still used in the clinics today for the prevention of organ rejection following transplantation. These are just two examples of nature-derived products that found their way into clinical applications. Due to the broad distribution of natural products and thanks to the technological advances in chemical analytical tools to characterize and discover them, we will be hearing a lot more about these kinds of reports in the future (read also the Break: <strong><a data-mce-href="http://thesciencebreaker.com/2016/02/24/break009/" href="http://thesciencebreaker.com/2016/02/24/break009/" target="_blank">Fighting back antibiotic resistance: a new hope from the soil</a></strong>).</p>\r\n\r\n<p>Multiple sclerosis is a chronic inflammatory autoimmune disease affecting approximately 2.5 million people worldwide. It affects the central nervous system and it arises when hyperproliferative <a data-mce-href="https://en.wikipedia.org/wiki/Immune_system#Lymphocytes" href="https://en.wikipedia.org/wiki/Immune_system#Lymphocytes" target="_blank">immune cells</a> induce destruction of the biological substance (<a data-mce-href="https://en.wikipedia.org/wiki/Myelin" href="https://en.wikipedia.org/wiki/Myelin" target="_blank">myelin sheaths</a>) around nerve fibres. As a consequence, communication within the nervous system degrades and leads to the clinical manifestation of the disease. Although several approaches and drugs are already on the market helping patients to cope with the symptoms, it remains an incurable disease. Furthermore, available drugs have to be administered intravenously with severe side-effects.</p>\r\n\r\n<p>A new study led by a research team at the Medical University of Vienna alongside other international partners, highlights the potential application of <a data-mce-href="https://en.wikipedia.org/wiki/Cyclotide" href="https://en.wikipedia.org/wiki/Cyclotide" target="_blank">circular plant peptides (cyclotides)</a> in the treatment of multiple sclerosis. The team has demonstrated that following a plant-derived cyclotide oral treatment in a mouse model for multiple sclerosis, there is no progression of clinical disease symptoms. Importantly, this without inducing major adverse side-effects. Cyclotides are molecules present in very common plant families including coffee, violet flowers, grasses and plants of the nightshade family. They represent a large and diverse group of natural products which exhibit extraordinary stability and chemical properties, optimal prerequisites for oral drug development. Using this collection of nature-derived peptides in several pharmacological test systems, our experiments started by analysing the effects of cyclotide-enriched plant extracts of <em>Oldenlandia affinis</em> (coffee family) and <em>Viola tricolor</em> (violet family) towards certain immune cells (<a data-mce-href="https://en.wikipedia.org/wiki/T_cell" href="https://en.wikipedia.org/wiki/T_cell" target="_blank">T-lymphocytes</a>). We wanted to understand whether the cyclotides could modulate the proliferation of these cells. The proliferation capacity is essential for the immune system to fulfil its function against pathogenic invaders. However, when an autoimmune disease takes place, immune cells proliferation becomes hazardous due to its destructive potential against structures of the human body. Our results showed that certain cyclotides appear to be inhibiting this proliferation without inducing unwanted side-effects. Furthermore, detailed chemical analysis revealed that the natural occurring cyclotide kalata B1 (and later its synthetic analogue [T20K]kalata B1 - in short "T20K") acts as an active molecule to supress proliferation of immune cells. This led T20K to become a promising candidate for preclinical studies in the field of autoimmune diseases, especially multiple sclerosis.</p>\r\n\r\n<p>Having said all of that, our discovery represents a possible breakthrough for the prevention of multiple sclerosis: a single dose of the peptide could prevent attacks and typical disease symptoms in animals. However, T20K is not yet a drug on the market available for clinical use. Hence, at present, there is no direct benefit for patients. Over the next years T20K needs to undergo rigorous clinical testing to confirm safety and efficacy in humans. In this direction, a recent licensing agreement signed by the Medical University of Vienna and the Medical Center - University of Freiburg, led to establish a company called Cyxone in 2015 (<a data-mce-href="http://www.cyxone.com" href="http://www.cyxone.com" target="_blank">www.cyxone.com</a>) aimed at developing novel cyclotide drugs for the treatment of autoimmune diseases. Cyxone''s immediate focus is to bring T20K through a pre-clinical program required for clinical studies in multiple sclerosis. The main objective will be that of developing a safe and orally active drug. This is an exciting perspective as the Phase 1 clinical trial to develop a safe and orally active drug could begin as early as 2018.</p>\r\n', 3.5, '<p>Thell, K., Hellinger, R., Sahin, E., Michenthaler, P., Gold-Binder, M., Haider, T., Kuttke, M., Liutkevičiūtė, Z., Gransson, U., Gründemann, C., Schabbauer, G. and Gruber, C. (2016).<a href="http://www.pnas.org/content/113/15/3960.long" target="_blank">Oral activity of a nature-derived cyclic peptide for the treatment of multiple sclerosis.</a> <em>Proceedings of the National Academy of Sciences</em>, 113(15), pp.3960-3965.</p>', 1, '2016-11-16 14:59:11', 58, NULL, 1, 'https://doi.org/10.25250/thescbr.brk031'),
(70, 'Out of the darkness: how plants prepare for, and endure, life in the sun', '<p>All plants must sense, and respond to, their environment. This perception is of crucial importance to developing seedlings. Initially, they must grow in subterranean darkness in order to reach the surface - a process known as skotomorphogenesis (development in the dark). Once at the surface, and emerging into sunlight, plants grow more slowly, redirecting their energy to producing leaves, and packing them with the green light-capturing pigment chlorophyll. This process is called photomorphogenesis (development in the light), and it is controlled by proteins called phytochromes, which are able to sense sunlight, and then switch-on genes required to use light as a source of energy.</p>\r\n\r\n<p>In addition to this light-sensing mechanism, plants perceive and respond to the luminosity of their environment through a process called retrograde signalling. In this particular case, messages are passed from the cellular machinery to the cell nucleus (the information centre of the cell). The content of these messages tells the cell to put the brakes on photomorphogenesis, and prevent the plant from trying to harness the sun''s energy before its cells are ready to do so.</p>\r\n\r\n<p>In summary, on one side, the plant perceives light and starts preparing itself for life in the sun; on the other side, the plant remains in standby mode starting to harvest the sun''s energy only when it will be ready to do so. Not respecting this balance would lead the plant to sustain heavy damage from which it would not recover easily. On the other hand, spending too much time in standby mode would not allow the plant to gather the necessary energy to survive. Consequently, striking the right balance is crucial for a developing seedling! These two signalling pathways, photomorphogenesis and retrograde signalling, have been the focus of extensive study; but, it was not clear how they coordinate to control seedlings'' development.</p>\r\n\r\n<p>Now researchers at the Centre for Research in Agricultural Genomics and the Department of Bioengineering in Barcelona have collaborated with scientists at the University of California and the US Department of Agriculture to understand how this delicate balance between the two pathways is established and maintained. In order to do so, they studied the genes that determine photomorphogenesis in the model plant thale cress (<a data-mce-href="https://en.wikipedia.org/wiki/Arabidopsis_thaliana" href="https://en.wikipedia.org/wiki/Arabidopsis_thaliana" target="_blank"><em>Arabidopsis thaliana</em></a>), and they measured the length of seedlings'' shoots, and the opening of the emerging leaves, as physical indicators of the fact that the plant exited stand-by mode, and triggered photomorphogenesis.</p>\r\n\r\n<p>The researchers utilized two key tools to investigate these signalling pathways. The first was a modified strain of <em>Arabidopsis</em> in which the light-sensing pathway was permanently switched-on. Consequently, these plants were permanently undergoing photomorphogenesis, even in conditions where they should not have done so. The second tool was a chemical called lincomycin, which switches-on retrograde signalling - telling the cell to put the brakes on photomorphogenesis. By combining these two tools, and monitoring blockages along the two signalling pathways - like a plumber locating blockages in a series of water pipes - the researchers were able to identify the point at which the two pathways meet. This junction is a protein called GLK1. Extending previous studies, they showed that GLK1 is responsible for the promotion of photomorphogenesis, and, consistent with what happens in the plant, it is switched-off by retrograde signalling.</p>\r\n\r\n<p>Furthermore, they demonstrated that when light intensity is very strong, retrograde signalling blocks completely the production of GLK1. Consequently, photomorphogenesis returns to stand-by mode, and plants close their leaves and turn away from the light. Therefore, the retrograde signalling pathway also appears to act as a safety mechanism: it protects plants from exposing themselves to a potentially damaging level of sunlight, for example, when the sun is at its strongest, or when the plant''s light harvesting apparatus is damaged and not able to capture the sun''s light.</p>\r\n\r\n<p>This work is helping scientists understand the biochemical changes that take place in developing plants, how they adapt to changes in their environment, and how they protect themselves from potentially damaging levels of bright sunlight.</p>\r\n', 3, 'Martín G, Leivar P, Ludevid D, Tepperman J, Quail P, Monte E. <strong><a href="http://www.nature.com/articles/ncomms11431" target="_blank">Phytochrome and retrograde signalling pathways converge to antagonistically regulate a light-induced transcriptional network</a></strong>. <i>Nature Communications</i>. 2016;7:11431. doi:10.1038/ncomms11431.', 5, '2016-11-28 15:03:09', 36, NULL, 1, 'https://doi.org/10.25250/thescbr.brk032'),
(71, 'The surprising effects of a Paleo diet on diabetic patients', '<p>Diabetes is a widely spread disease in modern society, a condition in which the capacity of the body to manage blood <a href="http://hyperphysics.phy-astr.gsu.edu/hbase/Organic/sugar.html">glucose</a> is impaired. Type 2 Diabetes (T2D) is associated with many metabolic dysfunctions, such as insulin resistance (the inability of the tissues - mainly muscle and fat - to absorb glucose from the blood in response to the insulin hormone signal), hypertension and dyslipidemia (abnormal amount of lipids in the blood).</p>\r\n\r\n<p>T2D depends on both genetic and lifestyle factors: the typical Western diet (processed meat, high-fat dairy products and refined grains) is associated with T2D development<sup>1</sup>. The so-called Paleo(lithic) diets limit the food intake to items that were most likely consumed by our ancestors during the Paleolithic period (2.5 MYA - 10,000 BP), and are hypothesized to improve T2D and its associated metabolic impairments<sup>2,3</sup>. Paleo-diets are rich in meats, fish, fruits, vegetables and nuts and they do not include processed foods, dairy products and refined grains. They are alsolower in <a href="https://medlineplus.gov/ency/article/002415.htm">sodium</a>and higher in <a href="http://umm.edu/health/medical/altmed/supplement/potassium">potassium</a>, <a href="https://medlineplus.gov/antioxidants.html">antioxidants</a>, <a href="http://www.who.int/nutrition/topics/micronutrients/en/">micronutrients</a> and <a href="https://medlineplus.gov/dietaryfiber.html">fiber</a> and have a lower <a href="http://www.precisionnutrition.com/all-about-dietary-acids-and-bases">diet acid</a> content.</p>\r\n\r\n<p>To investigate if there is an improvement in the metabolism of diabetic people after following a Paleo diet, two groups of diabetic patients were assigned to either the Paleo diet or the standard diet recommended by the Amercian Diabetes Association (ADA)<sup>4</sup>, for two weeks. The Paleo diet consisted of meat, fish, poultry, eggs, fruit, vegetables, tree nuts, canola oil, mayonnaise and honey. Dairy products, legumes, cereals, grains and potatoes were excluded. The total number of calories of the Paleo and ADA diet was the same. The Paleo diet had a higher content of carbohydrates (437 vs 408 g) and a lower amount of proteins (138.5 vs 152.5 g). The total amount of fat was higher in the ADA diet (96 vs 90 g) and, importantly, the type of fats was different: the ADA diet had a higher amount of saturated fatty acids (21.5 vs 12 g) and slightly lower amounts of monounsaturated and polyunsaturated fatty acids (46 vs 49.5 g and 20.5 vs 21 g, respectively).</p>\r\n\r\n<p>Metabolic changes were assessed through a series of measures, mainly lipid blood content and specific parameters of blood glucose control. Blood lipids and cholesterol were lower on all patients after both diets but markedly more so in the group under the Paleo diet. A general improvement in glucose control was also found after both diets but again this was much more evident in the group consuming the Paleo diet. The fact that both diets led to a healthier metabolic frame is likely explained from the fact the subjects were conducting a poor dietary regimen before the trial, and that following the ADA diet also led to an improvement compared to previous habits.</p>\r\n\r\n<p>One of the main differences between the two diets that probably contributed to the better health improvement of the group following the Paleo diet, is that in that diet, the fiber content is much higher and it possibly attenuated the after meal glucose rise. Other differences are the higher content of antioxidants, mono- and polyunsaturated fats, low sodium and high potassium in the Paleo diet. Also the source of carbohydrates differed: fruits, vegetables and honey in the Paleo diet, and rice, bread and pasta in the ADA diet.</p>\r\n\r\n<p>One month after the two-week diet trial all the patients were examined again and it was found that the improvements observed during the study tended to go back to their initial values, indicating that in order to have stable, long-term effects of a diet, it should be a permanent lifestyle habit rather than a temporary, time-limited intervention. Besides the specific kind of diet, it is important to underline how strongly food habits influence our health.</p>\r\n', 3, 'Masharani U, Sherchan P, Schloetter M et al. <a href="http://www.nature.com/ejcn/journal/v69/n8/full/ejcn201539a.html" href="http://www.nature.com/ejcn/journal/v69/n8/full/ejcn201539a.html">Metabolic and physiologic effects from consuming a hunter-gatherer (Paleolithic)-type diet in type 2 diabetes.</a> <i>European Journal of Clinical Nutrition</i>. 2015;69(8):944-948. doi:10.1038/ejcn.2015.39.', 1, '2016-12-14 15:49:33', 59, NULL, 2, 'https://doi.org/10.25250/thescbr.brk033');
INSERT INTO `breaks` (`id`, `title`, `content`, `reading_time`, `original_article`, `subject_id`, `date_created`, `author1_id`, `author2_id`, `editor_id`, `imzy_link`) VALUES
(77, 'The Dark Side of Love', '<p>Love isn''t all rainbows and butterflies. Sure, in the beginning, everything is great. Everyone knows the age-old story: boy meets girl, boy courts girl, and if girl accepts then boy mates with girl. Life is good. Unfortunately, this honeymoon stage doesn''t endure. Initially, the interests of the sexes are aligned because mating benefits both males and females. From an evolutionary perspective, these benefits are expressed in terms of number and quality of offspring produced. The name of the game is passing on genes to the next generation, and "success" is measured in offspring number. In this way, sexual reproduction is cooperation between males and females. Everyone is happy...until they are not.</p>\r\n\r\n<p>After mating, things change. Conflict of interest arises when either the male or the female attempts to maximize their own reproductive success at the expense of the other- the schism in the once harmonious relationship. For example, one way for both sexes to potentially increase the quantity and/or quality of offspring they produce is to mate with other partners. However, as females produce a limited number of eggs, remating almost always results in a reduction in the number of offspring fathered by the first male in favour of the next male. Thus, female remating behaviour is under conflict as it would benefit the female but at a large cost to her previous male mate.</p>\r\n\r\n<p><a href="https://en.wikipedia.org/wiki/Sexual_conflict">Sexual conflict theory</a> suggests that males should develop tactics, which decreases the chances of their mates to commit infidelities thereby increasing the chances that they will produce their offspring. There are various examples of how males restrict the reproductive behaviour of the females they mated with, ranging from intensive physical guarding to the more hands-off approach of installing a mating plug within the female reproductive tract to block mating with a future mate. In some insect species, males use "chemical warfare" by transferring chemicals, called <a href="https://en.wikipedia.org/wiki/Pheromone">pheromones</a>, to the female during copulation making her less attractive, and thus less likely to remate. Sexual conflict theory, however, also predicts that females should develop counter-tactics, for example removing the anti-aphrodisiac pheromones, to increase their post-mating attractiveness and increasing their chances of taking on new mates. Shockingly, no female mechanism of this sort has been identified; and so we decided to investigate it.</p>\r\n\r\n<p>Chemical mate-guarding is arguably best understood in the fruit fly <em><a href="https://en.wikipedia.org/wiki/Drosophila_melanogaster">Drosophila melanogaster</a></em>. That''s right: attraction, sex, control, betrayal and infidelity are just a typical day in the love life of fruit flies. Males of this species produce at least two anti-aphrodisiac pheromones, which are transferred during mating and are found on the female directly after mating. One, 7-Tricosene a.k.a. 7-T, is produced on the outside of the male body, and is rubbed onto the female during the act. The other, cis-vaccenyl acetate a.k.a. cVA, is produced inside the male reproductive tract and is transferred to the female reproductive tract along with sperm. A recently mated female that receives both of these chemicals becomes less attractive.</p>\r\n\r\n<p>We wondered, first, why are there two anti-aphrodisiac pheromones and not just one? Second, can the female remove them? And third, does removal result in increased attractiveness?</p>\r\n\r\n<p>To tackle the first question we performed a series of experiments by which females with varying amounts of either one or both pheromones were presented to a male and his courtship was observed. We found that neither chemical alone caused a decrease in female attractiveness, but together they caused males to reduce their courtship behaviour. This suggest that the two chemicals interact together to decrease a female''s post-mating attractiveness. This also suggests that if females are capable of removing at least one of the chemicals then they could potentially disrupt this relationship and increase their attractiveness.</p>\r\n\r\n<p>Female fruit flies normally remove unused portions of the ejaculate from their reproductive tract a few hours after mating, termed sperm ejection. Since cVA is transferred alongside sperm we wondered if sperm ejection also removes cVA. By performing chemical analysis on virgin, recently mated, and recently mated but ejected females, as well as the "sperm mass" ejected from the female reproductive tract we found that females remove the cVA they gain during mating via sperm ejection. Interestingly, we found that after ejection, females regain their attractiveness and go on to not only to be more attractive but are more likely to remate than females that have not yet ejected. Finally, as we and others have determined that the timing of sperm ejection involves the female brain, the removal of cVA is likely an active process by which females control when they eject and therefore control their attractiveness.</p>\r\n\r\n<p>The results from our experiments have illustrated two important principles. First, although males have evolved a mechanism to reduce female attractiveness, benefiting his own reproductive success, female behaviour can influence the effect or influence of these reproductive restrictions. And second, whoever said the way to a man''s heart is through his stomach was wrong: it''s through his nose.</p>\r\n', 4, 'Laturney M., Billeter J. <a href="http://www.nature.com/articles/ncomms12322">Drosophila melanogaster females restore their attractiveness after mating by removing male anti-aphrodisiac pheromones</a>. <i>Nature Communications</i>. 2016; 7:12322. doi:10.1038/ncomms12322.', 4, '2017-01-13 13:16:31', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk034'),
(79, 'Fancy footwork: Darwin’s pigeons and the evolution of foot feathers ', '<p>Ever since Charles Darwin published <em><a href="https://en.wikipedia.org/wiki/On_the_Origin_of_Species" target="_blank">On the Origin of Species</a></em>, people have been fascinated with understanding the mechanisms of how species could change over time. Like sitting down with relatives around the dinner table during the holidays, it is sometimes hard for us to look at other species and understand how we could have a common ancestor, yet wind up looking and acting so differently. We know that heritable traits are encoded in DNA, and that changes in DNA (<a href="http://evolution.berkeley.edu/evolibrary/article/mutations_01" target="_blank">mutations</a>) are responsible for changes in traits. But this raises many questions, such as: how many mutations does it take to evolve a new trait? Which genes are mutated to form a new trait, and how do the mutations affect the gene? And when different species evolve similar traits, did that happen through similar mutations?</p>\r\n\r\n<p>Dr. Mike Shapiro''s lab at the University of Utah has chosen to investigate these questions using one of Darwin''s favorite species, the domestic rock pigeon (<em><a href="https://en.wikipedia.org/wiki/Rock_dove" target="_blank">Columba livia</a></em>). Domestic pigeons are incredibly diverse, with over 350 recognized breeds worldwide. In this study, we focused on the evolution of one remarkable trait in particular, which is the transformation of foot scales into feathers. Like most bird species, most pigeon breeds are scale-footed, but several breeds grow <a href="https://en.wikipedia.org/wiki/Ice_pigeon" target="_blank">feathers on their feet</a>, instead. To study this trait, we took several complementary approaches. In one, we mated a scale-footed pigeon to feather-footed one to shuffle the information embedded within their DNA. Such information were then tracked down in order to find which locations of their DNA correlated with the amount of foot- feathering in the descendants. In another, we compared the whole DNA sequences from multiple feather-footed breeds to scale-footed ones to find differences between the two groups. In a third, we identified genes that had different activation levels during leg development between feather-footed and scale-footed pigeon embryos. All of these approaches pointed to the same, surprising conclusion: there were two main DNA mutations largely responsible for foot-feathering, but they weren''t simply causing the foot skin to produce feathers instead of scales. Rather, they were partially re- programming the pigeon''s <em>leg </em>into a <em>wing</em>.</p>\r\n\r\n<p>During <a href="https://en.wikipedia.org/wiki/Embryogenesis" target="_blank">development</a>, certain genes act to "tell" cells to which body part they belong. For instance, during limb development, a gene called <em>Tbx5 </em>programs the developing forelimb to develop into a wing, while another gene called <em>Pitx1 </em>programs the developing hindlimb to develop into a leg. These same genes are involved in arm and leg development in humans. Normally, they act in the correct limb, so each limb develops properly. In foot- feathered pigeons, however, one mutation was causing <em>Tbx5 </em>to be active in both the forelimb and hindlimb, and another mutation was reducing the amount of <em>Pitx1 </em>activity in the hindlimb. As a result, the leg partially changed its development into a "wing" like one, which resulted in the growth of feathers instead of scales. In addition, these mutations also caused the hindlimb of feather-footed pigeons to develop differences in muscle and skeletal anatomy, too.</p>\r\n\r\n<p>Although the vast majority of birds are scale-footed, pigeons aren''t the only species that have evolved foot-feathering; some domestic chicken breeds (<em><a href="https://it.wikipedia.org/wiki/Gallus_gallus" target="_blank">Gallus gallus</a></em>) have, as well. We performed similar experiments and found that changes in <em>Tbx5 </em>expression contributed to foot-feathering in chickens, too. This means that both pigeons and chickens independently evolved similar changes in the exact same gene to produce foot-feathering. Even more strikingly, previous research had found that the other "foot-feathering" gene, <em>Pitx1</em>, was also involved in the evolution of pelvic spines in a <em>very</em>-distantly related organism, the three-spined stickleback fish (<em><a href="https://it.wikipedia.org/wiki/Gasterosteus_aculeatus" target="_blank">Gasterosteus aculeatus</a></em>) <em>(Shapiro et al., 2004). </em></p>\r\n\r\n<p>So to revisit the questions posed at the beginning of the article, we see that for foot-feathering in pigeons, two mutations were sufficient to produce a dramatic transformation in the appearance of the leg, that these mutations altered expression of two genes important in programming limb-type identity, and that mutations in these same genes are responsible for hindlimb evolution in other animals, too. As we ponder the shared origins and diverse trajectories that life has taken on this planet, this study shows that sometimes a small number of mutations can have a very big effect!</p>', 3.5, 'Domyan, E., Kronenberg, Z., Infante, C., Vickrey, A., Stringham, S., Bruders, R., Guernsey, M., Park, S., Payne, J., Beckstead, R., Kardon, G., Menke, D., Yandell, M. and Shapiro, M. (2016).<a href="https://elifesciences.org/content/5/e12115" target="_blank">Molecular shifts in limb identity underlie development of feathered feet in two domestic avian species.</a> <em>eLife</em>, 5.', 4, '2017-01-20 09:20:56', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk035'),
(80, 'Amoebas trap bacteria using nets of DNA: the same mechanism as human immune cells', '<p>Our multicellular bodies containing trillions of cells seem to have little in common with <a href="https://en.wikipedia.org/wiki/Protist" target="_blank">protists</a>, the tiny single-celled creatures inhabiting every drop of water, which spend their days eating bacteria or each other, parasitizing larger organisms or living from light. And yet, this is how our ancestors looked like. It took almost one billion years to fine-tune the coordination between cells so they can altruistically cooperate within our tissues and organs. However, there are still several features in our cells which don''t let us forget our humble unicellular predecessors. For example, the motile "tails" of sperm cells are nothing else than <a href="https://en.wikipedia.org/wiki/Flagellum#Eukaryotic" target="_blank">flagella</a> which protists use to propel themselves through water. The immune cells circulating through our bodies and defending us from invaders have another hint. The mechanism by which they engulf and <a href="https://en.wikipedia.org/wiki/Phagocytosis" target="_blank">swallow</a> the harmful bacteria reflects the eating habits of our ancestors. Now researchers from Geneva, Switzerland and Houston, Texas discovered another fascinating heritage, also within the immune system.</p>\r\n\r\n<p>It has been known for over ten years that several types of immune cells of humans and other animals can trap and kill pathogenic bacteria by casting nets made of DNA. Yes, the same DNA which stores the information about our bodies and rules over cellular processes can also be used as a lethal weapon against pathogens. The bacteria get immobilized in the sticky DNA like in a fishing net and then are killed by antimicrobial chemicals covering the DNA strings. Discovery of this addition to the immune cells'' arsenal, besides their already known abilities to swallow bacteria and to secrete antimicrobial substances, was astonishing. However, it is now becoming even more interesting as we learned that this mechanism is not a recent innovation by advanced multicellular animals but an ancient feature shared with at least one lineage of protists, the <a href="https://en.wikipedia.org/wiki/Dictyostelid" target="_blank">social amoebas</a>.</p>\r\n\r\n<p>Most amoebas live lonely unicellular life but the so called social amoebas developed a remarkable instance of multicellularity. The individual cells of social amoebas spend most of the time crawling around in soil just like their solitary relatives. When stressed, for example by food scarcity, the amoebas start to communicate using chemical signals and join to form a millimeter-sized multicellular stage called slug. The slug then travels for many hours like a single body to find an ideal spot to form a final life stage, the fruiting body, which resembles a mushroom, and will then release spores. Cells within the two multicellular stages are differentiated into groups with diverse functions and even form tissues. Like any other multicellular body, the slug attracts the unwelcome attention of hungry bacteria, and so it needs an immune system. This role is played by a special population of cells which swallow and kill the invading bacteria just like in the human body. It was a great surprise for the researchers when they saw these cells showing also a behavior clearly resembling the casting of DNA nets by human immune cells.</p>\r\n\r\n<p>They observed the amoeba cells releasing a cloud of unknown material when the slug was crawling across a patch of pathogenic bacteria, and the stricken bugs were dying. The researchers performed series of biochemical experiments that showed that the released material is indeed DNA. They also added bacteria-sized beads to the slug''s environment and saw that these are rapidly trapped by the DNA. So, they knew they were witness to a familiar phenomenon. But is it related to the animal DNA nets mechanism? Or did it evolve separately in the social amoebas? After all, they are not closely <a href="https://en.wikipedia.org/wiki/Eukaryote#Classification" target="_blank">related</a> to us (more than plants but less than fungi) and our <a href="https://en.wikipedia.org/wiki/Most_recent_common_ancestor" target="_blank">last common ancestor</a> lived more than a billion years ago. To answer this final question, researchers genetically manipulated the social amoebas in order to understand the regulatory mechanisms which rule over the DNA nets formation and found that the mechanism is largely identical between amoebas and humans, i.e. the genes in charge of coordinating the process are the same in both lineages. It is very unlikely that such a complex mechanism would evolve twice separately, with the same regulatory genes. Thus, the researchers conclude that trapping and killing bacteria by casting poisoned nets of DNA is an ancient feature that must have been present already in our single-celled ancestors. The social amoeba may now become a <a href="https://en.wikipedia.org/wiki/Model_organism" target="_blank">model organism</a> for studying human immunity.</p>', 3.5, 'Zhang X, Zhuchenko O, Kuspa A, Soldati T. <a href="http://www.nature.com/articles/ncomms10938" target="_blank">Social amoebae trap and kill bacteria by casting DNA nets</a>. <em>Nature Communications</em>. 2016;7:10938. doi:10.1038/ncomms10938.', 4, '2017-01-27 10:50:38', NULL, NULL, 2, 'https://doi.org/10.25250/thescbr.brk036'),
(81, 'Cloudy days cost yield until scientists hacked photosynthesis', '<p>Throughout the growing season seemingly benign clouds pass over millions of acres of crops and inadvertently rob plants of their productivity, costing untold bushels of potential yield. Researchers recently reported in the journal Science that they have engineered a solution and increased the productivity of a crop in the field by 14- 20 percent - they believe this fix could be applied to staple food crops to help meet future global food demands.</p>\r\n\r\n<p>When it''s too bright outside, you put on your sunglasses. Return to the shade, and you can quickly take them off. Plants have evolved their own sun protection, called <a href="https://en.wikipedia.org/wiki/Photoprotection" target="_blank">photoprotection</a>, which leaves turn on in full-sun and turn off when shaded by a cloud or another leaf. Without this process, full sunlight would cause oxidation and bleach the leaves - the plant equivalent to a severe sunburn.</p>\r\n\r\n<p>Unlike you, it can take minutes or hours for the plant to remove its sunglasses and return to <a href="https://en.wikipedia.org/wiki/Photosynthesis" target="_blank">photosynthesizing</a>- turning light into yield - at maximum capacity. During this time, the plant siphons off much of the light energy as heat, yet it could have used this energy to make more plant mass. Researchers predicted that this costs plants 7.5-40 percent of their yield, depending on the type of plant and temperature.</p>\r\n\r\n<p>Steve Long, Gutgsell Endowed Professor of Plant Biology and Crop Sciences at the University of Illinois, teamed up with Krishna Niyogi, a professor at the University of California, Berkeley, to boost the levels of three proteins and speed up plant recovery from photoprotection. Their modified plants proved 14-20 percent more productive than unaltered plants in replicated field tests.</p>\r\n\r\n<p>Postdoctoral researchers Katarzyna Glowacka and Johannes Kromdijk in Long''s lab conducted this study using tobacco because it is easy to genetically modify, yet it is a real crop that serves as a good proxy for food crops that are much harder to genetically engineer. By growing tobacco in field trials, they could study how the plants respond in the real world with uncontrolled variables, such as sunlight, clouds, and rain. They have no intention of using this "improved" tobacco beyond these experiments; it was simply a test bed to see if the technology worked.</p>\r\n\r\n<p>Now that they know it does, they are embarking on the harder task of transferring this to important food crops - rice, cassava, and cowpea. More human calories are derived from rice than any other crop worldwide. Rice produces 176,000 calories/hectare/day while cassava produces more than 250,000. These crops are especially important to smallholder farmers in Sub-Saharan Africa and Southeast Asia who need to produce more food to provide for their families and communities.</p>\r\n\r\n<p>Will it work? The process that was modified in tobacco is similar to that used by all crops, so the researchers have every reason to expect that it will. The <a href="http://www.gatesfoundation.org/" target="_blank">Bill & Melinda Gates Foundation</a>, which funded this work, will help ensure that this technology reaches these farmers at a low cost.</p>\r\n\r\n<p>Today, we have a seeming abundance of food, so why do we need to bother trying to make more? Long points out that with regulatory hurdles, testing, and multiplication, it will take 15 to 20 years at minimum to get these findings from their lab and into farmers'' fields at scale. Meanwhile, by 2050, we are going to need to produce 70% more food than we are today according to the United Nations Food and Agriculture Organization. David Tillman, Regents Professor and McKnight Presidential Chair in Ecology at the University of Minnesota, suggests that it''s actually more like 100% more food. This lag between discovery and deployment means we are just one crop breeding cycle away from starvation. We therefore need to be innovating now for our future.</p>\r\n\r\n<p>But this study is promising. It shows that we may meet this impending demand - and in a way that many scientists once doubted: tweaking photosynthetic mechanisms to increase yield was once considered the holy grail of plant biology. Now it might be our best to usher in the next green revolution and meet the food demands of the 21st century.</p>', 3.5, 'Kromdijk J, Głowacka K, Leonelli L, Gabilly S, Iwai M, Niyogi K, Long S. <a href="http://science.sciencemag.org/content/354/6314/857" target="_blank">Improving photosynthesis and crop productivity by accelerating recovery from photoprotection</a>. <em>Science</em>. 2016;354(6314):857-861. doi:10.1126/science.aai8878.', 5, '2017-02-02 09:24:37', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk037'),
(82, 'Aluminium in antiperspirants: an effective tool or a breast cancer threat?', '<p>Aluminium is the most abundant metal in Earth''s crust. Due to its abundance and to its remarkable physical and chemical properties - it is lightweight, durable, and resistant to corrosion - aluminium is widely present in many different industrial products, including sunscreens, lipsticks, toothpastes, anti-acid drugs, food additives and vaccines. As a salt, aluminium is used as an extremely efficient antiperspirant. The precise reasons for this property are not know, but are thought to involve physical obstruction of sweat ducts.</p>\r\n\r\n<p>Aluminium''s wide presence in industrial products could lead to the assumption that it is a safe material. However, surprisingly little is known about aluminium''s distribution and removal from the body after its absorption. Studies appeared in recent years (1, 2 and references therein) have shown that aluminium accumulates in organs such as the mammary gland and the brain. The reason for this accumulation is not known, since aluminium is not an essential element and has no known biological function.</p>\r\n\r\n<p>Whereas it is now generally accepted that aluminium is toxic for the brain (2, and references therein), with Alzheimer disease being one of its potential toxic effects, at the time we started our experiments, little was known about the role that aluminium could have in in breast cancer development.</p>\r\n\r\n<p>The hypothesis that aluminium might induce or contribute to breast cancer was first put forward by British researcher Philippa Darbre (3). She built her hypothesis on the following main elements. First, the reported increased incidence of breast cancer in Western societies during the last few decades. Second, the concomitant change in the localization of breast cancer, that arises now more frequently in the external part of the mammary gland, close to the axilla, where antiperspirants are applied on a daily basis. Third, the high levels of aluminium found in the human mammary gland. Fourth, the high content of aluminium present in the large majority of commercially available antiperspirants. Fifth, the capacity of aluminium to cross the human and mouse skin.</p>\r\n\r\n<p>Biology is an experimental science, where hypotheses must be confirmed, or infirmed, experimentally. In our experiments, we used human or mouse epithelial cells (the cells of the mammary gland that produce and transport milk and which breast cancer arises from). These cells can be "<a href="https://en.wikipedia.org/wiki/Cell_culture" target="_blank">cultivated</a>" in the laboratory and either analyzed <em>in vitro</em> or injected into appropriate mouse strains to observe their behavior within a living organism. As a matter of fact, these experimental models have been key players in building the knowledge of biology, physiology and oncology of the last 30 years.</p>\r\n\r\n<p>In our experiments, we exposed cultures of mammary epithelial cells for several weeks to concentrations of aluminium in the range of those measured in the human mammary gland. Remarkably, those cells became "transformed". That is, they exhibit functional changes typical of the process where normal cells become cancerous. Such changes include the accumulation of <a href="https://en.wikipedia.org/wiki/Mutation" target="_blank">mutations in their DNA</a>, and the capacity to overcome the proliferations'' limits of normal cells. Furthermore, these aluminium-exposed cells, when injected into the flank of mouse strains validated for research in oncology, formed tumors and metastases, whereas controls cells (cells which have not been treated with aluminium) did not.</p>\r\n\r\n<p>These effects are typical of well-known carcinogens. Whereas additional research is necessary to formally conclude that aluminium is a breast carcinogen, our results demonstrate that it is far from being biologically inert. They considerably strengthen the hypothesis that aluminium might contribute to breast cancer and perhaps to other human cancers.</p>\r\n<br>\r\n1. Darbre PD. Aluminium and the human breast. Morphologie. 2016, 100(329):65-74. </br>\r\n2. Mirza A, King A, Troakes C, Exley C. Aluminium in brain tissue in familial Alzheimer''s disease. J Trace Elem Med Biol. 2017, 40:30-36. </br>\r\n3. Darbre PD. Underarm cosmetics are a cause of breast cancer. Eur J Cancer Prev. 2001, 10(5):389-93.\r\n', 3, 'Mandriota S, Tenan M, Ferrari P, Sappino A. <a href="http://onlinelibrary.wiley.com/doi/10.1002/ijc.30393/full" target="_blank">Aluminium chloride promotes tumorigenesis and metastasis in normal murine mammary gland epithelial cells</a>. <em>International Journal of Cancer</em>. 2016;139(12):2781-2790. doi:10.1002/ijc.30393.', 1, '2017-02-09 15:21:07', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk038'),
(92, 'Capturing Mother Nature at work: seeing how plants make vitamin B6', '<p>Vitamins are essential for life. They perform a huge variety of tasks within metabolism, with many helping to promote biochemical reactions in our bodies. In general, we cannot make vitamins from scratch, and so we must obtain them from our diet. Plants and microorganisms can make these compounds <em>de novo</em>, and are therefore a good source of vitamins for humans, which is why it''s essential that we incorporate such resources within a healthy, varied diet.</p>\r\n\r\n<p>The study of vitamins is important for an understanding of health but it is also crucial in combating many infectious diseases. Ideally, treatments for microbial infections are designed to target essential functions of the microbes that do not overlap with what humans and animals have, i.e. killing the microbes while leaving our bodies free from harmful side effects. For this reason, drugs that target vitamin production in pathogens would be ideal candidates for new antibiotics - a goal of growing importance in a world of increasing antibiotic resistance - as humans cannot produce vitamins.</p>\r\n\r\n<p>The B vitamins are a family of eight different vitamins. One of these vitamins, vitamin B<sub>6</sub>, is involved in hundreds of biochemical reactions - more than any other known nutrient - and so vitamin B<sub>6</sub> deficiency is linked to a vast range of diseases from anaemia to epileptic-like convulsions in infants and depression in adults.</p>\r\n\r\n<p>Biological organisms drive chemical reactions using enzymes, miniature protein machines, which speed up reactions. Plants make vitamin B<sub>6</sub> with an enzyme called vitamin B<sub>6</sub> synthase, which coordinates a long sequence of diverse reactions, much like a biochemical version of an industrial production line. This production line is split between two locations within the physical structure of the enzyme itself (called the active sites). This is unusual: most enzymes promote just one or two kinds of reactions, before passing their products on to another enzyme to continue the biochemical manufacturing process.</p>\r\n\r\n<p>As most enzymes are much too small to see by eye, even with the most powerful light microscopes, scientists use a technique called X-ray crystallography to capture snapshots of these miniature machines at work. When water evaporates from a solution, the material that is left behind may stick together in a regular arrangement, forming a crystal. This is the process that produces salt crystals on rocks along a sea-shore, or sugar crystals around the rim of a honey jar. Enzymes can crystallise out of solution in this way too. Such crystals produce patterns when illuminated with X-rays, from which the 3D architecture of the enzyme can be determined. Solving 3D models of enzymes in the presence of the compounds they use for their reactions, e.g. metabolites, allow scientists to decipher how these machines work.</p>\r\n\r\n<p>Researchers at the University of Geneva purified vitamin B<sub>6</sub> synthase found in the plant, thale cress (<em>Arabidopsis thaliana</em>), and determined its structure by X-ray crystallography. They discovered that it forms a double ring, containing 12 copies of the enzyme, resembling two doughnuts stacked on top of one another. They were able to see the two different sites where vitamin B<sub>6</sub> is manufactured in the enzyme; and, where they expect raw materials are brought in, and the products released.</p>\r\n\r\n<p>Vitamin B<sub>6</sub> synthase uses three different chemical ingredients to build vitamin B<sub>6</sub>. The researchers found that when they added two of these chemicals to the crystals, the enzymes that make up the crystals began the process of making vitamin B<sub>6</sub>. However, in the absence of the third chemical ingredient, the biochemical production line stops, allowing scientists to see what the enzyme looks like part way through assembling vitamin B<sub>6</sub>. Remarkably, at this exact point on the assembly line, they saw how vitamin B<sub>6</sub> synthase transfers the partly processed chemical ingredients from the first to the second location in the enzyme, the latter taking advantage of a single amino acid acting as a swinging-arm. This amino acid is located between the two assembly locations in the enzyme, and swings back and forth between them, collecting the processed chemical from the first site, and carrying it across to the second site.</p>\r\n\r\n<p>The Geneva study was the first time that this single amino acid swinging-arm mechanism had been observed within an enzyme. More recently, these observations were corroborated by an independent study conducted by researchers based at the Universities of Southampton in the UK and Heidelberg in Germany<sup>1</sup>.</p>\r\n\r\n<p>The novel results of this research are helping scientists understand how enzymes coordinate long sequences of complex biochemical reactions, as well as improving our understanding of the production of vitamins, essential for health and fitness, and a potential target for the development of new antibiotics. </p>\r\n\r\n<font size="1"><strong>ADDITIONAL REFERENCE:<br /></strong>1. Rodrigues <em>et </em>al., <a href="http://www.nature.com/nchembio/journal/v13/n3/full/nchembio.2273.html" target="_blank">Lysine relay mechanism coordinates intermediate transfer in vitamin B6 biosynthesis</a>, Nature<em> Chemical Biology </em>(2017)</font> ', 4, 'Robinson G, Kaufmann M, Roux C, Fitzpatrick T. <a href="http://www.pnas.org/content/113/40/E5821.long" target="_blank">Structural definition of the lysine swing inArabidopsis thalianaPDX1: Intermediate channeling facilitating vitamin B6biosynthesis</a>. <em>Proceedings of the National Academy of Sciences</em>. 2016;113(40):E5821-E5829. doi:10.1073/pnas.1608125113.', 5, '2017-02-16 09:24:32', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk039'),
(93, 'What happens to our genes in the twilight of death?', '<p>Death -- the ultimate end of everyone''s journey. What is there to study? Is anything interesting happening? Aside from religious and philosophical discourses, valuable knowledge might be obtained from tangible physical facts. Consider an analogy: a disaster happens in a chemical plant that results in its halt and potential destruction. A chemical plant is a complex chain of reactors linked together by a multitude of control networks that normally determine its functionality. Most disaster-like processes involve a sequence of events that occurs due to the availability of residual energy and materials. Understanding this process helps us determine whether it may be stopped and reverted. It might even provide information on what parts are salvageable. Similar to a disaster at a chemical plant, our study investigated gene expression in postmortem animals in order to project our findings to adult humans that have suddenly died.</p>\r\n\r\n<p>Humans are complex creatures capable of elaborate functions, such as the ability to talk, walk, and monitor the environment - all at the same time. This capacity is due to the coordinated "turning-on" and "turning-off" of thousands of genes at the right time, in the right place, and in the right amount (1). The genes that are "turned-on" (<em>i.e.</em>, activated) produce messages (transcripts) that ultimately enable elaborate functions. There is a lack of information about these processes in death, such as when a person suddenly dies from a massive heart attack or severe brain injury. Several studies have investigated the effects of cell death on gene expression; however, the study of gene expression in death has received little to no attention so far.</p>\r\n\r\n<p>We extracted gene transcripts (<em>i.e.</em> messages from genes which are "turned-on") from dead zebrafish and the livers and brains of dead mice that were incubated for times spanning from life to 48-96 h postmortem. At each time sample, the abundances of gene transcripts were precisely determined by using a technique called "<a href="http://peteranoble.com/noble_pubs/pozhitkov_2014.pdf" target="_blank">Gene Meter</a>" (2,3). A gene was identified as being activated (<em>i.e.</em>, "turned-on") when its transcripts were more abundant in the dead tissue than in flash-frozen, <em>i.e.</em>, "live" tissue. We determined the functions of the activated genes by surveying the literature and they were characterized into major categories, ranging from genes involved in embryo development to those involved in detecting infection or injury.</p>\r\n\r\n<p>Hundreds of genes were activated in the twilight of death -- the period of time between the cessation of elaborate functions and the death of all cells. Interestingly, the genes were activated reproducibly in several individuals. They were not all activated at the same time - nor were they activated at random. Rather, the genes were activated sequentially, suggesting an <em>order </em>to the activation of genes in death. These findings are relevant to forensic science because the timing of the activations could be used to precisely determine the time of death (<em>i.e.</em>, postmortem interval) in suspicious crimes and natural deaths (4).</p>\r\n\r\n<p>As expected, genes involved in survival and in stress response were activated in the twilight of death. Products from these genes are designed to cope with perturbations and to help restore homeostasis.</p>\r\n\r\n<p>What was unexpected was that genes involved in embryo development and cancer were activated. Developmental genes, typically activated during embryonic stages, are usually silenced (for life) following those stages (5). Activation of these genes suggests that they are no longer silenced presumably because either the postmortem physiological conditions resemble those during development or simply because "the brakes are burned". It is possible that the activation of these genes involves the unwinding of the compacted genomic DNA that in turn allows access to other genes that were previously silenced (<em>e.g.</em>, development genes). The activation of cancer-associated genes was a serendipitous finding, discovered through a survey of cancer databases. Apparently, the number of activated cancer genes increased from minutes after death to up to 24 h postmortem and then stopped. This has implications to liver transplant recipients because cancer-associated genes may be activated in the donor and transferred to the recipient, resulting in an increased risk of cancer. Apparently, patients with liver transplants have a higher incidence of cancer than control groups (6,7).</p>\r\n\r\n<font size="1"><strong>ADDITIONAL REFERENCES:<br /></strong>1. François Jacob and Jacques Monod, "Genetic Regulatory Mechanisms In The Synthesis Of Proteins", <em>Journal of Molecular Biology</em> 3, no. 3 (1961): 318-356.<br />2. Alex E. Pozhitkov et al., "A Revised Design For Microarray Experiments To Account For Experimental Noise And Uncertainty Of Probe Response", <em>PLoS ONE</em> 9, no. 3 (2014): e91295.</span><br />3. A. Harrison et al., "Physico-Chemical Foundations Underpinning Microarray And Next-Generation Sequencing Experiments", <em>Nucleic Acids Research</em> 41, no. 5 (2013): 2779-2796.<br />4. Hunter M.C., Pozhitkov A.E. and Noble P.A. 2016. Accurate predictions of postmortem interval using linear regression analyses of gene meter expression data. Accepted February 23, 2017, <em>Forensic Science International</em><br />5. Vladimir B. Teif et al., "Nucleosome Repositioning Links DNA (De)Methylation And Differential CTCF Binding During Stem Cell Development", <em>Genome Research</em> 24, no. 8 (2014): 1285-1295.</span><br />6. Fredrik Aberg et al., "Risk Of Malignant Neoplasms After Liver Transplantation: A Population-Based Study", <em>Liver Transplantation</em> 14, no. 10 (2008): 1428-1436.<br />7. Elizabeth B Haagsma et al., "Increased Cancer Risk After Liver Transplantation: A Population-Based Study", <em>Journal of Hepatology</em> 34, no. 1 (2001): 84-91.</font> ', 3.5, 'Pozhitkov A, Neme R, Domazet-Lošo T, Leroux B, Soni S, Tautz D, Noble P. <a href="http://rsob.royalsocietypublishing.org/content/7/1/160267" target="_blank">Tracing the dynamics of gene transcripts after organismal death</a>. <em>Open Biology</em>. 2017;7(1):160267. doi:10.1098/rsob.160267.', 1, '2017-02-28 13:47:41', NULL, NULL, 2, 'https://doi.org/10.25250/thescbr.brk040'),
(103, 'Our internal fight against loneliness', '<em>"A guy needs somebody -- to be near him... A guy goes nuts if he ain''t got nobody." </em><br /> \r\n\r\n<em>Of Mice & Men, John Steinbeck.</em> <br />\r\n\r\n<p>As social creatures, all aspects of our daily lives are powerfully shaped by our social experiences. The social bonds that we nurture throughout our lives provide us with protection, support, companionship, and enjoyment: enabling the human race to survive and flourish. Our innate craving for social connection is plainly revealed by the intensely unpleasant experience of loneliness - a concept that the characters of John Steinbeck''s <em>''Of Mice and Men''</em> were only too familiar with. While, arguably, we all have an intuitive knowledge of what loneliness feels like, it can be defined as ''a perceived absence of satisfying social connections''. Given the vast array of social opportunities available in this day and age, it might seem paradoxical that loneliness is increasing throughout society. This comes with the burden of low emotional state and poor mental health, but also brings with it damaging effects to physical well-being.</p>\r\n\r\n<p>Like any threat to well-being, our body is poised to counteract threats to our social network. Loneliness motivates us to seek out social contact, in a similar way to how we might be driven to find food if we felt hungry. So perhaps loneliness exists in order to encourage social contact in times of need...but then how does this occur? What makes loneliness so unpleasant, and how does this make us crave social contact? Whether it''s eating, drinking, exercising, or socialising, the cells and pathways in our brain are crucial for guiding our every move. When we''re hungry, for example, brain regions responsible for food-seeking become active, and direct our behaviour towards detecting and acquiring food. So when we''re lonely, are there brain regions which encourage us to engage in social contact? And if so, what are these regions, and what neural machinery underlies this type of behaviour?</p>\r\n\r\n<p>These questions lie at the intersection of psychology and neuroscience, but require deeper analysis than we are capable of in the human brain. In our study, we therefore chose to look at what happens in the brain of another social species - mice - who, like us, prefer to reside in social groups than in isolation. So how can we begin to study loneliness in mice? We first considered how the brain might respond to mice being unexpectedly isolated from their daily companions. This led us to find a group of cells in the brain that show changes in their activity levels just one day after mice are socially isolated. These cells were a group of <a href="https://en.wikipedia.org/wiki/Dopaminergic_pathways" target="_blank">dopamine neurons</a> located in a region of the brain known as the <a href="https://en.wikipedia.org/wiki/Dorsal_raphe_nucleus" target="_blank">dorsal raphe nucleus (DRN)</a>. This finding suggested that these cells are able to sense a lack of social contact in the animal''s environment.</p>\r\n\r\n<p>This led us to our second question - what happens to the behaviour of mice when we trigger activity in these cells? Remarkably we saw two distinct changes: firstly, mice became more social; and secondly, they chose to avoid activation of these cells when possible. While we cannot claim to know that these mice are experiencing loneliness, these changes are reminiscent of ''loneliness-like'' behaviour, i.e. they suggest an undesirable emotional state, but also an increased motivation to be sociable. From this, we might therefore draw the conclusion that increased activity in these DRN cells is signalling the negative aspects of being lonely, and directing behaviour towards social opportunities.</p>\r\n\r\n<p>However, just like humans, not all mice are created equally, and not all will experience the same social scenario in the same way. Social hierarchies quickly form in groups of mice, and this results in a different quality of social interactions for different mice. Dominant mice might be seen as the popular kid in the playground, or the school bully, with subordinates yielding to them in confrontations, and giving up their lunch money on demand. Remarkably, this difference in social rank was unveiled within our results: mice with a higher rank were more responsive to changes in DRN activity. Therefore, one possibility is that dominant mice value maintaining their current social group more than lower-ranking mice do, and so this makes them more sensitive to brain activity that signifies loss of that social contact.</p>\r\n\r\n<p>Although there is a long history of loneliness discussion in literature dating back to ancient philosophers, it was not until the 1960''s that we saw a flurry of investigation by psychologists. As loneliness research continues its transition into the neuroscience field, we continue to delve deeper into this phenomenon, and strive to understand how our brain activity shapes this experience, with its emotional and physical repercussions. Further study will enable us to appreciate how our brains are wired to make us social creatures, and how differences in social skills might arise in different individuals. This will ultimately contribute to our understanding of how aberrant social behaviours and a lack of motivation for socialising can arise in psychiatric disorders such as autism.</p>', 4, 'Matthews G, Nieh E, Vander Weele C, Halbert S, Pradhan R, Yosafat A, Glober G, Izadmehr E, Thomas R, Lacy G, Wildes C, Ungless M, Tye K. <a href="http://www.cell.com/cell/abstract/S0092-8674(15)01704-3" target="_blank">Dorsal Raphe Dopamine Neurons Represent the Experience of Social Isolation</a>. <em>Cell</em>. 2016;164(4):617-631. doi:10.1016/j.cell.2015.12.040.', 2, '2017-03-09 14:05:56', NULL, NULL, 2, 'https://doi.org/10.25250/thescbr.brk041'),
(104, 'Could we reverse memory loss in Alzheimer’s patients? Mice answer yes!', '<p><a href="https://simple.wikipedia.org/wiki/Alzheimer''s_disease" target="_blank">Alzheimer''s disease</a> is the most common form of <a href="https://en.wikipedia.org/wiki/Dementia" target="_blank">dementia</a>. A striking characteristic is memory loss. In the brain, <a href="https://simple.wikipedia.org/wiki/Neuron" target="_blank">nerve cells or neurons</a> make connections, named synapses, to process information. When the synapses are not functional or when the neurons are not well connected anymore, cognitive deficits arise including loss of memory.</p>\r\n\r\n<p>In Alzheimer''s disease, a molecule called amyloid beta (A&beta;), present in a healthy brain becomes deregulated and starts to accumulate in the brain of patients forming the sadly well-known amyloid plaques, one of the hallmarks of the disease. We know that A&beta; induces synapse loss triggering deficit in memory, but how this occurs remains uncertain.</p>\r\n\r\n<p>A&beta; can affect the levels of certain molecules in the brain, one in particular is called Dickkopf-1 (Dkk1). This is interesting because Dkk1 is an inhibitor of a family of proteins named Wnts, which have recently been suspected to be dysregulated in Alzheimer''s disease. Importantly, an upregulation of Dkk1 is found in the brains of Alzheimer''s patients. Also, A&beta; elevates Dkk1 levels and blockade of Dkk1 protects synapses from A&beta;-mediated degeneration. Our group decided to further investigate the effects of Dkk1 by varying its level in the brain. To achieve this, we used a laboratory technique widely used to engineer genetically modified mice that can produce different levels of a specific molecule at the scientist''s will. To understand the effect of Dkk1 in the brain, we used a mouse which has more Dkk1 (let''s call it a "Dkk1-mouse"<em>)</em> than healthy mice. Hence, the Dkk1-mouse could mimic the changes brought on by A&beta;.</p>\r\n\r\n<p>The Dkk1-mouse with high levels of Dkk1 showed memory deficits. When we studied its brain, we saw a reduction of the number and function of synapses in the hippocampus, a brain area that controls learning and memory. As the only difference between the Dkk1-mouse and a healthy one was the presence of Dkk1, we concluded that Dkk1 induces the loss of synapses, their dysfunction and eventually the loss of memory. So, in short the more Dkk1, the more synaptic dysfunction, the greater decline in memory.</p>\r\n\r\n<p>Given the levels of Dkk1 can be modulated in the Dkk1-mouse, we questioned whether the loss of synapses and memory are reversible. After memory loss, we restored the levels of Dkk1 back to healthy levels. By evaluating the performance of the Dkk1-mouse in memory tests and examining the state of the synapses, we observed that memory recovered back to normal and synapses regenerate. The function of Dkk1 as a blocker of Wnts could be one possible explanation for these encouraging findings. Wnts are essential to maintain functional synapses in the adult brain. Therefore, too much Dkk1 blocks Wnts leading to synapse loss and ultimately memory loss.</p>\r\n\r\n<p>These exciting results suggest that by blocking Dkk1 levels even after the manifestation of cognitive deficits, brain function could be restored offering a possible therapeutic approach for the treatment of Alzheimer''s disease.</p>', 2.5, 'Marzo A, Galli S, Lopes D, McLeod F, Podpolny M, Segovia-Roldan M, Ciani L, Purro S, Cacucci F, Gibb A, Salinas P. Reversal of Synapse <a href="http://www.sciencedirect.com/science/article/pii/S0960982216307758" target="_blank">Degeneration by Restoring Wnt Signaling in the Adult Hippocampus</a>. <em>Current Biology</em>. 2016;26(19):2551-2561. doi:10.1016/j.cub.2016.07.024.', 1, '2017-03-16 08:32:42', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk042');
INSERT INTO `breaks` (`id`, `title`, `content`, `reading_time`, `original_article`, `subject_id`, `date_created`, `author1_id`, `author2_id`, `editor_id`, `imzy_link`) VALUES
(105, 'Lower calorie intake allows monkeys to live long and prosper', '<p>The recent report in Nature Communications settles a persistent controversy in biology of aging research; namely, whether or not caloric restriction (CR), reduced calorie intake without malnutrition, confers health and longevity benefits in nonhuman primates. The University of Wisconsin-Madison and the National Institute on Aging (NIA) at NIH study teams came together to analyze data from the two parallel studies that up until now had reported contrasting outcomes of CR. In 2009, the UW-Madison study team reported significant benefits in survival and reductions in cancer, cardiovascular disease, and insulin resistance for monkeys who ate less than their peers. In 2012, however, the NIA study team reported no significant improvement in survival, but did find a trend toward improved health. Working together, the competing laboratories analyzed data gathered over many years from almost 200 monkeys from both studies. The authors concluded that lower calorie intake in adulthood and later in life confers benefits in terms of both health and survival in primates. A major outcome of these analyses was the beneficial effects of CR on health and disease incidence that was detected for both studies. One of the caveats specific to primate species was that starting on the diet too early in life may not be beneficial for longevity, unlike rodents where the earlier the animals go on the diet the longer the resulting lifespan.</p>\r\n\r\n<p>The CR paradigm was first reported in the 1930s. Since then its effects in increasing longevity and delaying the onset of cancer, cardiovascular disease, neurodegeneration, and diabetes have been firmly established. In the last two decades there has been a large body of work focused on how this simple dietary intervention can change a process as complicated as aging. Much of the mechanistic work has been conducted in short-lived species such as worms, flies, and rodents, but the significance for human beings of those insights relies on a demonstration that the CR paradigm is relevant in primates. The published work puts that concern to rest. At a more basic level the study shows that the rate of aging can be changed in nonhuman primates indicating that it might also be changeable in humans. DNA in humans and <a href="https://en.wikipedia.org/wiki/Rhesus_macaque" target="_blank">rhesus monkeys</a> is 93% identical and there are profound similarities between humans and rhesus monkeys in the rate of aging, the types of diseases of aging that occur, and how they manifest clinically. This makes it extremely likely that the mechanisms engaged by CR to delay aging in monkeys will be translatable to human health and human aging. The last thing to add is that this study makes a strong case for aging itself as a target for pharmacological intervention: if the negative outcomes of aging could be delayed by taking a pill, that would potentially off-set a whole host of aging associated diseases and disorders simultaneously. This strategy contrasts with classic studies that look for ways to target and treat each age associated disease and disorder one at a time.</p>\r\n\r\n<p>As for the take-home message, things have never looked better in aging research. Using CR as a model of delayed aging it is only a matter of time before cellular and molecular aspects of aging that create disease vulnerability are uncovered. These insights will enable researchers to identify effective ways to off-set those changes to improve and extend healthy aging. The anti-aging pill may be closer than we think.</p>', 3, 'Mattison J, Colman R, Beasley T, Allison D, Kemnitz J, Roth G, Ingram D, Weindruch R, de Cabo R, Anderson R. <a href="http://www.nature.com/articles/ncomms14063" target="_blank">Caloric restriction improves health and survival of rhesus monkeys</a>. <em>Nature Communications</em>. 2017;8:14063. doi:10.1038/ncomms14063.', 1, '2017-03-24 17:24:50', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk043'),
(106, '“Who is really, really smart?”  Early differences in boys’ and girls’ assumptions about intelligence', '<p>Common stereotypes associate brilliance with men more than women. Evidence for this association is everywhere around us, but maybe easiest to detect in portrayals of brilliance and genius on TV and in the movies. The vast majority of characters that are supposed to be brilliant, such as Sherlock Holmes or Dr. House, are men. Because of this stereotype, women may feel they are less likely to be brilliant and thus less likely to succeed in careers that are thought to require this characteristic. As a result, women might be underrepresented in these careers. In fact, in a <a href="http://science.sciencemag.org/content/347/6219/262" target="_blank">paper</a> published in the journal <em>Science</em> in 2015, researchers found that women are indeed underrepresented in fields that are thought to require <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0150194" target="_blank">brilliance</a> - fields that include some of the most prestigious careers in our modern society, such as being a scientist or an engineer.</p>\r\n\r\n<p>Might this gender imbalance have its roots in early childhood? To answer this question, we investigated the development of the "brilliance = men" stereotype. If this stereotype is acquired early, it may gradually shape the sorts of careers that girls see themselves as being suited for.</p>\r\n\r\n<p>We first investigated the developmental trajectory of children''s beliefs about which gender is "really, really smart" - a child-friendly way of talking about brilliance. We told a sample of children from the United States a short story about a person who was really smart, without providing any clues to the person''s gender. We then showed children pictures of 4 unfamiliar adults (2 men and 2 women) and asked them to guess which one of them was the protagonist in the story. We looked for whether they picked men or women as being "really, really smart". Our results showed an interesting developmental difference. At the age of 5, boys'' and girls'' answers were very similar. Both boys and girls picked people of their own gender as being "really, really smart". However, starting at age 6, girls became less likely than boys to do this. Girls picked females as "really smart" less often than boys picked males. These answers were given by both white and nonwhite children, and did not seem to vary as a function of parental education and income. This may be the beginning of the "brilliance = males" stereotype we see in adults.</p>\r\n\r\n<p>We then wanted to see if this stereotype related to the activities children were interested in. To answer this question, we showed 5-, 6-, and 7-year-olds an unfamiliar activity and told them that it was for children who are "really, really smart". We then asked children a number of questions to see how interested they were in this activity. The results showed a similar trajectory to children''s beliefs about brilliance. At age 5, boys and girls were equally interested in the activity for "really smart children", but girls became less interested in it (relative to boys) at the age of 6 and 7. In fact, girls'' interest in this activity was lower if they endorsed the stereotype that brilliance is a male quality. Six- and 7-year-old girls'' lower interest towards the brilliance game didn''t have anything to do with the game itself: when we described the exact same game as being for kids who "try really, really hard", girls were just as interested in it as boys.</p>\r\n\r\n<p>To conclude, our studies suggest the stereotype equating brilliance with males seems to be acquired early and is related to the activities that boys and girls are interested in. It is possible that, in the long term, this stereotype steers many young women away from careers that are perceived as requiring brilliance.</p>', 3, 'Bian, L., Leslie, S. and Cimpian, A. (2017).<a href="http://science.sciencemag.org/content/355/6323/389" target="_blank">Gender stereotypes about intellectual ability emerge early and influence children''s interests.</a> <em>Science</em>, 355(6323), pp.389-391.', 1, '2017-04-06 08:13:56', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk044'),
(109, 'Symbiogenesis: how algae and bacteria shaped new genes together', '<p>Genes are an essential component of every living being. They are encoded in the DNA, and contain the information needed to produce a fully-functional organism. Deciphering the origin of new genes in organisms is important to understand how living beings adapted to their environment. Genes can emerge through numerous and diverse mechanisms. For example, new genes in plants and animals are mostly formed by recycling those already present in their own DNA while bacteria often acquire new genes by stealing them from other bacteria living in their environment. Symbiosis, the close association between unlike organisms, can also have an impact on the way genes are formed and this is what our research showed in algae.</p>\r\n\r\n<p>Unlike animals which obtain energy from food, algae use light thanks to a process called photosynthesis. Photosynthesis is not an invention of algae, some bacteria have used it for billions of years. Algae just stole it roughly one billion years ago when their ancestor engulfed and domesticated a photosynthetic bacterium in its cell. The captured bacterium has been completely integrated by the algae host, which resulted in a new cellular compartment in algal cells, called the plastid, where the photosynthesis takes place. Some DNA of this captured bacterium is still present within the plastids, but most of its genes have simply been lost during the transformation process. However, a significant number of bacterial genes has also been transferred to the DNA of the algal cell. Algal cells thus carry a mosaic DNA in their nucleus, since genes from two symbionts, the host lineage and from the captured photosynthetic bacterium, co-localize.</p>\r\n\r\n<p>As said previously, a common way to create new genes in algae is by recycling existing genes present in their DNA. Sometimes this process involves the combination of two different genes which are pasted together, producing mosaic genes. In algae, these mosaic genes could be very unusual, because they could form through the combination of genes from the algae and those stolen from the photosynthetic bacterium. Such genes have been called symbiogenetic genes (S-genes) because of their symbiotic origins. We looked in detail for S-genes in algae in order to know whether such genes existed and what functions they conferred to the algae.</p>\r\n\r\n<p>We disentangled the history of genes in the species of algae in order to detect the origin of each part of genes and we detected 67 S-genes among the 17,000 genes investigated. Importantly, the products of S-genes (the proteins) have very specialized functions in the cell. They mostly work within the plastid, suggesting that most of these novel genes are involved in getting the new energetic central to work. Indeed, many S-genes are involved directly and indirectly in photosynthesis. They are involved in the detoxification of the algal cell but also in <a href="https://en.wikipedia.org/wiki/Phototaxis" target="_blank">phototaxis</a> (i.e. the ability of organisms or cells to move directionally in response to a light stimulus), which is essential for the algae to orient themselves most efficiently to receive light. Of note, some S-genes seem ancient because they are present in a broad diversity of algae (and even land plants that derived from these algae) and thus they may have contributed to the successful establishment of the photosynthetic bacterium during the first steps of the evolution of algae.</p>\r\n\r\n<p>The discovery of this new class of genes (i.e. the S-genes) complexifies the knowledge that we have about gene creation but also helps to understand how the first plants faced the challenges of being photosynthetic. In a broader perspective, S-genes <a href="http://www.the-scientist.com/?articles.view/articleNo/33711/title/Steal-My-Sunshine/" target="_blank">extend the concept of mosaicism</a> in algae to the subgenic level (even their genes are mosaic, having different origins). Since the number of proposed mosaic organisms (i.e. of organisms with compartments and genes acquired from different sources) keeps increasing, it is likely that other S-genes will be discovered in other mosaic organisms.</p>\r\n', 3.5, 'Méheust R, Zelzion E, Bhattacharya D, Lopez P, Bapteste E. <a href="http://www.pnas.org/content/113/13/3579.long" target="_blank">Protein networks identify novel symbiogenetic genes resulting from plastid endosymbiosis</a>. <em>Proceedings of the National Academy of Sciences</em>. 2016;113(13):3579-3584. doi:10.1073/pnas.1517551113.', 4, '2017-05-03 12:58:16', NULL, NULL, 2, 'https://doi.org/10.25250/thescbr.brk046'),
(110, 'The power of our adaptive immunity against Alzheimer’s Disease ', '<p>One of the fears that arise with aging is being afflicted with dementia. <a href="https://simple.wikipedia.org/wiki/Alzheimer%27s_disease" target="_blank">Alzheimer’s disease</a> (simply ”Alzheimer” from now onwards) is the most common type of dementia worldwide, representing up to 60% of total cases of dementia in western countries. Alzheimer is a serious world-health threat that involves 5.2 million of patients only in the United States. Furthermore, due to the increasing average age within modern societies, the number is predicted to rise in the future. Since multiple causes lead to the onset of the disease, Alzheimer is called multifactorial disease. The scientific community considers the involvement of our <a href="https://simple.wikipedia.org/wiki/Innate_immune_system" target="_blank">innate immune system</a> as part of such multiple causes. As a matter of fact, during the early stage of the disease, the chronic activation of <a href="https://simple.wikipedia.org/wiki/Glial_cell" target="_blank">microglia cells</a> causes neuronal cell damage that, in turn, leads to dementia. Microglia are innate immune cells important for the clearance of pathogens, defending the brain environment. In Alzheimer’s patients, microglia cells are not able to perform their job properly, promoting the accumulation of a misfolded protein known to be the sadly-famous marker of this disease: the beta-amyloid protein. Due to that, the chronic inflammation that follows within the brain, leads to neuronal damage that culminates with the shrinking of the brain.</p>\r\n\r\n<p>Innate immunity rarely works independently of adaptive immune response. <a href="https://simple.wikipedia.org/wiki/Adaptive_immune_system" target="_blank">Adaptive immunity</a> represents the second line of defence and its special power resides in the ability to memorise different types of pathogens. Such “memory” activates a more potent immune response during the second encounter with the same pathogen. The so-called “memory cells” are the main players of this mechanism - which is the basis of vaccines. In order to test whether or not adaptive immunity is also involved in the onset of Alzheimer, Marsh and colleagues generated a mouse strain affected by Alzheimer, called Rag-5xfAD, lacking the key cells involved in adaptive immune response (natural killer (NK), B and T cells). When the researchers compared Rag-5xfAD with another mouse strain that equally shows Alzheimer but possesses all immune cells (WT-5xfAD), they observed an increment in the total amount of beta-amyloid plaques. Moreover, the researchers found that this increase in beta-amyloid plaques was due to a defect in their clearance instead of an increment in their production. Indeed, they showed that the absence of adaptive immune cells leads to an even more impaired ability of microglia in maintaining the level of beta-amyloid plaques low. Interestingly, Marsh and collaborators prove that this more pronounced inability of microglia was caused by the absence of immunoglobulin G (IgG) in the brain. IgG is a molecule, produced by B cells ( elements of the adaptive immunity), that recognizes foreign particles, dead cells and misfolded protein to help their removal from our organism. In this study, the researchers found that IgG helps microglia in the clearance of beta-amyloid plaques. In fact, the injection in the brain of non-specific IgG molecules efficiently reduces the number and the size of beta-amyloid plaques in Rag-5xfAD mice. Furthermore, the reacquisition of the adaptive immune cells, by bone marrow transplantation, improves beta-amyloid clearance in Rag-5xfAD mice.</p>\r\n\r\n<p>Marsh et al. added a new piece to the puzzle of the Alzheimer’s picture. The researchers proposed a role for the adaptive immune cells in the clearance of beta-amyloid, whose accumulation causes brain damage. These peripheral immune cells can produce IgG molecules, which then accumulate in the brain through a still unknown mechanism, and improve microglia activity. The use of IgG in the clinic to cure Alzheimer has already been tested in humans, with a reported decrease in number and size of beta-amyloid plaques, but without any improvement in cognitive functions, meaning that further studies are still needed to further understand the molecular mechanism behind this disease.</p>', 3, 'Marsh S, Abud E, Lakatos A, Karimzadeh A, Yeung S, Davtyan H, Fote G, Lau L, Weinger J, Lane T, Inlay M, Poon W, Blurton-Jones M. <a href="http://www.pnas.org/content/113/9/E1316.abstract" target="_blank">The adaptive immune system restrains Alzheimer&rsquo;s disease pathogenesis by modulating microglial function</a>. <em>Proceedings of the National Academy of Sciences</em>. 2016;113(9):E1316-E1325. doi:10.1073/pnas.1525466113.', 1, '2017-05-10 09:31:16', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk047'),
(118, 'Howler monkeys: living a life in colour helps finding better food', '<p>In terms of their ability to see colours, primates are unique compared to other mammals. Many primates have trichromatic colour vision and can see differences among red, orange, yellow, and green hues. What is particularly fascinating, however, is how much variation there is <em>among</em> primates in their ability to see colour. Take humans, for example. Compared to our closest relatives - African and Asian apes and monkeys - a sizeable number of humans are red-green "colour-blind". This trait is far more common among men, because the gene linked to perception of reddish hues is located on the X-chromosome (one of the so-called sex-chromosomes: men posses only one of them, whereas women have two - in the latter case a second spare gene is always available in case one it doesn''t function properly). Interestingly, variation among individuals is even greater among monkeys living in Mexico, Central and South America (Neotropics). Here, almost all of the day-active species possess many (4 - 8 or more) different types of colour vision in a single group of monkeys. All males and some females have a form of red-green colourblindness, while some females can tell apart reds from greens the way most humans can; this is true of all day-active species except for <a href="https://en.wikipedia.org/wiki/Mantled_howler" target="_blank">howler monkeys</a>. Due to a unique evolutionary event, all howler monkeys, male and female, possess a similar type of trichromatic vision and can tell apart a red fruit from a green leaf. </p>\r\n\r\n<p>Perhaps not surprisingly, fruit eating has been suggested as a principle driver of the evolution of colour vision in primates. Almost all primates eat fruit, and many ripe fruits eaten by primates are yellow, orange or red: hues that only a trichromat could see against green leaves. Yet, while howler monkeys do eat fruits when they are available, they are famous for eating huge amounts of leaves. The goal of our research was to test an idea proposed several years ago based on the study of African primates - that colour vision could also be good for finding new, immature leaves that are the most nutritious and easy to digest. </p>\r\n\r\n<p>We studied which trees howler monkeys ate from in Sector Santa Rosa, in northwestern Costa Rica. The primates at this field site have been studied for years, and the forest is fascinating because of how seasonal the rainfall, fruit production and patterns of leafing are. Using a field-portable equipment, we collected and measured the colour of leaves from the trees howler monkeys ate from. We also analyzed DNA from howler feces to examine the genes linked to colour vision at this site. As expected, we found that all the howler monkeys had trichromatic colour vision, although a few had slight variations from the others. We also found that for about half of the plant species we looked at, the young leaves were indeed reddish in colour. This included several plant species (for example, fig trees) that were seasonally very important in their diet when overall food availability was low. We then predicted how the leaves of different trees would look to the howler monkeys using models of their colour vision. We found that the reddish-coloured young leaves would be far more visible against mature green leaves to howler monkeys with trichromatic colour vision, than they would be to other colourblind monkeys that share the forests with them.</p>\r\n\r\n<p>Previous studies have found that trichromatic colour vision is useful to primates for finding and selecting fruit. We show that additionally it is useful to howler monkeys for finding young leaves. When there is little else to eat, howler monkeys can eat mature leaves; however, they prefer to eat fruits and young leaves when they are available. Therefore, their unique colour vision helps them with two important foraging tasks, which may both contribute to the evolution of their unique form of trichromacy. We also suggest that colourblind howler monkeys may have been weeded out by natural selection. Interestingly, colourblind people and smaller neotropical monkeys are better at breaking camouflage, which seems to help monkey hunt insects; red-green colourblindness may persist in these populations due to some foraging advantages. Unlike their neighboring monkeys, however, howlers do not hunt camouflaged prey to a large extent, so colourblind howlers may be at a strong disadvantage.</p>\r\n\r\n<p>In summary, we show that red-green colour vision is found in all howler monkeys, and it likely helps howlers to find preferred young, reddish leaves, which are an important part of their diet. Natural selection for trichromatic colour vision may be stronger for howler monkeys than it is for other primate species living in the same areas.</p>', 4, 'Melin A, Khetpal V, Matsushita Y, Zhou K, Campos F, Welker B, Kawamura S. <a href="https://doi.org/10.1002/ece3.2716" target="_blank">Howler monkey foraging ecology suggests convergent evolution of routine trichromacy as an adaptation for folivory</a>. <em>Ecology and Evolution</em>. 2017;7(5):1421-1434. doi:10.1002/ece3.2716.', 4, '2017-07-03 07:55:23', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk052'),
(111, 'Human gut parasite has a sinister use for its stolen genes', '<p>It is well established knowledge that bacteria routinely <a href="https://en.wikipedia.org/wiki/Horizontal_gene_transfer" target="_blank">exchange genes</a> between unrelated species, creating an extensive network of information flow independent of sexual reproduction. By acquiring new genes, each being a blueprint for a single protein, the bacteria gain also the functions the proteins perform within the cell. This phenomenon is forcing itself on our attention especially with the spread of antibiotic resistance among bacteria, which is threatening our ability to fight infections. But is swapping of genes important also in more complex organisms like animals, plants, or single-celled <a href="https://en.wikipedia.org/wiki/Protist" target="_blank">protists</a>? Their DNA might be better protected against change as it is enwrapped in membranes of the nucleus, but it is unclear if this barrier plays a significant role. Answers to this puzzle will have far-reaching implications both for evolutionary theories and practical understanding of diseases. The jury is still out, but new data from a human gut parasite add to the growing pile of evidence that free movement of genes is indeed happening across the whole tree of life.</p>\r\n\r\n<p>Roughly one billion people worldwide have their guts infected with a microscopic sphere-shaped protist called <a href="https://en.wikipedia.org/wiki/Blastocystis" target="_blank"><em>Blastocystis</em></a> which is distantly related to <a href="https://en.wikipedia.org/wiki/Kelp" target="_blank">kelp</a> and <a href="https://en.wikipedia.org/wiki/Oomycete" target="_blank">water molds</a>. The parasite often goes unnoticed, but in some individuals, it manifests with severe digestive malfunctions reminiscent of the <a href="https://en.wikipedia.org/wiki/Irritable_bowel_syndrome" target="_blank">irritable bowel syndrome</a>. A collaboration of Canadian scientists has now thoroughly sequenced and assembled the complete genetic information of <em>Blastocystis</em> and searched it for genes potentially stolen from other organisms, and identified 167 genes which clearly represent recent gains. Most of these genes came from bacteria, often from groups coinhabiting guts of humans and other animals together with <em>Blastocystis</em>, and about 20 originated from more advanced organisms, possibly past hosts of the parasite.</p>\r\n\r\n<p><em>Blastocystis</em> did not gain and retain the genes randomly. The researchers say that most of the genes they identified may provide a direct advantage for the parasitic lifestyle of <em>Blastocystis</em>. Some of them are likely involved in scavenging and utilizing nutrients which the parasite finds in the gut environment. Others may serve in evading recognition by the immune system or surviving its attacks. The newly identified genes may therefore be important for the parasite''s pathogenesis, in other words, the mechanisms leading to disease. Still a lot of work has to be done before we can talk about practical utilization of this knowledge, but now we know the path. The genes which <em>Blastocystis</em> stole and incorporated into its own arsenal may one day provide targets for newly developed drugs, becoming an Achilles'' heel of the parasite. The most promising targets are the genes which are also shared with other, unrelated, parasitic protists, since these are evidently crucial for parasitism. By targeting them, the drugs can damage the parasite''s interactions with host cells and damper its ability to invade. These genes may also be safer targets because they came from organisms even more evolutionary distant from humans than the parasite itself, which means a lower risk of side effects for the patient.</p>\r\n\r\n<p>Regardless of the future therapeutic applications, the study of <em>Blastocystis</em> and its stolen genes has already provided decisive insights into the mechanisms of biological evolution. It is becoming more and more difficult to ignore the importance of gene transfer into, and between, complex organisms endowed with a nucleus. Stolen genes clearly can benefit the recipients and help them to colonize new environments, including the environments inside a host in the case of parasites.</p>', 3, 'Eme L, Gentekaki E, Curtis B, Archibald J, Roger A. <a href="http://www.sciencedirect.com/science/article/pii/S0960982217301380" target="_blank">Lateral Gene Transfer in the Adaptation of the Anaerobic Parasite Blastocystis to the Gut</a>. <em>Current Biology</em>. 2017;27(6):807-820. doi:10.1016/j.cub.2017.02.003.', 4, '2017-05-18 15:23:15', NULL, NULL, 2, 'https://doi.org/10.25250/thescbr.brk048'),
(112, 'Toxic brain cells are a new target for treating neurodegeneration', '<p>After decades of research into the causes of <a href="https://en.wikipedia.org/wiki/Neurodegeneration" target="_blank">neurodegenerative diseases</a>, there is still no cure. Instead of focusing on the neurons that die in these diseases, they may be treatable by blocking the metamorphosis of helpful support cells into toxic support cells, or by pharmaceutically countering the neuron-killing toxin these harmful cells secrete.</p>\r\n\r\n<p><a href="https://simple.wikipedia.org/wiki/Astrocyte" target="_blank">Astrocytes</a>, a type of <a href="https://simple.wikipedia.org/wiki/Glial_cell" target="_blank">glial cell</a>, are the most abundant cell in the brain. Though we have known about them for over 100 years, very little was understood about their function both in normal development, and in response to injuries and in neurodegeneration. Key advances in the past decade have shown that they strongly promote the connections between neurons (called synapses), as well as provide nutrients and remove waste byproducts to help keep these neurons healthy and functioning normally. But what happens when things go wrong?</p>\r\n\r\n<p>Astrocytes can rapidly transform into two main reactive states as a response to injury and disease. One state is a scar-forming or "A2" reactive astrocyte, which corrals the lesion to stop expansion of the injury. They are considered to be beneficial. But the second, called "A1" reactive astrocytes, release a toxin that kills neurons - further exacerbating the injury. We showed that A1 astrocytes are abundant in various human neurodegenerative diseases like Alzheimer''s, Parkinson''s, and Huntington''s diseases, amyotrophic lateral sclerosis (ALS, or Lou Gehrig''s disease), and multiple sclerosis. What is most amazing about this presence in human neurodegeneration is that these A1 toxic astrocytes are present exactly in regions where the neurodegeneration is most prominent, and not elsewhere in the brain - strongly suggesting that A1 astrocytes contribute to the death of neurons in neurodegenerative disorders. But how do non-reactive astrocytes turn into nasty reactive A1s?</p>\r\n\r\n<p>Another type of glial cells, microglia, so called because they are incredibly small, are the resident immune cells of the brain (<em>read also the Break:</em> <a href="https://www.thesciencebreaker.com/break.php?breakId=110" target="_blank">The power of our adaptive immunity against Alzheimer''s Disease</a>). These cells are primed to respond rapidly to a wide array of insults to the brain: infection, trauma, disease, and many others. We have been studying the interactions between microglia and astrocytes for some time, and have recently discovered a cocktail of signals that microglia secrete in response to these injuries. This cocktail released by microglia into their environment is both required and sufficient to turn non-reactive astrocytes into reactive A1s.</p>\r\n\r\n<p>Once we knew how A1 reactive astrocytes are made in the brain, we went on to study what they do. We knew that non-reactive astrocytes can promote the formation of neuron-neuron connections (synapses) that allow the propagation of electrical signals from neuron to neuron. In a healthy brain, these synapses are constantly formed - a process that is needed to form new memories for example (<em>read also the Break</em>: <a href="https://www.thesciencebreaker.com/break.php?breakId=41" target="_blank">Where is the Engram</a>?). First, we found that A1 cells lack the ability to promote the formation of synapses.</p>\r\n\r\n<p>Another important role of astrocytes is their capacity to provide nutritional support to neurons and keep them healthy. Although A1 astrocytes still appear to provide this support to neurons, we found that they also secrete a toxic factor that can kill many different types of neurons. After we knew that it is only the A1 reactive astrocyte and no other cells in the brain that can kill all these neurons, we tried blocking the formation of A1 astrocytes in an injury model where we knew that neurons would normally die. We found that without the presence of A1 astrocytes neurons did not die.</p>\r\n\r\n<p>In neurodegenerative diseases, many neurons die over time, and the ones that don''t die show reduced formation of synapses. These are precisely the changes that A1 reactive astrocytes bring about. And we have found A1 reactive astrocytes suspiciously in all the right places in human neurodegenerative disease. We are now working to see if A1s possibly drive neurodegenerative diseases, and hope to provide opportunities for the development of new treatments for diseases in which our brain cells die.</p>', 3.5, 'Liddelow S, Guttenplan K, Clarke L, Bennett F, Bohlen C, Schirmer L, Bennett M, M&uuml;nch A, Chung W, Peterson T, Wilton D, Frouin A, Napier B, Panicker N, Kumar M, Buckwalter M, Rowitch D, Dawson V, Dawson T, Stevens B, Barres B. <a href="http://www.nature.com/nature/journal/v541/n7638/full/nature21029.html" target="_blank">Neurotoxic reactive astrocytes are induced by activated microglia</a>. <em>Nature</em>. 2017;541(7638):481-487. doi:10.1038/nature21029.', 1, '2017-05-24 10:03:33', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk049'),
(113, 'The cutting EDGE: Bringing genomics to everyone', '<p>Diabetes, infertility, cancer, allergies, Alzheimer''s disease - the key to one day preventing or even curing such afflictions and diseases (both infectious and genetically driven) may be locked in our own genetic code and the code of <a href="https://simple.wikipedia.org/wiki/Human_microbiome" target="_blank">microorganisms that inhabit our bodies</a>. Genomics, the study of this DNA code, has recently become much more promising as a result of two things: (1) vast improvements in quality and quantity of <a href="https://en.wikipedia.org/wiki/DNA_sequencing" target="_blank">DNA-sequencing</a>, and (2) an exponential decrease in the cost of such DNA-sequencing. For example, what originally cost $3 billion for the <a href="https://simple.wikipedia.org/wiki/Human_genome" target="_blank">human genome sequence</a> is now less than $1000 to resequence it. However, two key problems cause a significant bottleneck for data interpretation: (1) the sheer volume of genomic data and (2) the shortage of expertise necessary to understand and act it.</p>\r\n\r\n<p>To help address both problems, we have created <a href="https://edge.readthedocs.io/en/v1.5/index.html" target="_blank">EDGE (Empowering the Development of Genomics Expertise) Bioinformatics</a>. EDGE includes a variety of innovative and integrated bioinformatic tools, housed within a user-friendly, web-based platform that requires little input and is mostly point-and-click. Preconfigured workflows can be quickly and easily selected to address a wide variety of goals and projects, including analyzing a genome or a much more complex metagenomic (microbiome) or clinical sample.</p>\r\n\r\n<p>While the human genome contains approximately 6 billion nucleotides ("letters" of DNA in each human cell), it has been estimated that the human body also harbors at least as many bacterial cells as human cells. Analyzing a microbial community or a microbiome in a clinical sample may reveal thousands of different bacterial and viral species, residing in a complex mixture with human cells. Taking into account the microbes inhabiting the human body, researchers have estimated that the gene content of the typical microbiome is over 300 times larger than the human genome. Characterizing the volume of data from such a sample is a daunting challenge.</p>\r\n\r\n<p>Realistically, few biologists have the computational expertise or resources to accomplish such types of analyses, thus requiring the aid of experts to develop custom computer code to process genomic data. EDGE Bioinformatics holds promise to revolutionize the way scientists analyze genomic data by making sophisticated tools available via intuitive and easy-to-use web-based interfaces. Using a few mouse clicks, EDGE receives raw data from sequencers and creates reports and graphics based on the data - usually in minutes or hours instead of days or weeks of wrangling with sequence data. EDGE Bioinformatics truly makes it possible for nearly <em>any</em> biologist or physician with access to genomic data to use this technology and to derive meaning from it. The software is entirely open source, and can be downloaded at no cost and used locally or remotely via the web.</p>\r\n\r\n<p>With EDGE on a local computer server or in the cloud, it is now possible to bring the power of complex, sequencing data analytics to smaller facilities, including clinics, hospitals, and university laboratories. Scientists are beginning to use genomics to assess human genome mutations contributing to cancer, and human microbiome shifts associated with Crohn''s disease, irritable bowel syndrome, allergies, Alzheimer''s disease, and countless other areas of research, like algal biofuels and climate effects. In the field of infectious disease, using sequencing data and EDGE, it will be possible in the future for a nurse to swab your saliva and later tell you if your symptoms are more likely caused by a viral or bacterial infection. This could reduce the unnecessary and problematic use of antibiotics (read also the Break: <a href="https://www.thesciencebreaker.com/break.php?breakId=56" target="_blank">Collateral damage: antibiotics disrupt the balance in the gut</a>). It is also envisioned that hospitals could use EDGE to ascertain which pathogens inhabit the hospital environment, implement countermeasures, and improve infection-control procedures.</p>\r\n\r\n<p>Important societal issues such as climate change and eradicating the spread of disease require an in-depth understanding of how microorganisms function (read also the Break: <a href="https://www.thesciencebreaker.com/break.php?breakId=62" target="_blank">Cold adaptation: gut bacteria can make the difference</a>). The key to that understanding is locked in the genetic code. Sequencing technology has become readily available to tackle such scientific challenges, and EDGE now makes the tools for understanding genomic data readily available as well.</p>', 3.5, 'Li P, Lo C, Anderson J, Davenport K, Bishop-Lilly K, Xu Y, Ahmed S, Feng S, Mokashi V, Chain P. <a href="https://doi.org/10.1093/nar/gkw1027" target="_blank">Enabling the democratization of the genomics revolution with a fully integrated web-based bioinformatics platform</a>. <em>Nucleic Acids Research</em>. 2016;45(1):67-80. doi:10.1093/nar/gkw1027.', 6, '2017-06-14 09:12:30', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk050'),
(114, 'A Weekend Camping is Just What the Doctor Ordered', '<p>The invention of electrical lighting has permitted work and social activities to continue beyond sunlight, however it has also caused an unnatural desynchrony between human''s biology and the environment. The body''s internal timekeeping system (also called <a href="https://simple.wikipedia.org/wiki/Circadian_rhythm" target="_blank">circadian clock</a>) is designed to predict environmental time in order to appropriately coordinate behaviors, such as sleeping and eating, with the solar night and day. The fine-tuning of such delicate timing is made possible thanks to our ability to perceive the quality (<em>i.e.</em> colors) and intensity of the surrounding light. However, with the invention of electrical light, the sun is not the only light source providing time-of-day information to the internal clock. Specifically, exposure to electrical light at night - whether from a lamp, smartphone, or television - serves as a time cue informing the body''s internal clock that it is still daytime. As a result, our internal clock and sleep timing is shifted later relative to the natural light/dark cycle. While this may be beneficial when attempting to meet a deadline, cramming for an exam, or socializing, it can make it difficult to awaken early for work or school the following morning. Furthermore, this desynchrony between the internal clock and environmental light/dark cycle has been shown to be associated with negative consequences such as reduced school performance and increased risk of accidents, mood disorders, diabetes, and obesity. Therefore, a remedy to realign the internal clock with environmental time is relevant for public health and well-being.</p>\r\n\r\n<p>We have previously shown that going camping for one week during the summer can help re-synchronize the internal circadian clock to solar time (Wright et al., <em>Current Biology</em>, 2013). Specifically, one week of exposure to only natural sunlight shifts the timing of the internal clock earlier such that the biological night begins near sunset and ends near sunrise. However, the natural light/dark cycle changes with seasons such that the environmental nighttime is much longer during the winter. Therefore, in the current study, we sought to determine how the circadian clock responds to seasonal changes in the natural light/dark cycle.</p>\r\n\r\n<p>We took one group of participants camping for six days near the winter solstice in the Rocky Mountains of Colorado with exposure to only natural light (i.e., sunlight, moonlight, and campfires; no flashlights, no personal electronic devices, etc.). We found that after exposure to the winter natural light/dark cycle, the start of the biological night and start of sleep were timed earlier than in the modern environment with exposure to electrical light. Additionally, the biological night was expanded after camping in the winter compared to the summer suggesting a substantial seasonal circadian response to the natural light/dark cycle. Furthermore, participants were exposed to ~13x higher light levels during waking hours while camping compared to the modern environment.</p>\r\n\r\n<p>We were then curious as to how rapidly this re-synchronization could occur. Therefore, we took participants camping for one weekend in the summer to examine whether this shorter duration of exposure to natural light would be sufficient to re-synchronize the clock to solar time. We found that a weekend camping did indeed shift the internal clock earlier. Specifically, a weekend camping trip was sufficient to achieve ~69% of the shift in internal clock observed after a week camping.</p>\r\n\r\n<p>These findings indicate that our internal clocks are timed later in the modern environment with electrical lighting, both in the summer and winter. This late timing often makes it difficult for people to fall asleep at night, yet they are still required to wake up early for work and school obligations. As a result, many people obtain less sleep during the workweek and attempt to catch-up on lost sleep during the weekend. This weekend catch-up sleep often causes individuals to miss morning exposure to light, thus even further delaying their internal clocks. As a result, the desynchrony between the internal clock and the environment is often exacerbated on Monday mornings. We suggest that a weekend spent camping can help to attenuate this delay and resynchronize the clock to the natural environment, making it easier to wake up for work and school. Furthermore, we found that, likely due to spending more time indoors, people are exposed to less light during the daytime in the modern environment. Therefore, we recommend that increasing exposure to sunlight by taking a morning walk or eating lunch outside can help to facilitate communication between the sun and the internal clock and promote synchrony with the environment.</p>', 4, 'Stothard E, McHill A, Depner C, Birks B, Moehlman T, Ritchie H, Guzzetti J, Chinoy E, LeBourgeois M, Axelsson J, Wright K. <a href="https://doi.org/10.1016/j.cub.2016.12.041" target="_blank">Circadian Entrainment to the Natural Light-Dark Cycle across Seasons and the Weekend</a>. <em>Current Biology</em>. 2017;27(4):508-513. doi:10.1016/j.cub.2016.12.041.', 1, '2017-06-21 08:25:03', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk051');
INSERT INTO `breaks` (`id`, `title`, `content`, `reading_time`, `original_article`, `subject_id`, `date_created`, `author1_id`, `author2_id`, `editor_id`, `imzy_link`) VALUES
(119, 'Driving down malaria', '<p>Mosquitoes are the deadliest animals on earth, having <a href="https://www.gatesnotes.com/Health/Most-Lethal-Animal-Mosquito-Week" target="_blank">killed more people than wars and plagues combined.</a> This is because they spread debilitating diseases like malaria - which affects more than 200 million people each year. Despite a momentous effort to combat the disease over the past two decades, we are unlikely to completely eliminate malaria without new interventions. Scientists have now turned to the tools of <a href="https://simple.wikipedia.org/wiki/Genetic_engineering" target="_blank">genetic engineering</a> for new solutions.</p>\r\n\r\n<p><a href="http://www.who.int/mediacentre/factsheets/fs094/en/" target="_blank">Malaria</a> is caused by a parasite, <em><a href="https://simple.wikipedia.org/wiki/Plasmodium" target="_blank">Plasmodium falciparum</a>,</em> that needs to develop in both a human and mosquito host to survive. If we make it impossible for the parasite to develop in the mosquito, then we could stop malaria dead in its tracks. Recent advances in genetic engineering have made this possible by introducing new traits into the mosquito that affect its own survival, or its ability to carry the parasite. The problem is no longer the process of engineering, but how to spread this modification throughout mosquitoes in the wild.</p>\r\n\r\n<p>To explain the problem, we need to consider what happens to a genetic modification over time. Any modification that we make to a mosquito chromosome is inherited like any other gene. Just like us, mosquitoes contain two copies of every chromosome and each of their offspring inherit just one of the two copies. If a few modified mosquitoes were released into the wild, that modification would not spread because it only ever gets inherited by half of the offspring.</p>\r\n\r\n<p>In 2003, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1691325/" target="_blank">Professor Austin Burt postulated an exciting method</a> to overcome this problem using something called a "gene drive". Gene drives are genetic modifications that can spread through an entire population by biasing their own inheritance. This means that such modifications will over-ride the classical <a href="https://simple.wikipedia.org/wiki/Mendelian_inheritance" target="_blank">Mendelian inheritance laws</a>, with a mechanism also called "Super-Mendelian". By increasing their chances of being passed on, they ensure that more and more mosquitoes will have gained the modification over time.</p>\r\n\r\n<p>Drawing upon Burt''s original theory, we used a pair of molecular scissors called "<a href="http://www.yourgenome.org/facts/what-is-crispr-cas9" target="_blank">CRISPR</a>" to engineer gene drives that work through a mechanism of copy and paste. A gene drive made with CRISPR was inserted into a specific region within one chromosome of the mosquito "<em><a href="https://simple.wikipedia.org/wiki/Anopheles" target="_blank">Anophels gambiae</a>"</em> and programmed to cut its unmodified partner chromosome. When the broken chromosome is damaged it repairs as precisely as possible by using the modified partner chromosome as a template, but in doing so it unwittingly copies the gene drive inside. In this way, it doesn''t matter which of the two chromosome copies are inherited because the gene drive is present on both (see figure "<strong><a href="http://www.openwetware.org/images/4/4e/How_gene_drives_spread.png" target="_blank">How gene drives spread</a>"</strong>). Amazingly, we saw that our gene drive mosquitoes passed on the modification to more than 99% of their offspring!</p>\r\n\r\n<p>In our lab, we designed three different gene drives that would make mosquitoes infertile. This sounds weird, because it shouldn''t be possible to spread the gene drive if the mosquitoes are unable to produce offspring? We achieved this by programming our gene drives to affect only female mosquitoes and only when both of their parents were modified. Thus, the gene drive has no detrimental effect when only one of their parents is a carrier, allowing it to spread when gene drive mosquitoes mate to those in the wild (see figure "<strong><a href="http://www.openwetware.org/images/7/78/Using_suppression_gene_drives_to_eliminate_malaria.png" target="_blank">Using gene drives to eliminate malaria</a>"</strong>). To test this hypothesis, mosquitoes carrying one of the three gene drive were released into a cage of unmodified mosquitoes and their reproduction was tracked over time. More and more mosquitoes were found to carry the modification with each generation, <a href="http://www.nature.com/nbt/journal/v34/n1/full/nbt.3439.html" target="_blank">showing for the first time</a> that a gene drive can be designed to invade natural population of dangerous insects.</p>\r\n\r\n<p>If mosquitoes carrying such gene drive are released into the wild, it is expected that they could spread throughout an entire population within just 20-30 mosquito generations. Because mosquitoes reproduce very quickly, this could take just a couple of years starting from a relatively small release. As the gene drive gets into more and more mosquitoes, mating between two gene drive carriers will become more frequent, and result in the generation of female mosquitoes that are infertile. This will lead to a <a href="https://www.safeopedia.com/definition/2884/population-crash" target="_blank">population crash</a> and potentially reduce the number of biting mosquitoes below the critical threshold needed to support continued malaria transmission.</p>\r\n\r\n<p>A critical problem for malaria control is delivering effective interventions to those who need it most. With gene drive, the mosquitoes do the hard work. This self-sustaining, species-specific and long-term intervention could be used to boost current control strategies like bed-nets, insecticides and vaccines. Gene drives may be the last intervention needed to tip the fight against malaria in our favour - we want to make malaria a disease of the past.</p>', 4, 'Hammond A, Galizi R, Kyrou K, Simoni A, Siniscalchi C, Katsanos D, Gribble M, Baker D, Marois E, Russell S, Burt A, Windbichler N, Crisanti A, Nolan T. <a href="https://doi.org/10.1038/nbt.3439" target="_blank">A CRISPR-Cas9 gene drive system targeting female reproduction in the malaria mosquito vector Anopheles gambiae</a>. <em>Nature Biotechnology</em>. 2015;34(1):78-83. doi:10.1038/nbt.3439', 1, '2017-07-18 09:12:46', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk053'),
(120, 'Hips don’t lie: attraction revealed by dancing body language', '<p>Dancing forms a huge part of human social life, and humans often get together to dance when romance is in the air but what makes a good dancer?</p>\r\n\r\n<p>Human courtship is complex, yet we currently know a lot about the physical factors involved in one person deciding that they are attracted to another person. Certain physical characteristics, facial features, and body size and shape have all been extensively investigated, and they each provide key information about the health, and reproductive qualities of the person being viewed.</p>\r\n\r\n<p>Movement has been less well-studied, mainly because it is difficult to disentangle other characteristics such as facial attractiveness, and body size and shape when asking people to judge the movements of others. Our team have pioneered a technique which enables us to record movements such as walking and dancing and convert those video clips into featureless avatars which convey the movements exactly, but do not portray any other identifying features.</p>\r\n\r\n<p>We then show clips of the avatars walking and dancing and ask males and females to rate them on the characteristics we are interested in, e.g. "how good a dancer is this person?". We record the movements using a hi-tech motion capture system that also enables us to take precise measurements of individual movement parameters in three-dimensions. We can then see which patterns of movements are associated with perceptions of dance quality.</p>\r\n\r\n<p>In previous studies we found that good male dancing is signified by larger and more variable movements of the upper body, we then went on to show that such movements provide an honest indication of the physical strength of the male - stronger males are viewed as better dancers, and interestingly this information appears to be directed more at potential male rivals than it does at potential female partners.</p>\r\n\r\n<p>This now brings us to female dance, it is unlikely that female dance quality is based on upper body strength, in fact anecdotal evidence suggests that female movements are viewed in relation to the hips. In our study just published in the journal Scientific Reports, we recruited 39 females (none were professional dancers) and asked them to dance to a basic drum beat rhythm. We converted their movements into avatars and showed these to males and females, who were asked to rate teach dancer in terms of how good a dancer they were. We found that three movement components were key predictors of dance quality: (i) Greater hip swing (ii) Greater arm movements, that also displayed moderate asymmetry (iii) More asymmetric thigh movements.</p>\r\n\r\n<p>In addition, the rhythmicity of the dance was important, better rated dancers strutted their stuff more closely to the beat of the music.</p>\r\n\r\n<p>These findings might reveal the biological significance of movements in human mate preferences. For example, moving one''s limbs independently with a degree of asymmetry can be difficult, especially when dancing to music, and so this might show to an observer that the dancer has high quality motor control, damage to the nervous system typically results in abnormalities in the movement system and so this is a good way of signaling your neurological health. Hip swing and thigh movements may serve to indicate the degree of femininity, in turn these might also signal your hormonal status. Consistently with that, other studies have shown that female walking patterns change over the menstrual cycle, with females in a more reproductive phase (ovulation) walking in a more "feminine" and "sexy" manner.</p>\r\n\r\n<p>We have taken the first steps in the scientific understanding of what dance movements are seen as optimal, and what these movements might signal to same-sex rivals and opposite sex partners. We can now move forward to try and understand the functional significance of these movements - i.e. what do they tell us about our biological and psychological make-up. On top of that...we will also be able to provide sound advice to enable people to improve their dancing.</p>', 3.5, 'McCarty K, Darwin H, Cornelissen P, Saxton T, Tov&eacute;e M, Caplan N, Neave N. <a href="https://doi.org/10.1038/srep42435" target="_blank">Optimal asymmetry and other motion parameters that characterise high-quality female dance. <em>Scientific Reports</em></a>. 2017;7:42435. doi:10.1038/srep42435.', 1, '2017-07-27 08:44:49', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk054'),
(121, 'Guiding plant growth electronically', '<p>In the not too distant future computers may be used to directly monitor and control the growth of plants. For a moment, consider the prevalence of sophisticated electronic medical technologies already in use today. Patients now regularly receive <a href="https://en.wikipedia.org/wiki/Retinal_implant" target="_blank">retinal</a> and <a href="https://simple.wikipedia.org/wiki/Cochlear_implant" target="_blank">cochlear</a> implants to restore vision and hearing. Meanwhile, those suffering from diabetes or heart disease mitigate their conditions using electronic devices that provide continuous real time feedback and regulation of <a href="https://en.wikipedia.org/wiki/Insulin_pump" target="_blank">insulin levels</a> and <a href="https://en.wikipedia.org/wiki/Artificial_cardiac_pacemaker" target="_blank">heart rhythm</a>.</p>\r\n\r\n<p>The recent development of new <a href="https://en.wikipedia.org/wiki/Organic_electronics" target="_blank">organic electronic materials</a> could make the difference in this promising and rapidly expanding field of biomedical technology. Based on special classes of <a href="https://simple.wikipedia.org/wiki/Polymer" target="_blank">polymers</a>, organic electronic materials and devices have demonstrated their capacity for mediating precise interactions with both the electronic and chemical signals of living systems.</p>\r\n\r\n<p>Soft and flexible biocompatible medical implants and wearable technologies can be made possible by incorporating such polymer-based electronics. Indeed, the development of organic bioelectronics has largely been driven by the vision of technology that integrates seamlessly, on and in our bodies, to digitally interface with the fundamental electrical and chemical signals of our cells, tissues, and organs. Might the capabilities of such materials and medical therapeutics have overlapping applications for plants?</p>\r\n\r\n<p>The regulation of chemical signals within and between cells is not a unique feature of humans. Rather, chemical signaling is a ubiquitous and defining characteristic of life. Plants have evolved highly complex and sophisticated chemical signaling systems that allow them to coordinate their growth, monitor their surroundings, and react appropriately to dynamic environmental conditions. Thus, based on the capabilities of bioelectronics to interface chemically and electrically - in a distributed fashion and at the cellular scale - it seems likely that there will be a wide range of potentially important botanical, agricultural, and arboriculture applications for such technologies.</p>\r\n\r\n<p>To begin exploring this possibility, in a collaborative and multidisciplinary effort, materials scientists and device engineers at the <a href="http://www.liu.se/loe" target="_blank">Laboratory of Organic Electronics</a> at Linköping University teamed up with plant biologists at the <a href="http://www.upsc.se/" target="_blank">Umeå Plant Science Center</a> to adapt organic electronic devices suitable for plants. The combined expertise of these two leading Swedish research institutions led to the demonstration of the first organic electronic devices - and one of the first electronic devices of any kind - capable of interacting with the <a href="https://simple.wikipedia.org/wiki/Plant_hormone" target="_blank">hormonal signaling networks within plants</a>.</p>\r\n\r\n<p>For organic electronic devices to fully speak in the signaling language of plants, the underlying materials must have the capability to transport and deliver plant hormone signaling compounds with great precision. To facilitate the delivery of such compounds, we first developed an entirely new material system based on branching tree-like polymers known as dendrites. Controlling the density and spacing of the dendrite network provides a means for enabling electronically mediated transport and release of plant hormones.</p>\r\n\r\n<p>Utilizing organic electronic delivery devices based on this "dendrolyte" system, we then demonstrated targeted delivery of <a href="https://en.wikipedia.org/wiki/Auxin" target="_blank">auxin</a>, perhaps the most well-known and most important plant hormone, to the roots of living plants. Specifically, <em><a href="https://en.wikipedia.org/wiki/Arabidopsis_thaliana" target="_blank">Arabidopsis thaliana</a></em> seedlings genetically modified to fluoresce in the presence of increased auxin levels enabled us to confirm and monitor hormone delivery. Delivering auxin in this manner was also shown to induce rapid changes in physiology. Among other roles, it is known that specific concentrations of auxin in the root tips have a direct influence on growth and elongation. Thus, by precisely controlling the location and amount of auxin delivered, we demonstrated that this technology could influence a living plant''s physiological function.</p>\r\n\r\n<p>This study was only the first step in adapting organic electronic devices for plants. In the near term, we plan to improve our technology and to conduct more sophisticated experiments to better understand the underlying mechanisms of plant growth and development. However, we see these results as a precursor to more capable organic electronic technologies to manipulate a plant''s signaling systems to govern important biologic processes such as flowering, growth, resistance, and overall physiology.</p>\r\n\r\n<p>There is still much to be learned about how plants function and how to influence and interact with their chemical signaling systems. However, we anticipate a not-to-distant future where bioelectronic-based technologies control maturation in orchards and vineyards, tailor the shape and sizes of trees or monitor the health and stresses of entire forests. We even imagine more exotic scenarios, such as the unique challenges involved in non-terrestrial applications. For example, fine-tuned control of plant growth may provide a means to counteract the low gravity environment of space.</p>', 4, 'Poxson D, Karady M, Gabrielsson R, Alkattan A, Gustavsson A, Doyle S, Robert S, Ljung K, Grebe M, Simon D, Berggren M. <a href="https://doi.org/10.1073/pnas.1617758114" target="_blank">Regulating plant physiology with organic electronics</a>. <em>Proceedings of the National Academy of Sciences</em>. 2017;114(18):4597-4602. doi:10.1073/pnas.1617758114', 5, '2017-08-09 09:27:29', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk055'),
(122, 'The mutation that allowed our brain to grow', '<p>During human evolution, one of the most remarkable events was the expansion of the upper layer of the brain: the so-called neocortex. This event took place about 2 million years ago and allowed us to develop the cognitive abilities that characterize modern day humans. In a recent paper published in the scientific journal <em>Science, </em>the geneticist Marta Florio and her colleagues pinned down a single genetic event that could help explain this sudden expansion of the neocortex. They identified a gene, called ARHGAP11B, which is present in our DNA but also in that of most humanoid species like, for example, the chimpanzee. This gene affects the generation and division of brain cells in a specific part of the neocortex. In this part of the brain there is sufficient room for cell growth, which means that it can support rapid brain expansion.</p>\r\n\r\n<p>DNA and genes are sequences of molecules called nucleotides. There are four different nucleotides, which are represented by the letters A, C, T, and G -and specific combinations of these nucleotides encode for specific genes. Genes are the recipes for proteins, and proteins, in turn perform most of the essential tasks in a cell, and indeed, in the whole body. The gene described in this study, <em>ARHGAP11B, </em>is actually a duplicate of another gene that is more widespread in the tree of life: <em>ARHGAP11A. </em>This gene codes for a protein that is important for the communication between cells. Because this is a basic and essential cellular function, the gene is almost identical in other organisms, a phenomenon called <a href="http://proteopedia.org/wiki/index.php/Introduction_to_Evolutionary_Conservation" target="_blank" rel="noopener">evolutionary conservation</a>.</p>\r\n\r\n<p>As mentioned above, genes are sequences of nucleotides. The sequences that encode a gene, however, are not continuous: there are pieces of non-important sequence in between the important pieces. It is like reading an interesting article in a magazine that places advertisements every other page. The chunks of sequence in a gene that encode for a protein are called "<u>ex</u>ons", because they are important for the <u>ex</u>pression of the protein, and the non-important pieces in between them are called <u>in</u>trons. The "original" <em>ARHGAP11A</em> gene consists of 12 exons, the parts that will be translated into the protein, but some of these were lost during the course of evolution in the "copy" gene, <em>ARHGAP11B</em>. To be more precise, our current <em>ARHGAP11B </em>gene is only read until the end of exon 6, in contrast to the end of exon 12 in the original gene.</p>\r\n\r\n<p>By looking at the DNA sequence of both <em>ARHGAP11A </em>and<em> -B </em>it is clear that the sequence is almost identical, especially in the protein-coding exons. One exception is a single nucleotide ("letter") in exon 5: a C that is substituted for a G. At first sight this seems like a minor change, but in fact this single nucleotide completely changes the way the protein is made<em>.</em> In fact, the structure of a big part of the protein now becomes very different and unique to humans. An earlier study already determined that this new version of the protein changes its original function and that this particular change gives brain cells in the neocortex the opportunity to grow and divide at an unprecedented level.</p>\r\n\r\n<p>Based on sequence information available for closely related non-human primates, Marta Florio and her team could show that, in all other primates, the <em>ARHGAP11B</em> gene lacks the conversion from C to G in exon 5, confirming it to be a human-specific event. The next thing to do was to test the effects of the different versions of the gene, encoding slightly different proteins, on animal development. The different varieties of the <em>ARHGAP11B</em> gene, one with and one without the C to G conversion, were introduced into mouse embryos, and their brain development was closely examined. As expected, more dividing cells were observed in the neocortex of mice that had received the human-specific <em>ARHGAP11B </em>gene.</p>\r\n\r\n<p>From all this, the conclusion is that a seemingly simple conversion from C to G in exon 5 of <em>ARHGAP11B</em> has led to a human-specific protein structure that changed its function. While it lost its original function, it gained an important role in allowing specific brain cells to grow and divide allowing the expansion of the neocortex. This is regarded as one of the key steps in the development of our current cognitive abilities.</p>\r\n\r\n<p>The molecular mechanisms by which the protein supports this function are still unknown, so this will undoubtedly be a topic of future research. Also, it is unlikely that this protein is solely responsible for the expansion of our neocortex. It is probably one of multiple (yet unidentified) key players, but this shouldn''t obscure the importance of the present study. It shows us how a minor genetic change has had widespread effects even altering the course of human evolution.</p>', 4, 'Florio M, Namba T, Pa''a''bo S, Hiller M, Huttner W. <a href="https://doi.org/10.1126/sciadv.1601941" target="_blank" rel="noopener">A single splice site mutation in human-specific ARHGAP11B causes basal progenitor amplification</a>. <em>Science Advances</em>. 2016;2(12):e1601941-e1601941. doi:10.1126/sciadv.1601941.', 4, '2017-08-24 10:54:01', NULL, NULL, 2, 'https://doi.org/10.25250/thescbr.brk056'),
(123, 'How environment shapes…your nose', '<p>An important function of the human nose is to warm and humidify the air that we breathe in. This is important so as to prevent the inner respiratory tract from drying up, which can lead to infections. Because the shape of the nose and the geometry of the inner airways are tied to the efficiency of the air-conditioning process, it is thought that these traits might have been shaped in part by <a href="https://simple.wikipedia.org/wiki/Natural_selection" target="_blank" rel="noopener">natural selection</a>. This could help explain some of the variation in nose shapes and sizes across human populations living in different climates. Indeed, several studies looking at variation in the shape of the nasal aperture and internal nasal cavity in human skulls across different environments have shown that this might be true. For example, the nasal aperture in the human skull tends to be larger, on average, in warmer climates and narrower in cooler climates.</p>\r\n\r\n<p>There is another evolutionary reason, besides natural selection, that can explain why people from across the globe have widely varying nose shapes: genetic drift. <a href="https://simple.wikipedia.org/wiki/Genetic_drift" target="_blank" rel="noopener">Genetic drift</a> is the name given to the accumulation of genetic differences among isolated populations. Because of the link between genotype (the genetic background of every individual) and phenotype (the physical manifestation of the genotype), genetic drift also leads to physical differences between populations. In fact, we know, based on DNA sequence variation in the human genome and our knowledge of human demographic history, that genetic drift has had a large influence on human evolution. Because of this, we asked: Could genetic drift alone explain the differences in nose shape among human populations? We compared the variation in seven measurements of nose shape both within and among four groups: West Africans, East Asians, Northern Europeans, and South Asians. Out of the seven traits we looked at, we found that the differences in the width of the nose and nostrils among the four populations cannot be explained by genetic drift alone. Varying natural selection pressures could have driven these traits to look different in geographically distant populations.</p>\r\n\r\n<p>Then the question became: Could climate be the selection pressure under question? To answer this, we evaluated how closely, if at all, nose width/nostril width variation followed the geospatial variation in temperature and humidity. We found that nostril width is positively correlated with both temperature and humidity, to the effect that wider nostrils are more common in warmer climates and narrow nostrils are more common in cooler climates. This trend is similar to previous findings that the width of the nasal aperture (in the skull) is positively correlated with temperature and humidity.</p>\r\n\r\n<p>How does having narrow nostrils benefit us in cold weather and vice versa? Based on research on fluid dynamics of air in different geometries of the nasal cavity, we know that air entering the airways tends to get more turbulent if the airways are narrow. This turbulence is thought to promote better "mixing" of the air and the mucosal membranes lining the nasal cavity. As such, narrower airways might have been beneficial in cooler climates and may have been less important, or even detrimental, in hotter climates.</p>\r\n\r\n<p>Why does this matter? Human adaptation is important for understanding prevalence of various health risks. For example, the risk of skin cancer is higher near the equator. Darker skin has more melanin, a natural sunscreen, which helps to reduce this risk. Hence people near the equator tend to have more melanin in their skins. However, far away from the equator, it is important to be able to absorb enough ultraviolet light to trigger the synthesis of <a href="https://simple.wikipedia.org/wiki/Vitamin_D" target="_blank" rel="noopener">vitamin D</a>. Thus, people away from the equator tend to have be lighter. We understand the link between climate, skin pigmentation, and health better today because of a better grasp of adaptation. While the relationship between nose shape and climate isn''t as strong or clear as that for skin pigmentation, it might be associated with the risk of respiratory infection and disease, which is also important from a medical perspective.</p>', 3.5, 'Zaidi A, Mattern B, Claes P, McEcoy B, Hughes C, Shriver M. <a href="https://doi.org/10.1371/journal.pgen.1006616" target="_blank" rel="noopener">Investigating the case of human nose shape and climate adaptation</a>. <em>PLOS Genetics</em>. 2017;13(3):e1006616. doi:10.1371/journal.pgen.1006616.', 4, '2017-08-29 08:39:14', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk057'),
(124, 'Food for thought: recipe for bigger brains', '<p>There are hundreds of species of <a href="https://en.wikipedia.org/wiki/Primate" target="_blank" rel="noopener">primates</a> spread all across the globe that vary greatly in both brain size and intelligence. At the extremes, the great apes (our closest living relatives) have brains that are slightly larger than those of newborn humans, while mouse lemurs have brains less than 1/100<sup>th</sup> of that size. And although some species are very innovative, have strong self-control, display impressive spatial memory, and/or use many different types of tools, there are many other species which do not possess these skills.</p>\r\n\r\n<p>Many hypotheses have been put forth in an attempt to explain why some species have evolved large brains, specifically those whose brains are larger than expected for their body size. Animals with larger bodies tend to have larger brains because they require more processing power for maintenance and control of bodily functions. Part of what makes humans unique is the fact that we have the largest brains relative to the size of our bodies (a.k.a. relative brain size). Our brains themselves are nowhere near the largest in the animal kingdom - think elephants and whales, for example.</p>\r\n\r\n<p>The earliest ideas about what drove some species to have relatively large brains were focused on diet<em>.</em> Most primate species eat primarily fruit or leaves, and some also incorporate insects or small animals. Since fruit is clumped in time and space, and often requires extraction from protective skins, researchers suggested that fruit-eating may require greater cognitive complexity and flexibility than leaf-eating. Furthermore, fruit is a relatively higher quality food source, which could help compensate for some of the energetic costs associated with having a larger brain.</p>\r\n\r\n<p>Over the past few decades, however, the prevailing idea has been that increased social complexity is what drives primate brains to become larger - also known as the <em>social brain hypothesis</em>. Researchers supported this idea with studies showing that primate species living in larger groups have relatively larger brains. Other studies also claimed to show that species with the most socially complex systems should have relatively larger brains; however, studies disagreed as to whether monogamous species or <a href="https://en.wikipedia.org/wiki/Polygynandry" target="_blank" rel="noopener">polygynandrous</a> (multi-male multi-female) species have more complex systems and relatively larger brains.</p>\r\n\r\n<p>We set out to resolve the following contradictions with a new study. First, does diet or sociality (or both) explain differences in relative brain size across primates? Second, do species with certain mating systems have relatively larger brains than those with other systems? The main problem seemed to be that not many species (i.e. less than 50) were included in most of the older studies, so we collected data from over 140 primate species. We also utilized the newest statistical techniques to see if that impacted the results. Humans were not included in our study because our brains are so exceptionally large. We also do not have a clear average group size or mating system (i.e. not all human societies consider monogamy to be the norm).</p>\r\n\r\n<p>We found that primate species that eat higher quality, harder to find foods, such as fruit and/or small animals, have relatively larger brains than those that eat leaves, a low quality and abundant food source. Moreover, our results also indicate that species with larger group sizes do not have relatively larger brains, and that different mating systems do not explain differences in relative brain size across species. These findings not only refuted one of the main predictions of the social brain hypothesis, but also supported the older ideas about the influence of diet on primate brain size evolution. Given that primates often deal with ecological challenges within social contexts, it is likely that selective pressures from both the physical and social environments impacted primate brain evolution.</p>\r\n\r\n<p>As for us humans, it''s a bit harder to tease apart what initially drove the evolution of our huge brains, but the results from our study suggest that diet may have played at least as strong of a role as social factors. Human evolution is marked by an increase in meat and seafood consumption. These are not only high quality foods, but the technology required to hunt for and cook them demands complex cognitive abilities. It''s possible that selection for technological intelligence related to obtaining such foods came first, with associated neurological changes providing the scaffolding for the subsequent development of complex social skills.</p>', 4, 'DeCasien A, Williams S, Higham J. <a href="https://doi.org/10.1038/s41559-017-0112" target="_blank" rel="noopener">Primate brain size is predicted by diet but not sociality</a>. <em>Nature Ecology &amp; Evolution</em>. 2017;1(5):0112. doi:10.1038/s41559-017-0112.', 4, '2017-09-04 09:46:45', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk058'),
(125, 'Homo floresiensis – little species, big mystery', '<p>A new kind of human, <em>Homo floresiensis</em> was a surprise discovery in 2003 by an Australian-Indonesian archaeological team who were trying to find the origins of the first Australians. Their focus was <a href="https://en.wikipedia.org/wiki/Liang_Bua" target="_blank" rel="noopener">Liang Bua cave</a>, on the island of Flores, Indonesia. Instead of finding modern human bones, they discovered something completely unprecedented. Deep in the excavation were bones representing a number of very different and very small individuals dated to between 100.000 and 60.000 years ago. These bones represent a new species and are named <em>Homo floresiensis</em>.</p>\r\n\r\n<p>14 years on, and we know a lot about <em>H. floresiensis</em>. Individuals were short - about 1 meter tall. They had a small brain of only 426 cm<sup>3</sup> (ours average 1300-1500cm<sup>3</sup>); had backward sloping foreheads, yet they possessed an expanded <a href="https://simple.wikipedia.org/wiki/Pre-frontal_cortex" target="_blank" rel="noopener">frontal cortex</a>. This implies they could do some smart things such as plan, learn from mistakes, and hand down information from generation to generation. They lacked a chin, and instead had some ape-like structures inside the jaw. Wrist bones were also ape-like. The arms were relatively long and its shoulders were shrugged and hunched forward. This species walked upright. Its walk, however, would have been somewhat odd because its feet were quite long compared to its legs. It had to lift those feet up higher than we do, just to get ground clearance.</p>\r\n\r\n<p>The mystery is - where does this species fit on the human evolutionary tree? Two hypotheses were originally proposed:</p>\r\n\r\n<p>1)That <em>H. floresiensis</em> was the dwarfed descendant of a <em>Homo</em> <em>erectus </em>population that evolved under conditions of isolation on a small island (the "island rule").</p>\r\n\r\n<p>The "island rule" stipulates that body size of mammals alters when a founder population reaches an island, becomes reproductively separated from its mainland origin group and faces an environment different from that of its mainland cousins. For example, a smaller body size could be expected as an evolutionary response to a limited food supply, and conversely a larger body size may occur in the absence of predation. <em>H. erectus </em>is the only known early hominin species from Indonesia. It is much larger than <em>H. floresiensis</em> and lived on Java 1.5 million years ago. There is no evidence for <em>H. erectus</em> on Flores, but then, Flores is relatively unknown archaeologically.</p>\r\n\r\n<p>2)That <em>H. floresiensis</em> derived from an early lineage of <em>Homo</em>, similar to species that are known from around 2 million years ago in Africa, which were all relatively short. This would imply that an unknown founder population of archaic hominins arrived on Flores. Until recently we had no indication that a small hominin species had got to Flores, but in 2016 archaeologists at Mate Menge, about 74 kms from Liang Bua cave, discovered a <em>H. floresiensis</em>-like, but smaller, partial jaw of an adult individual, and some teeth, that are dated to 700,000 years ago.</p>\r\n\r\n<p>We tested the two hypotheses by comparing characteristics of the cranium, jaws, dentition, shoulders, arms and legs of <em>H. floresiensis </em>with <em>Australopithecus afarensis</em>, <em>A. africanus</em>, <em>A. sediba</em>, <em>H. habilis</em>, <em>H. ergaster</em>, <em>H. georgicus</em>, <em>H. naledi</em>, <em>H. erectus</em>, <em>H. floresiensis</em> and <em>H. sapiens</em>. This is the first time that the <em>H. floresiensis</em> question had been addressed using a "whole-of-body" approach. Most previous work had focused on crania, mandibles and teeth.</p>\r\n\r\n<p>Our results showed that <em>H. floresiensis</em> and <em>H. habilis </em>were closely related. <em>H. habilis </em>a species known only from Africa, dated to around 1.4-1.8 million years ago. <em>H. floresiensis </em>and <em>H. habilis</em> form "sister species" in our phylogenetic tree, which means that they most likely shared an immediate common ancestor. <em>H. erectus, </em>on the other hand, formed a sister species to the 1.5 million year old <em>H. ergaster</em> from East Africa. That is, <em>H. floresiensis</em> and <em>H. erectus </em>are not closely related at all and in additional tests we performed we found no statistical support for a close phylogenetic relationship between these two species.</p>\r\n\r\n<p><em>H. floresiensis</em> was a relict population that had descended from an unknown lineage of small, ~2 million year old hominins that lived half a world away. This represents an earlier diffusion of hominins out of Africa than we expected, and we now have the tantalizing prospect of one day finding <em>H. floresiensis</em> in Africa.</p>', 3.5, 'Argue D, Groves C, Lee M, Jungers W. <a href="https://doi.org/10.1016/j.jhevol.2017.02.006" target="_blank" rel="noopener">The affinities of Homo floresiensis based on phylogenetic analyses of&nbsp;cranial, dental, and postcranial characters</a>. <em>Journal of Human Evolution</em>. 2017;107:107-133. doi:10.1016/j.jhevol.2017.02.006.', 4, '2017-09-05 08:39:56', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk059'),
(126, 'One root for every soil: a double-personality tale ', '<p>In biology, life success is based on the ability to have offspring. For that, before reproduction, organisms must deal with different and sometimes unfriendly situations. The capacity to overcome environmental challenges increases the possibility to have descendants, and will determine then that organism''s life success.</p>\r\n\r\n<p>A widespread strategy to deal with uncomfortable situations is to move. For example, some birds have exploited this ability by flying to warmer places during winter. However, plants are fixed to the soil, and their survival will depend on the ability to react quickly to environmental changes without being able to escape.</p>\r\n\r\n<p>One way to understand the tight connection between plants and their environment is to study the organs that anchor them. Roots have two very different roles. On one hand, they restrict the movement of the plants. But on the other, they are very plastic, and can modify their architecture and composition in response to external cues.</p>\r\n\r\n<p>The main component of roots are its vessels: thin tubes within them that transport the minerals and water that all plants need for growing. These vessels are surrounded by a tissue called endodermis, which protects them from the environment. Apart from its protective function, the endodermis is also responsible for absorbing nutrients. Under tough conditions, such as drought or high levels of salt, the endodermis can focus more into its protective role by changing its composition. However, when focusing on protecting the vessels, the endodermis is less effective in its role of absorbing nutrients.</p>\r\n\r\n<p>Scientists from the University of Lausanne observed how deficiencies of several important minerals affect the composition of a root''s endodermis. For that, they used the plant model <em><a href="https://en.wikipedia.org/wiki/Arabidopsis_thaliana" target="_blank" rel="noopener">Arabidopsis thaliana</a></em>, frequently used in research since it is easy to grow, it produces seeds in a relatively short time, and a huge collection of mutants is available for scientific purposes. In this study, mutant plants with an abnormal root composition and mutants that are not able to assimilate some minerals were grown under different soil conditions. The way that the different mutants reacted to the different soils gave information on how certain mineral deficiencies affect the composition of the root. The results showed that the lack of potassium and sulfur changes the endodermis to become more protective and less absorbent. Contrarily, the lack of iron and manganese makes roots more prone to absorb, but less protective. The two opposite responses are mediated by two different plant hormones, abscisic acid and ethylene, involved in many other processes of the plant. Their results also showed that these responses can be reversed, and they can be triggered every time an environmental change occurs.</p>\r\n\r\n<p>In conclusion, not only drought or salt, but also certain nutrients, can trigger changes in root composition. This finding helps us to understand the trade-off between protecting the inner root and absorbing nutrients from the soil. Plants can therefore protect their roots in response to unfavorable conditions, but they can also lose this protection to become more absorbent if, as a consequence, some nutrient becomes limited. These results show us how, despite not being able to move, plants can still cope with the lack of minerals and water in the soil.</p>\r\n\r\n<p>In the same way that, to know Dr. Jekyll, it is necessary to know Mr. Hyde, plant roots are organs with two opposite functions that are tightly connected. Integrating both functions is the key to understanding how plants are able to grow under different soil conditions. As Dr. Jekyll said, "of the two natures that contended in the field of my consciousness, even if I could rightly be said to be either, it was only because I was radically both." (Strange Case of Dr. Jekyll and Mr. Hyde, Robert Louis Stevenson).</p>', 3, 'Barberon M, Vermeer J, De&nbsp;Bellis D, Wang P, Naseer S, Andersen T, Humbel B, Nawrath C, Takano J, Salt D, Geldner N. <a href="https://doi.org/10.1016/j.cell.2015.12.021" target="_blank" rel="noopener">Adaptation of Root Function by Nutrient-Induced Plasticity of Endodermal Differentiation</a>. <em>Cell</em>. 2016;164(3):447-459. doi:10.1016/j.cell.2015.12.021.', 5, '2017-09-12 14:07:27', NULL, NULL, 2, 'https://doi.org/10.25250/thescbr.brk060'),
(127, 'The daily life of Neandertals', '<p><a href="https://en.wikipedia.org/wiki/Neanderthal" target="_blank" rel="noopener">Neandertals</a> are our closest evolutionary relative. They have been extinct for approximately 40,000 years, but lived across Europe and West Asia where they co-existed and interbred with humans. Despite their remarkable physical and genetic similarity to us humans, our understanding of their lifestyle is limited. Who were our enigmatic genetic cousins?</p>\r\n\r\n<p>Understanding the diet of an ancient group can give insight into many aspects of their lives. What foods they ate shows how they interacted with their environments. In turn, knowing how the landscape was exploited and managed provides insight into knowledge, tools, and social structures that may have been in place. On a more personal note, food is a central part of day-to-day life and knowing what someone ate can help us see the world through their eyes. Recently, a new method has been developed to directly recover elements of an individual''s diet from their dental calculus. This often-ignored deposit on the teeth has shed new light onto the Neandertals.</p>\r\n\r\n<p><a href="https://en.wikipedia.org/wiki/Calculus_(dental)" target="_blank" rel="noopener">Dental calculus</a> is a calcified layer of dental plaque, also known as tarter. Dental plaque is a slightly sticky layer of microbes that grow on the teeth. Small fragments of anything placed in the mouth may become stuck in the sticky plaque and preserved during the calcification. For three Neandertals, one from <a href="https://en.wikipedia.org/wiki/Spy_Cave" target="_blank" rel="noopener">the cave at Spy in Belgium</a> and two from <a href="https://en.wikipedia.org/wiki/Sidr%C3%B3n_Cave" target="_blank" rel="noopener">the caves at El Sidron in Spain</a>, their dental calculus was cleaned from their teeth. Then, using a technique called metagemomics all the preserved DNA within the dental calculus was retrieved and used to identify those ancient fragments, a direct insight into Neandertal diet.</p>\r\n\r\n<p>The diets of the two regions were markedly different. DNA from the Spy individual indicated that he had consumed what we may expect of a Stone Age hunter: wooly rhino and sheep, alongside mushrooms. Meanwhile, the dental calculus of the El Sidron individuals did not contain any indication of meat, instead revealing a diet that contained mushrooms, pine nuts, forest moss, and poplar. This paints a picture of the Neandertals ability to adapt to their environment and survive on varying resources. However, the DNA didn''t just give an indication of what these individuals ate as they sat down for dinner.</p>\r\n\r\n<p>Contrasting with the Neandertal brutish stereotype, the Spy individual appears to have been self-medicating, a complex behaviour requiring the understanding of the effects of various, specific resources. He was likely combating the pain from a tooth abscess (which was visible by looking at the jaw bone) and a gut parasite (whose DNA was recovered from the dental calculus), and he was doing it with aspirin and antibiotics. The natural sources of these modern medicines were identified from the dental calculus. This individual had eaten material from the poplar tree, which contains the painkiller salicylic acid, (the active ingredient in aspirin), and the antibiotic producing fungus penicillium. Humans would, of course, also use these resources through the ages, culminating in today''s modern medicines. The knowledge that Neandertals used these resources should be pulling down the boorish Neandertal stereotype and replacing it with a nuanced appreciation of their abilities.</p>\r\n\r\n<p>While the behavioural similarities between humans and Neandertals are becoming evident, it is the microbes that are showing just what has changed. As noted, dental calculus is primarily made up of microbes. This microbial community is part of the microbiome, which plays key roles in health and disease, and is vertically inherited (read also the Break: <a href="https://doi.org/10.25250/thescbr.brk050" target="_blank" rel="noopener">The cutting EDGE: Bringing genomics to everyone</a>). Despite being evolutionarily separate, ancient humans and Neandertals have exchanged at least one microbe. The full genome of a gum disease causing archaea (<em>Methanobrevibacter oralis</em>) was reconstructed from the Neandertal dental calculus. This genome was a subspecies of the one found in human mouths, which is what you''d expect when humans and Neandertals were different species (or subspecies). However, the archaea subspecies had diverged when humans and Neandertals were already two different species. How this transfer occurred is only speculation: via the environment, from sharing tools, even from kissing. However, it occurred, further demonstrating that humans and Neandertals directly interacted with one another.</p>\r\n\r\n<p>Our picture of the Neandertals as a complex, intelligent group is growing ever clearer. Here, ancient DNA from the dental calculus of Neandertals inferred dietary differences between Neandertal groups and knowledge of self-medication. The transfer of at least one oral pathogen also strongly indicates that our ancestors and our genetic cousins had a social relationship. This research delivers new information in the ever more humane view of the cavemen.</p>', 4, 'Weyrich, L. S., S. Duchene, J. Soubrier, L. Arriola, B. Llamas, J. Breen, A. G. Morris, K. W. Alt, D. Caramelli, V. Dresely, M. Farrell, A. G. Farrer, M. Francken, N. Gully, W. Haak, K. Hardy, K. Harvati, P. Held, E. C. Holmes, J. Kaidonis, C. Lalueza-Fox, M. de la Rasilla, A. Rosas, P. Semal, A. Soltysiak, G. Townsend, D. Usai, J. Wahl, D. H. Huson, K. Dobney and A. Cooper (2017). "<a href="https://doi.org/10.1038/nature21674" target="_blank" rel="noopener">Neanderthal behaviour, diet, and disease inferred from ancient DNA in dental calculus</a>." Nature 544(7650): 357-361.', 4, '2017-09-19 08:38:51', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk061');
INSERT INTO `breaks` (`id`, `title`, `content`, `reading_time`, `original_article`, `subject_id`, `date_created`, `author1_id`, `author2_id`, `editor_id`, `imzy_link`) VALUES
(128, 'How cats conquered the Ancient world: a 9,000-years DNA tale', '<p>When animals became domesticated, they gained protection from starvation, predation and disease but lost freedom. This is very well exemplified with the dog, the first animal that had been domesticated and that is very different from its ancestor, the wolf, in terms of behavior, morphology and physiology, which is also seen in its genome. But what about the cat that in many cases is still difficult to distinguish from wildcats, even at the genomic level, and whose behavior is not too different from the one of wildcats either?</p>\r\n\r\n<p>It was known from analyses of the <a href="https://en.wikipedia.org/wiki/Mitochondrial_DNA" target="_blank" rel="noopener">mitochondrial DNA</a> from modern cats that all domestic cats, including feral ones, descend from the north African <a href="https://en.wikipedia.org/wiki/African_wildcat" target="_blank" rel="noopener">wildcat <em>F. s. lybica</em></a>, that is also present in southwest Asia. This is the region where humans first started during the <a href="https://en.wikipedia.org/wiki/Neolithic" target="_blank" rel="noopener">Neolithic period</a> to domesticate plants, settled down and began farming. They also started to domesticate those animals that they used to hunt and eat, such as sheep, goats and pigs. The archeological find of a cat skeleton in a ca. 9,500-year-old child burial on Cyprus suggests that early Neolithic farmers had transported the cat to this island. This also suggests that cats had been drawn into the human sphere of life, presumably through the "vector" rodents that thrived on cereals stocks accumulated by the farmers who settled, for the first time in human history, in the Fertile Crescent. Rodents destroy the harvest through feeding upon it but also through polluting it with their excrements. Moreover, they gnaw on organic material, such as ropes on ships or leather parts of weapons. Cats preyed on these animals delivering the farmers from these pests and venomous animals. A commensal relationship was established that benefitted both partners.</p>\r\n\r\n<p>We performed a <a href="https://en.wikipedia.org/wiki/Paleogenetics" target="_blank" rel="noopener">paleogenetic</a> analysis of the mitochondrial DNA of more than 300 archeological cat remains from the last 10,000 years in Europe, South West Asia and North Africa. In agreement with the hypothesis that the translocation of animals by Neolithic farmers is indicative of an early taming process, the phylogeographic pattern of cats changed after the Neolithization. Indeed, the Anatolian lineage of ancient native <em>F. s. lybica</em> wildcat, was found in more recent remains from southeast Europe as well as elsewhere in southwest Asia where it is known, from archeological data, that the Anatolian farmers had migrated at later periods.</p>\r\n\r\n<p>Many archeologists had been convinced for a long time that the cat had been tamed in Egypt because of the particular role that cats had played in ancient Egypt. This was deduced from tombs and statues, but also described by Greek-Roman writers such as Herodotus. Indeed, so far it was the abundant Egyptian iconography that represented the major source of information about the prominent place, worshipped and adored, allocated to cats in ancient Egypt. Due to this particular status, we were not too surprised when we found a particular mitochondrial lineage in cat remains from Egypt dating to the 1<sup>st</sup> millennium BCE. We were surprised, however, to see this lineage spreading very fast and efficiently to other locations within a few centuries, including a Viking site at the Baltic Sea. In Anatolia by Roman times, the native lineage was not the majority anymore, but was surpassed by the Egyptian lineage. In addition, we discovered the mitochondrial lineage of the Central and south Asian wildcat in remains from a Roman port in Egypt. This port was located at the Red Sea and it was characterized by very tight trading bonds with India. Putting together these different pieces of evidence a composed picture arose that showed us that the dispersal of cats must have been occurred mainly by ships where they either embarked themselves or were embarked by seafarers to protect the food supply and the ropes from rodents. They might have been compensated with new delicacies, such as fish. Thus, cats seem to have conquered the world by boat - a conclusion we would not have been able to draw without the analysis of so many samples!</p>\r\n\r\n<p>We could also add another jigsaw piece to the tale of cat domestication by analyzing the coat pattern: while the fur of all wildcats has stripes, many domestic cats have a patchy fur marking. According to our samples, the gene behind this pattern made its appearance after the 13<sup>th</sup> century CE, and then spread from Anatolia throughout the Ottoman Empire. This is evidence in support of the hypothesis that cats had not been selected very much over most of the time they used to live in human company, but rather kept on being independent, almost wild hunters with the difference that they just did not run away from humans.</p>', 4, 'Ottoni C, Van Neer W, De Cupere B, Daligault J, Guimaraes S, Peters J, Spassov N, Prendergast M, Boivin N, Morales-Mu&ntilde;iz A, Bălăşescu A, Becker C, Benecke N, Boroneant A, Buitenhuis H, Chahoud J, Crowther A, Llorente L, Manaseryan N, Monchot H, Onar V, Osypińska M, Putelat O, Quintana Morales E, Studer J, Wierer U, Decorte R, Grange T, Geigl E. <a href="https://doi.org/10.1038/s41559-017-0139" target="_blank" rel="noopener">The palaeogenetics of cat dispersal in the ancient world</a>. <em>Nature Ecology &amp; Evolution</em>. 2017;1(7):0139. doi:10.1038/s41559-017-0139.', 4, '2017-09-20 08:33:37', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk062'),
(129, 'Saving the injured: The value of rescued veterans in a predatory ant species', '<p>In general, we imagine ants as little pieces of something bigger, with the value of the individual being marginal, even sacrificing themselves if necessary for the good of the colony. <a href="https://en.wikipedia.org/wiki/Megaponera" target="_blank" rel="noopener">Matabele ants</a> (<em>Megaponera analis</em>) are a specialized termite predator of sub-Saharan Africa. In the early morning and evening hours they leave their nest in raiding parties of up to 500 ants to kill termites at their foraging sites (where they are more vulnerable than inside their nests). Termites though developed ways to defend themselves against predators: a special caste of soldier-termites have developed large heads with strong jaws to fend off and kill predators that are out for an easy snack. Matabele ants, being specialized on overcoming this defenses, easily overwhelm the termites'' defenses and after 10-15 minutes have killed up to 1000 termite workers which they then carry back to their nest. These battles come at a cost though; some of the ants loose legs and antenna or have termite soldiers clinging to them after the fight. Remarkably we found that after the hunt nestmates take care of their injured by carrying them back to the nest.</p>\r\n\r\n<p>We now wondered if this behaviour was truly focused on injured individuals? If so what was the mechanism regulating it and what was the value of these injured ants for the colony?</p>\r\n\r\n<p>To answer the first question we presented the ants during their return journey with artificially injured ants (by cutting off two legs of healthy ants). These ants were always helped, in contrast to dead or healthy ants. Being now certain that this behaviour was focused on injured ants we could move on to more detailed observations.</p>\r\n\r\n<p>Through experiments with different ant glands, we found out that injured ants gave out a "call for help" after the fight with <a href="https://en.wikipedia.org/wiki/Pheromone" target="_blank" rel="noopener">pheromones</a> in their mandibular gland. This attracted their nestmates, which started to antennate (investigate) the injured individual. The injured ant then assumed a fetal position (pulling in its legs) facilitating thereby its transportation and got carried back to the nest. Interestingly the help didn''t stop there. If the injured ant still had some termites clinging on to its legs they got removed inside the safe confounds of the nest, thereby making her again a fully functioning member of the colony.</p>\r\n\r\n<p>What happens though with the ants that lost one or two legs? These ants carry permanent injuries, which should be a major handicap. Directly after the injury, these ants move roughly 30% slower than healthy ants, but after only 24 hours they adapt their locomotion with the remaining legs so that they reach running speeds similar to healthy ants, making them again useful members for the colony.</p>\r\n\r\n<p>But why do these injured ants not simply return by themselves to the nest? The reason is simple: due to their injuries they cannot keep up with the returning raid, thus becoming isolated from their nestmates and thereby easy prey for spiders or other predatory ants. Hence roughly a third of the injured ants would die during the return journey if not helped. All of these injured veterans (if helped) are observed to participate again in future hunts against termites, contributing again fully to the colony. This allows the colony to be up to 28% larger than a colony that would not help their injured.</p>\r\n\r\n<p>This study was the first detailed observation of a behaviour focused on injured individuals that were in no imminent threat of death, but were under an abstract threat through an increased mortality risk if not helped. This shows that rescue behaviour can evolve even in species that are very likely not aware of the implications of their behaviour (reduced predation risk) and through mechanisms that have nothing to do with empathy (in our case its chemical communication).</p>', 3, 'Frank E, Schmitt T, Hovestadt T, Mitesser O, Stiegler J, Linsenmair K. <a href="http://doi.org/10.1126/sciadv.1602187" target="_blank" rel="noopener">Saving the injured: Rescue behavior in the termite-hunting ant Megaponera analis</a>. <em>Science Advances</em>. 2017;3(4):e1602187. doi:10.1126/sciadv.1602187.', 4, '2017-09-25 08:58:40', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk063'),
(130, 'Red in Tooth and Claw: another weapon against antibiotic resistance', '<p>Bacteria are an integral part of human life. These organisms are on your skin, in your mouth, your ears, and your gut. After birth, a diverse population is acquired by the age of three and remarkably the population is quite similar and just as diverse in adulthood, practically remaining unchanged. Although the <a href="https://simple.wikipedia.org/wiki/Human_microbiome" target="_blank" rel="noopener">microbiome</a> is essential to our survival, the world of bacteria is incredibly complex. While most bacteria do not harm us and they often protect us, pathogenic bacteria cause harm of their hosts. By understanding how these organisms function and interact with their surroundings is critical to identifying new ways to treat harmful bacterial infections and has been one of the most important scientific advances in achieving a long healthy life for members of society.</p>\r\n\r\n<p>Competition and natural selection in nature are rampant, and bacteria as well have their own means of outcompeting other organisms: one is through the production of natural products (or small molecules). A notable and well-known natural product is <a href="https://simple.wikipedia.org/wiki/Penicillin" target="_blank" rel="noopener">penicillin</a>. Penicillin was the very first antibiotic discovered when <a href="https://en.wikipedia.org/wiki/Alexander_Fleming" target="_blank" rel="noopener">Alexander Fleming</a> noticed that mold had contaminated his colonies of Staphylococcus aureus inhibiting their growth. Shortly following, he identified penicillin, resulting in a new family of compounds that have been undeniably one of the most important discoveries of the twentieth century. Commonly encountered infections could be treated simply by use of this antibiotic.</p>\r\n\r\n<p>Although penicillin paved the way for antibiotic development, another highly important family of antibiotics came with the discovery of <a href="https://en.wikipedia.org/wiki/Vancomycin" target="_blank" rel="noopener">vancomycin</a> in 1956. Vancomycin is unique as it targets fundamental <a href="https://en.wikipedia.org/wiki/Cell_wall#Bacterial_cell_walls" target="_blank" rel="noopener">cell wall</a> building blocks (unlike penicillin) present across bacteria that affects the integrity of their cell wall. Think of stones required to build a castle. Without a proper castle wall, the insides are compromised. For bacteria, this results in death. Because of how vancomycin works, it is highly difficult for bacteria to develop a resistance. This makes vancomycin quite efficacious and it is employed when penicillin cannot get the job accomplished, resulting in it being a "last-resort" drug for nearly 50 years in the clinic.</p>\r\n\r\n<p>Despite vancomycin''s efficaciousness and long-standing utility in the clinic, there has been a rise in resistant bacterial strains towards vancomycin. Hospitals are now confronted with the frightening reality that drugs to treat such infections are sometimes not only ineffective, but might also result in breeding pathogenic bacteria that are even more resistant to any prescribed drug. Particular strains of bacteria are now a major threat and hospitals need a general solution. Thus, it is not surprising that vancomycin-resistant strains have recently been identified by the World Health Organization among those that pose the greatest threat to worldwide human health.</p>\r\n\r\n<p>Researchers within our lab sought to address this problem. How do we design an antibiotic that is not only potent against vancomycin-resistant strains, but that might also "stand" the test of time? That is, can we change vancomycin in such a way that results in an antibiotic not only effective against vancomycin-resistant strains, but are also not prone to bacteria developing resistance to the drug. Can we design a truly durable antibiotic? Through experimentation, we identified how to synthetically change vancomycin such that it can now attack bacteria on multiple fronts simultaneously. One change overcomes the molecular basis of vancomycin resistance. Two additional changes each introduced new independent ways in which the compound could prevent productive bacterial cell wall assemblage (the castle''s wall). By employing such a strategy, these vancomycin-resistant strains were shown to be unable to create a counter-strategy (namely to develop resistance) towards our vancomycin analogue that can kill bacteria by three independent mechanisms of action. This concept was so effective, that we anticipate this being a general strategy in the future of antibiotic drug development.</p>\r\n\r\n<p>Although still at an early stage of development, we are exceedingly excited about the potential. Our goals are to further study and understand if this new vancomycin analogue is efficacious and safe when given to an infected host. We also anticipate developing new ways to produce the new vancomycin analogues so that their cost and availability will not be a barrier to their use. There is still much to do and we are merely at the trail-head of our journey.</p>', 3.5, 'Okano A, Isley N, Boger D. <a href="https://doi.org/10.1073/pnas.1704125114" target="_blank" rel="noopener">Peripheral modifications of [&Psi;[CH 2 NH]Tpg 4 ]vancomycin with added synergistic mechanisms of action provide durable and potent antibiotics</a>. <em>Proceedings of the National Academy of Sciences</em>. 2017:201704125. doi:10.1073/pnas.1704125114.', 6, '2017-10-03 08:32:35', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk064'),
(131, 'By default, people treat immoral actions as impossible', '<p>Imagine that you''re on the way to the airport to catch a flight, but your car breaks down on the side of the road. In this situation, some of the solutions you would immediately consider seem obvious: you might call a friend, hail a taxi, or find public transportation. Upon reflection, though, the number of possible solutions you could consider is infinite. In principle, you could consider everything from trying to fix your car by banging on it, to selling your car for a ride to the airport, to hijacking the next car that drives by, to attempting to levitate and fly to the airport.</p>\r\n\r\n<p>When facing open-ended problems like this, humans seem to have an impressive capacity for focusing in on the small subset of solutions that are both likely to work and would be worth pursuing. What''s more, we seem to be able to come up with these kinds of solutions quickly and seemingly automatically. All of this suggests that we have some default way of representing the possibilities that are relevant in a given situation. It''s not as though each time we face a new problem, we begin by considering all of the things that are technically possible, and then proceed to rule out which of those are not feasible or would have negative consequences, until we are finally left with a small subset of options that would be good to pursue. Instead, we seem to start off by implicitly treating only a small subset of options as real possibilities.</p>\r\n\r\n<p>While this kind of ability will likely feel familiar, it is still not well understood how such a default representation of possibility works. In fact, almost all the existing empirical research has focused on human''s ability to reason explicitly and deliberatively about what would be possible, and almost none has tried to investigate the default, implicit way that humans think about possibilities. Recently, our studies (recently <a href="https://www.ncbi.nlm.nih.gov/pubmed/28420792" target="_blank" rel="noopener">published</a> in the <em>Proceedings of the National Academy of Sciences</em>) began to make some progress on this question.</p>\r\n\r\n<p>Participants in these studies were asked to read short stories about people who face different problems (like one''s car breaking down on the way to the airport) and were then asked to make judgments about what would be possible or impossible for a person to do in that situation. Half of participants were asked to make these judgments extremely quickly (1500<em>ms</em>), forcing them to rely on the default way they think about these possibilities. The other half of participants were allowed to reflect before answering and were given unlimited time to respond. Both groups were asked about the same set of 144 different possibilities, some of which were completely ordinary (like taking a taxi), other of which were physically impossible (like levitating and flying), and others of which were immoral (like hijacking a car) or irrational (like selling one''s car for a ride to the airport). If you''re interested, you can try out the study for yourself, <a href="https://www.testable.org/experiment/31/971139/start" target="_blank" rel="noopener">here</a>.</p>\r\n\r\n<p>Participants'' responses were analyzed to ask how their representation of possibility changed when they had to answer quickly or had time to reflect. For the ordinary actions like taxing a taxi, there was no real difference: people judged ordinary actions to be possible whether they answered quickly or had time to reflect. There was a striking difference, though, for immoral and irrational actions. When participants carefully reflected before answering, they <em>rarely</em> judged that it was impossible for someone to pursue immoral or irrational solutions. In contrast, when they had to answer quickly, participants judged that it was actually impossible to pursue these solutions almost 40% of the time. In short, participants'' default way of thinking about what was possible tended to treat immoral actions as impossible.</p>\r\n\r\n<p>Stepping back from the details of this study, it may initially seem puzzling that we would implicitly treat immoral and irrational events as if they were impossible. However, one way of understanding why this would be is to notice that this kind of error might often be useful. In most situations, immoral and irrational actions will not actually be good solutions for the problems we face. Thus, a good shortcut for finding promising solutions among the infinite number of possible actions may be to only ever consider options that are both physically viable and also morally good. Most of the time, the rest of the possibilities can simply be ignored, as if they weren''t even really possible.</p>', 4, 'Phillips J, Cushman F. <a href="https://doi.org/10.1073/pnas.1619717114" target="_blank" rel="noopener">Morality constrains the default representation of what is possible</a>. <em>Proceedings of the National Academy of Sciences</em>. 2017;114(18):4649-4654. doi:10.1073/pnas.1619717114.', 1, '2017-10-11 08:02:10', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk065'),
(132, '‘Laughing’ together: bridging avian-mammalian differences', '<p>We like animals that we perceive as being similar to ourselves. It is not a coincidence that those animals that humans consider similar to them in terms of appearance, intelligence and/or sociality, also enjoy the highest levels of protection in modern societies (for example primates, dolphins and dogs). Interestingly, some "human" features that distinguish us, such as bipedalism and vocal learning, are widespread amongst birds. Nonetheless, birds have long been regarded as less similar to humans than mammals, and thus less relatable. In the past decades, research on bird cognition, mostly <a href="https://en.wikipedia.org/wiki/Corvidae" target="_blank" rel="noopener">corvids</a> (crows and ravens) and <a href="https://en.wikipedia.org/wiki/Parrot" target="_blank" rel="noopener">psittacines</a> (parrots), has shown that these animals have strikingly similar capabilities to our mammalian cousins. This may come as a surprise, considering that the closest common ancestor of mammals and birds went extinct before the rise of the dinosaurs. The similarities between mammals and birds suggest that social and survival strategies, not <a href="https://simple.wikipedia.org/wiki/Phylogeny" target="_blank" rel="noopener">phylogenetic relatedness</a>, are key for the development of similar patterns of behaviour.</p>\r\n\r\n<p>Kea parrots, like humans, play a lot. While play behaviour is common in the animal kingdom, especially amongst warm-blooded vertebrates, the fact that kea continue to play into adulthood is unusual. Furthermore, kea produce a loud, distinct play call, which most other animals do not produce. Loud play calls may be dangerous because they can alert predators to the location and activity of the playing animal. Kea do not face this problem, as they evolved with few predators in the Southern Alps of New Zealand. Having described the play call in a previous study, we became interested in what effect it has on other kea. The literature suggested that it could be similar to the "play bows" of dogs or the "play faces" of primates: an overt signal that allows others to distinguish play behaviour from other behaviours, like aggression. To test whether this was also the case for kea play calls, we needed to recreate the situation artificially. This was done by playing back recordings of the call to wild kea. To rule out factors that were shared between the play call and other calls / sounds, and thus not unique to this distinct vocalisation, we also played back several control sounds to the same birds.</p>\r\n\r\n<p>We found a large increase in play behaviour in the kea that heard the play calls. This increase was clear compared to the pre- and post-experimental periods, as well as any period when the birds heard a control sound. Interestingly, many kea did not join ongoing bouts of play, but began to play spontaneously with other non-playing kea, or engaged in solitary play (locomotor or object play). Additionally, the play call did not cause the birds to look for the source of the call any more than an artificial pure tone. Taken together, this suggests that the kea were not invited to play or informed about play happening somewhere close by. Rather, hearing the play call increased their intrinsic motivation to play. This type of effect is called an "emotional contagion", because the caller''s emotional state affects that of the listener. Playfulness is considered a positive emotional state, so the kea play call is an example of a "positive emotional contagion".</p>\r\n\r\n<p>Although kea are the first bird species to display a positive emotional contagion, similar effects have been described in mammals. Both chimpanzees and rats have play vocalisations that act as positive emotional contagions by increasing playfulness in others. This phenomenon is also found in humans, in the social effects of laughter. Especially in small children, gleeful laughter during play has positive emotional contagion effects similar to those found in primates and rats; on this basis, one finds parallels between the play vocalisations of primates and rats, and the laughter of humans. Our findings suggest that the play call of the kea functions similarly. This further bridges the perceived gap between birds and mammals, including human beings. We hope that studies like this one will help spread the knowledge that birds and mammals are not so different from one another. If people are aware that birds can play and vocalize in a similar way to humans, this may lead to better treatment of these species in our modern world.</p>', 3.5, 'Schwing R, Nelson X, Wein A, Parsons S. <a href="%20http://dx.doi.org/10.1016/j.cub.2017.02.020" target="_blank" rel="noopener">Positive emotional contagion in a New Zealand parrot</a>. <em>Current Biology</em>. 2017;27(6):R213-R214. doi:10.1016/j.cub.2017.02.020.', 4, '2017-10-17 10:02:03', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk066'),
(133, 'When and where neonicotinoids are bad for bees', '<p>The impact of neonicotinoids on bees has always been a contentious issue, not least because reported evidence of their effects has been variable. Neonicotinoids are most often applied to crop seeds, rather than sprayed directly on plants. As the plant grows the pesticide moves though the roots, leaves and flowers providing protection against pests. Unfortunately pests are not the only thing feeding on plants, and bees collecting their pollen and nectar can also become exposed. Levels of neonicotinoids found in pollen and nectar are often very low (e.g. in parts per billion) so the effect on bees can therefore be more subtle, with one of the most widely reported impacts being the loss of foraging honeybees who become disorientated due to the impact of the chemicals on their brains. Over time this has been shown to reduce the number of bees in colonies, and has the potential to affect long term colony survival (<i>Ed.</i> read also the breaks: <a href="https://doi.org/10.25250/thescbr.brk008" target="_blank" rel="noopener">A dangerous habit: bees prefer pesticide-contaminated nectar</a> and <a href="https://doi.org/10.25250/thescbr.brk019" target="_blank" rel="noopener">The use of neonicotinoid pesticides affects wild bee populations</a>). While many studies have identified negative effects on honeybees and wild bees they have often been criticised for not reflecting what we see in the real world. For example, laboratory doses of neonicotinoids often represent the upper limit of what you may find under agricultural conditions and these experiments are often very short term. The study just published by the Centre for Ecology & Hydrology, and funded in part by industry, describes the biggest "real world" experiment of neonicotinoid effects. The results provide important new evidence to inform this debate.</p>\r\n\r\n<p>The study comprised a large scale trial on 33 farms in three countries (UK, Germany and Hungary) where we sowed <a href="https://en.wikipedia.org/wiki/Rapeseed" target="_blank" rel="noopener">oilseed rape</a> that was either treated or not treated with the two most commonly used neonicotinoids (<a href="https://en.wikipedia.org/wiki/Clothianidin" target="_blank" rel="noopener">Clothianidin</a> or <a href="https://en.wikipedia.org/wiki/Clothianidin" target="_blank" rel="noopener">Thiamethoxam</a>). Large areas of the crop (on average 60 ha) were sown on each farm representing typical practice. For honeybees, we found for the first time evidence of negative effects of a neonicotinoid under real agricultural conditions. Specifically, bees on Hungarian and UK farms exposed to clothianidin in the previous year suffered higher mortality over the winter. This is the first time this has been shown and is something considered by the <a href="http://www.efsa.europa.eu/" target="_blank" rel="noopener">European Food Safety Authority (EFSA)</a> (who assess the safety of pesticides for consumers in the EU) to be one of the three most important measures with which they assess impacts of pesticides on honeybees. Another key measure for understanding pesticide impacts on honeybees is a reduction in honeybee numbers when they feed on the treated crop. Again, we also find that when honeybees foraged on clothianidin treated oilseed rape their numbers were reduced, although only in the UK. In Germany, we either didn''t find an effect of neonicotinoids, or in some cases we even found a positive effect (egg numbers when the crop was flowering). These differences between countries were surprising, and pointed to a more complex story. However, by looking in detail at each country we were able to make some deductions about what was happening. As is the case when people are sick or have a poor diet so they become far more likely to succumb to other stresses, so the same thing appears to be happening for the honeybees. In Hungary and the UK honeybees were more diseased (Varroa mites and the fungal parasite Nosema) and had poor diets dominated by oilseed rape (there was few other flowers to feed on). It was in these countries where we found negative effects of neonicotinoids. In contrast, the bees in Germany were much healthier and had a more diverse diet not dominated by oilseed rape. It is also likely that the late flowering crop in Germany may have contained lower levels of neonicotinoids as over longer periods of time they would be more likely to disappear. As a result while the bees were exposed to less pesticides, the crop benefiting from neonicotinoids use may have provided a healthier more flowering crop which the bees temporarily benefited from.</p>\r\n\r\n<p>In addition to the honeybees we also looked at two wild bee species (a bumblebee, <a href="https://en.wikipedia.org/wiki/Bombus_terrestris" target="_blank" rel="noopener"><em>Bombus terrestris</em></a> and a solitary mason bee, <a href="https://en.wikipedia.org/wiki/Osmia_bicornis" target="_blank" rel="noopener"><em>Osmia bicornis</em></a>) and found that their reproductive success (new queens in the case of bumblebees and egg cells for solitary bees) was significantly reduced when their food contained higher concentrations of neonicotinoids. Overall, this study is further evidence that neonicotinoids can have negative effects on bees under real world conditions. However, it also highlights that the negative effects (at least for honeybees) are often context dependent. A good agricultural environment rich in wild flowers (particularly early in the year) and better approaches to managing honeybee health may mean that it may be possible to reduce the effects of exposure to neonicotinoids. However, good evidence derived from new robust and large-scale field experiments would be needed to test if this is true.</p>', 4, 'Woodcock B, Bullock J, Shore R, Heard M, Pereira M, Redhead J, Ridding L, Dean H, Sleep D, Henrys P, Peyton J, Hulmes S, Hulmes L, S&aacute;rospataki M, Saure C, Edwards M, Genersch E, Kn&auml;be S, Pywell R. <a href="https://doi.org/10.1126/science.aaa1190" target="_blank" rel="noopener">Country-specific effects of neonicotinoid pesticides on honey bees and wild bees</a>. <em>Science</em>. 2017;356(6345):1393-1395. doi:10.1126/science.aaa1190.', 3, '2017-10-26 09:36:08', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk067'),
(134, 'The belligerence of breeding: female aggression after mating', '<p>Sex changes us. In addition to the overwhelming Puberty-Blues kind of way, sex induces physical changes that occur across the entire animal kingdom, from elephants to fruit flies. Yet, there''s one glaring aspect of sex that few scientists have studied - female aggression. That''s where my research comes in. I study how mating increases female aggression in fruit flies.</p>\r\n\r\n<p>Female aggression towards other females has traditionally been ignored in evolutionary biology. But females do compete, just over different things than males do. Female dung beetles compete over dung to feed their offspring, while female eclectus parrots fight over the best nesting sites to raise chicks. These contests have dramatic effects on females'' reproductive success and their offspring''s survival, but we still know very little about how and when females compete.</p>\r\n\r\n<p>Mating is one logical answer. Females invest heavily in their offspring, whether they lay hundreds of eggs, or raise one offspring to adulthood - you need a lot of fuel to keep going! But just because all of a sudden you''re hungrier, it doesn''t mean that there is more food available in the environment. You''re still competing with other females to get access to that food, but now you want a larger slice of the pie. That means to get more, you need to get aggressive.</p>\r\n\r\n<p>I tested this idea in fruit flies, an organism in which we can easily manipulate specific genes and behaviours to test what influences female aggression. I found that females that mated once, spent twice as long fighting over food compared to virgin females. At first glance, this could be explained by the fact that females that are producing eggs need more food and get more aggressive towards other females to get it. But, it turns out, it''s not that simple. If you perform the same experiment with females that cannot produce eggs, and who therefore shouldn''t need more food or need to get aggressive, you see exactly the same increase in aggression after mating. Sterile mated females still fought for twice as long as virgins, despite not needing extra food.</p>\r\n\r\n<p>If eggs aren''t responsible for females getting more aggressive, than what is? The answer - sperm. Females that mate but do not receive sperm do not show any increase in aggression after mating - they''re exactly the same as virgins. Sperm is necessary for females to become more aggressive. A substance that males make and transfer to females alters female behaviour towards other females.</p>\r\n\r\n<p>Creepy.</p>\r\n\r\n<p>In addition, sperm doesn''t act alone. It is accompanied by a multitude of proteins that tag along in the seminal fluid. In fruit flies there is one tiny protein called "sex peptide" which has huge effects on female behaviour and physiology. Females that receive sex peptide lay many more eggs, become less likely to mate with other males, eat more, and sleep less. I found out that sex peptide also plays a role in increasing female aggression after mating. Females that did not receive sex peptide during mating showed an intermediate level of aggression, halfway between virgin females and females that received sex peptide. This is all due to one miniscule protein.</p>\r\n\r\n<p>Even creepier.</p>\r\n\r\n<p>The fact that two substances made by males and transferred to females during mating can alter female behaviour towards other females suggests that males may be changing female behaviour to suit themselves. Males and females have conflicts of interest over how many eggs a female should lay, when she should lay them, and how much she should invest in them. A female may prefer to spread out her energy across several batches of eggs, and have multiple mates, to increase the diversity of her offspring. A male, on the other hand, would prefer females to invest as much as possible in the current brood because they are his offspring, even if this comes at the expense of her future reproductive success.</p>\r\n\r\n<p>Sexual conflict has been demonstrated in fruit flies, where sex peptide can be good for males, but bad for females. Females receiving sex peptide over their lifetime died sooner and laid fewer eggs than those that did not receive sex peptide. It has not previously been proposed, however, that female-female aggression could be a behaviour under sexual conflict.</p>\r\n\r\n<p>And let''s not forget - sperm, seminal proteins, and sexual conflict are not just confined to fruit flies. Seminal proteins may affect female aggression in other species, potentially even humans, but so far we''ve only tested it in fruit flies. Who knows how sex has already changed us?</p>', 4, 'Bath E, Bowden S, Peters C, Reddy A, Tobias J, Easton-Calabria E, Seddon N, Goodwin S, Wigby S. <a href="https://doi.org/10.1038/s41559-017-0154" target="_blank" rel="noopener">Sperm and sex peptide stimulate aggression in female Drosophila</a>. <em>Nature Ecology & Evolution</em>. 2017;1(6):0154. doi:10.1038/s41559-017-0154.', 4, '2017-11-01 09:08:32', NULL, NULL, 1, 'https://doi.org/10.25250/thescbr.brk068');

-- --------------------------------------------------------

--
-- Table structure for table `core_team`
--

CREATE TABLE IF NOT EXISTS `core_team` (
  `id` int(11) NOT NULL,
  `last_name` varchar(40) CHARACTER SET latin1 NOT NULL,
  `first_name` varchar(40) CHARACTER SET latin1 NOT NULL,
  `position` varchar(200) CHARACTER SET latin1 NOT NULL,
  `biography` text CHARACTER SET latin1,
  `is_editor` tinyint(1) DEFAULT NULL,
  `email` varchar(50) COLLATE utf8_unicode_ci DEFAULT NULL,
  `research_institute` varchar(250) COLLATE utf8_unicode_ci DEFAULT NULL
) ENGINE=MyISAM AUTO_INCREMENT=19 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `core_team`
--

INSERT INTO `core_team` (`id`, `last_name`, `first_name`, `position`, `biography`, `is_editor`, `email`, `research_institute`) VALUES
(1, 'Caine', 'Massimo', 'Founder of TheScienceBreaker', 'Massimo, molecular biologist, is constantly on a mission to inspire scientists and laypeople around him with his passion for science. During the time spent on the bench, he followed his natural all-around curiosity, investigating several topics from medical diseases to plant physiology. Head of TheScienceBreaker, Massimo proudly chases his naïve dream of an engaged society where scientists and citizens are facing together the upcoming challenges for human civilizations. At the University of Geneva, Switzerland, Massimo works for BiOutils – an academic and laboratory-based platform for outreach in life sciences.', 1, 'max.caine@thesciencebreaker.com', NULL),
(2, 'Rivera-Rivera', 'Carlos Javier', 'Scientific Editor', 'Carlos has been in love with Nature ever since he can remember. He spent many days camping, snorkeling and otherwise exploring the tropical forests and seas of his native Puerto Rico. Adding passion to his love affair, he is now studying molecular phylogenetics, trying to uncover the intricate evolutionary processes that have shaped our DNA sequences. Knowing that communication is essential for any good romance, Carlos believes that the best way to communicate his love for Nature is by making the amazing world of research accessible to all.', 1, '', 'PhD student, University of Geneva'),
(6, 'Riggi', 'Margot', 'Scientific Editor', 'Margot is a dynamic and creative jack-of-all-trades. She chose to study Biology (she is currently doing a PhD in Molecular Biology at the University of Geneva), however, focusing on doing one single thing has always been difficult for her. Therefore, her activity within TheScienceBreaker as Scientific Editor enables her to broaden her playing field, while keeping herself informed about discoveries outside of her domain of expertise. Being her mission, Margot firmly believes in the importance of direct communication between scientists and laypeople.', 0, '', 'PhD student, University of Geneva'),
(7, 'Weber', 'Aurélia', 'Communication Officer', 'For a long time Aurélia thought that science was a mysterious and unreachable world, reserved for geniuses. As a consequence, she begun her studies in literature until something unexpected came along. Her curiosity took over and she decided to study Biology, obtaining (much to her surprise) a Master degree in Microbiology! Having realized that science is a world that can be discovered by anyone, she is now transmitting her scientific passion to students through her Communication Officer job at BiOutils.', 0, '', 'BiOutils, University of Geneva'),
(11, 'Preuten', 'Dr. Tobias', 'Editorial Advisor', '', 1, '', 'PhD student, University of Geneva'),
(13, 'Kozlov', 'Anatoly', 'Scientific Editor', 'Anatoly throughout his studies touched a number of different sciences – like physics, biology, philosophy. This made him think that each community has it''s own language. Even within the science different fields not always understand each other well – not to mention outsiders. However wonder is universal for all people. He believes that beauty of new discoveries could inspire everyone, no matter what person does in life. But knowledge has to be accessible. That''s why Anatoly wants to take a challenge of bringing complex concepts of modern science into the light of clarity.', 1, 'Anatoly.Kozlov@unige.ch', 'PhD student, University of Geneva'),
(14, 'Prosee', 'Reinier', 'Scientific Editor', 'Like the other members of TheScienceBreaker, Reinier has a wide range of personal and scientific interests. Between obtaining his master’s degree in Neuroscience (University of Oxford) and starting a PhD in Molecular Biology (University of Geneva), Reinier worked for a medical publishing company in London. This experience increased his awareness of the communication gap between scientists and the broader public. Being part of TheScienceBreaker, Reinier aims to help bridge this gap by opening up the scientific processes behind new developments so that it can be discussed in a free, open and more inclusive way.', 1, 'Reinier.Prosee@unige.ch', 'PhD student, University of Geneva'),
(15, 'Kusmider', 'Beata', 'Scientific Editor', 'Engineer of biotechnology by training and a PhD candidate in molecular biology, Beata believes that science not only needs public understanding, but also engagement. At the same time, she realizes that science advances quickly. So fast, that it is hard for the information to be transmitted to and assimilated by the broader public. Consequently, Beata wants to participate in dissemination of new discoveries and creation of a forum, where groups with different points of view meet and every perspective can be heard and addressed.', 1, 'Beata.Kusmider@unige.ch', 'PhD student, University of Geneva'),
(16, 'Lorenzo-Orts', 'Laura', 'Scientific Editor', 'As a kid, Laura loved to read any book she could reach. Soon she developed a passion for literature and philosophy. An interest about “life” began to emerge in her and she decided to study it from a scientific point of view. Now, she is doing a PhD in plant biology, where she participates actively in the process of understanding life. The more she learns about biology, the more she thinks life is extraordinary. Being aware that scientific discoveries are not within the reach of the general public, she is willing to share her pleasure of writing to make science available to everybody.', 1, 'Laura.Lorenzo@unige.ch', 'PhD student, University of Geneva'),
(17, 'Rossetti', 'Giacomo', 'Scientific Editor', 'Giacomo is a budding molecular biologist who has developed his interest for Science during… his classical studies! The study of literature, art, greek and latin culture made him trust in the beauty of truth and he is convinced that Science has the most powerful means to seek it. But he thinks that this is not enough: it is essential to communicate Science and true information properly and make it accessible to everyone, giving everybody the possibility to be engaged in a meaningful debate. Giacomo joined TheScienceBreaker fascinated by the perspective of dealing with the work of other colleagues as well as of being regularly updated with scientific discoveries. His motto: “Science for knowledge”.', 1, 'Giacomo.Rossetti@unige.ch', 'PhD student, University of Geneva');

-- --------------------------------------------------------

--
-- Table structure for table `editor_picks`
--

CREATE TABLE IF NOT EXISTS `editor_picks` (
  `id` int(11) NOT NULL,
  `breaks_id` int(11) DEFAULT NULL
) ENGINE=MyISAM AUTO_INCREMENT=9 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `editor_picks`
--

INSERT INTO `editor_picks` (`id`, `breaks_id`) VALUES
(1, 41),
(2, 106),
(3, 119),
(4, 44);

-- --------------------------------------------------------

--
-- Table structure for table `subject`
--

CREATE TABLE IF NOT EXISTS `subject` (
  `id` int(11) NOT NULL,
  `name` varchar(50) CHARACTER SET latin1 NOT NULL
) ENGINE=MyISAM AUTO_INCREMENT=8 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `subject`
--

INSERT INTO `subject` (`id`, `name`) VALUES
(1, 'Health & Physiology'),
(2, 'Neurobiology'),
(3, 'Earth & Space'),
(4, 'Evolution & Behaviour'),
(5, 'Plant Biology'),
(6, 'Microbiology'),
(7, 'Maths, Physics & Chemistry');

-- --------------------------------------------------------

--
-- Table structure for table `subscription`
--

CREATE TABLE IF NOT EXISTS `subscription` (
  `id` int(11) NOT NULL,
  `email` varchar(50) COLLATE utf8_unicode_ci NOT NULL
) ENGINE=MyISAM AUTO_INCREMENT=269 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `subscription`
--

INSERT INTO `subscription` (`id`, `email`) VALUES
(3, 'elena.dare@gmail.com'),
(10, 'arthurvillar@gmail.com'),
(9, 'caterina.dare@gmail.com'),
(11, 'max.caine@thesciencebreaker.com'),
(12, 'maboona@bluewin.ch'),
(40, 'bettina.caine@iol.it'),
(41, 'lale.caine@gmail.com'),
(42, 'karl.perron@unige.ch'),
(45, 'raphael.meheust@etu.upmc.fr'),
(53, 'dare.elisabetta@gmail.com '),
(52, 'marco@vignetizanatta.it '),
(51, 'scogliati@cnic.es'),
(54, 'e.ventura.guzman@gmail.com'),
(55, 'ctmschryvers@gmail.com'),
(56, 'cateshoe@yahoo.it'),
(57, 'lindavaragnolo@gmail.com'),
(58, 'patrick.linder@unige.ch'),
(59, 'jhulbert@bard.edu'),
(60, 'marianna.beggio@gmail.com'),
(61, 'siluacc@hotmail.it'),
(62, 'alain.rifat@gmail.com'),
(63, 'glorelisbv@outlook.com'),
(64, 'stefania.magnusdottir@uni.lu'),
(65, 'Elisa.Dellaglio@unige.ch'),
(66, 'Tina.Kyndt@UGent.be'),
(67, 'cbowler@biologie.ens.fr'),
(68, 'vincent.bourrier@unige.ch'),
(69, 'tr1@mit.edu'),
(70, 'tonegawa@mit.edu'),
(71, 'rmk1002@cam.ac.uk'),
(72, 'Sebastien.Kessler@newcastle.ac.uk'),
(73, 'daniel.kramer@berkeley.edu'),
(74, 'jan_engelmann@eva.mpg.de'),
(75, 'per.thor.straten@regionh.dk'),
(76, 'massimo.pizzato@unitn.it'),
(77, 'joanne.ryan@mcri.edu.au'),
(78, 'Jordan.Costafrolaz@unige.ch'),
(79, 'katri.korpela@helsinki.fi'),
(80, 'pablo.mendez@unige.ch'),
(81, 'Thomas.Stefanelli@unige.ch'),
(82, 'jhankey@vtti.vt.edu'),
(83, 'ralbright@carnegiescience.edu'),
(84, 'e.green@dkfz-heidelberg.de'),
(85, 'Daniele.Guido@unige.ch'),
(86, 'sjtr@ceh.ac.uk'),
(87, 'sburthe@ceh.ac.uk'),
(88, 'ehundey2@uwo.ca'),
(89, 'kmoser@uwo.ca'),
(90, 'niun42@gmail.com'),
(91, 'martin.schwarzer@ens-lyon.fr'),
(92, 'taryn.foster@research.uwa.edu.au'),
(93, 'ncubo@ucm.es'),
(94, 'christian.w.gruber@meduniwien.ac.at'),
(95, 'carsten.gruendemann@uniklinik-freiburg.de'),
(96, 'matone@cosbi.eu'),
(97, 'raphael.meheust@gmail.com'),
(98, 'epbapteste@gmail.com'),
(99, 'm.e.laturney@rug.nl'),
(100, 'jc.billeter@gmail.com'),
(101, 'eric.domyan@uvu.edu'),
(102, 'giovanni.tenderini@gmail.com'),
(103, 'g.candiani@hotmail.it'),
(104, 'ludovica_cosulich@yahoo.it'),
(105, 'mariaforza@gmail.com'),
(106, 'ugilio@gmail.com'),
(107, 'giacomo.pellicioli@gmail.com'),
(108, 'guaivor@yahoo.it'),
(109, 'martatorresmt@gmail.com'),
(110, 'nikolas_thun_hohenstein@hotmail.com'),
(111, 'mariaciani@gmail.com'),
(112, 'alviserocchetto@hotmail.it'),
(113, 'zazzuu@gmail.com'),
(114, 'eleonora.pagan@gmail.com'),
(115, 'giuliabox@yahoo.it'),
(116, 'chiarmag@hotmail.it'),
(117, 'costanza.madricardo@gmail.com'),
(118, 'fabrizio6684@gmail.com'),
(119, 'mschenkel1985@gmail.com'),
(121, 'miriamkirsch@ymail.com'),
(122, 'antonio.marzollo@gmail.com'),
(123, 'elenamicheluzzi@gmail.com'),
(124, 'cristhynagrigoletto@hotmail.com'),
(125, 'davrav@libero.it'),
(126, 'simone.vannozzi@hotmail.it'),
(127, 'alessandra.zumbo@gmail.com'),
(128, 'andy@caine.it'),
(129, 'studiocaine@caine.it'),
(130, 'enrico.cristante@gmail.com'),
(131, 'federico.miozzo@unige.ch'),
(132, 'fede.federici84@gmail.com'),
(133, 'marcello.venzi@gmail.com'),
(134, 'monica.bellunato@gmail.com'),
(135, 'cuccurita@libero.it'),
(136, 'fabulo84@gmail.com'),
(137, 'chiaramarcon@yahoo.it'),
(138, 'balzarini.valentina@gmail.com'),
(139, 'enrico@caine.it'),
(140, 'gcichel@tin.it'),
(141, 'ufficiostampa@villevenete.org'),
(142, 'darcy.christen@chuv.ch'),
(143, 'Christophe.Ungar@rts.ch'),
(144, 'aurelia.weber@unige.ch'),
(145, 'sandrine.zuchuat@unige.ch'),
(146, 'dietelmo.pievani@unipd.it'),
(147, 'gerolamo.lanfranchi@unipd.it'),
(148, 'paolo.laveder@cribi.unipd.it'),
(149, 'elisa.greggio@unipd.it'),
(150, 'lucamicconi@alice.it'),
(151, 'christophe.pache@gmail.com'),
(152, 'fabiakessas@gmail.com'),
(153, 'Candice.Yvon@unige.ch'),
(154, 'Quentin.Dietschi@unige.ch'),
(155, 'Margot.Riggi@unige.ch'),
(156, 'jack.howe@bio.ku.dk'),
(160, 'massimo.caine@gmail.com'),
(159, 'lara.pizurki@unige.ch'),
(161, 'a.marzo@ucl.ac.uk'),
(162, 'grahamcrobinson@hotmail.com'),
(163, 'mauro.tonolla@supsi.ch'),
(164, 'rmanderson5@wisc.edu'),
(165, 'cristina.fragoso@supsi.ch'),
(166, 'giancarlo.gianocca@supsi.ch'),
(167, 'luca.morici@supsi.ch'),
(168, 'andrea.delucchi@supsi.ch'),
(196, 'guillaumemuhlebach@gmail.com'),
(170, 'J.Verran@mmu.ac.uk'),
(171, 'Laura.Bowater@uea.ac.uk'),
(172, 'Theresa.Fitzpatrick@unige.ch'),
(173, 'francescorossetti@ied.edu'),
(174, 'Anatoly.Kozlov@unige.ch'),
(175, 'Beata.Kusmider@unige.ch'),
(176, 'Carlos.RiveraRivera@unige.ch'),
(177, 'Reinier.Prosee@unige.ch'),
(178, 'laura.lorenzo@unige.ch'),
(179, 'guzzonoscar@hotmail.com'),
(180, 'Giacomo.Rossetti@unige.ch'),
(181, 'laura.lanf@gmail.com'),
(182, 'Christian.Fankhauser@unil.ch'),
(183, 'didier.perret@unige.ch'),
(184, 'melanie.binkert@gmail.com'),
(185, 'aiteasinternational@yahoo.com'),
(186, 'massimo.caine@unige.ch'),
(187, 'eme.laura@gmail.com'),
(188, 'marc.heijde@vib-ugent.be'),
(189, 'p.salinas@ucl.ac.uk'),
(190, 'f.mcleod@ucl.ac.uk'),
(191, 'linbian2@illinois.edu'),
(193, 'marie-claude.blatter@sib.swiss'),
(194, 'Olivier.Gaumer@unige.ch'),
(195, 'Xavier.Ravinet@unige.ch'),
(197, ' celine.roux@unige.ch'),
(198, 'cb172@le.ac.uk'),
(199, 'galvando@gmail.com'),
(200, 'ines.milagre@babraham.ac.uk'),
(201, 'pchain@lanl.gov'),
(202, 'hannah.ritchie@colorado.edu'),
(204, 'nick.neave@northumbria.ac.uk'),
(205, 'yogesh.vegunta@unige.ch'),
(206, 'aurelie.hintermann@unige.ch'),
(207, 'melissa.berto@unige.ch'),
(209, 'hetty.manenschijn@unige.ch'),
(210, 'Karenaero@msn.com '),
(216, 'debbie.argue@anu.edu.au'),
(259, 'sage.mkrs@gmail.com'),
(213, 'andrew.hammond08@imperial.ac.uk'),
(215, 'erik.frank@uni-wuerzburg.de'),
(217, 'raoul.schwing@vetmeduni.ac.at'),
(218, 'phillips01@g.harvard.edu'),
(220, 'hannah.hilbertwolf@my.jcu.edu.au'),
(221, 'm.weiss@hhu.de'),
(266, 'poteiza@mcb.harvard.edu'),
(224, 'nikolai.klena@unige.ch'),
(225, 'gokceagsu@gmail.com'),
(226, 'bawood@ceh.ac.uk'),
(227, 'gdurner@usgs.gov'),
(229, 'stefan.bengtson@nrm.se'),
(230, 'sarah.sadavoy@cfa.harvard.edu'),
(231, 'mickael.mourlam@umontpellier.fr'),
(232, 'andrew.farrer@adelaide.edu.au'),
(233, 'alicia.daeden@unige.ch'),
(242, 'niculmi@gmail.com'),
(236, 'bertocchinif@unican.es'),
(237, 'james.kossin@noaa.gov'),
(243, 'dudareva@purdue.edu'),
(244, 'prisca.canola@college-stlouis.ch'),
(245, 'veronique.chanon@ecolint.ch'),
(247, 'pwood22@gmail.com'),
(248, 'd.moreau@auckland.ac.nz'),
(249, 'fzwujipojt@emmail.info'),
(250, 'tbriggs@autolikes.com'),
(251, 'kira.westaway@mq.edu.au'),
(252, 'a.mitra@swansea.ac.uk'),
(253, 'thomas.spallek@riken.jp'),
(254, 'eva-maria.geigl@ijm.fr'),
(255, 'h.petousis-harris@auckland.ac.nz'),
(256, 'laumarmon_25@yahoo.es'),
(257, 'pmaher@salk.edu'),
(258, 'watahiki@sci.hokudai.ac.jp'),
(260, 'tevans@astro.ex.ac.uk'),
(261, 'h.richly@imb-mainz.de'),
(262, 'nic.ulmi@gmail.com'),
(263, 'jburns@amnh.org'),
(264, 'annachiara.rosa@alumni.unitn.it'),
(265, 'jasmine.lee1@uqconnect.edu.au'),
(267, 'aknoll@oeb.harvard.edu'),
(268, 'jt2bb@virginia.edu');

--
-- Indexes for dumped tables
--

--
-- Indexes for table `admin`
--
ALTER TABLE `admin`
  ADD PRIMARY KEY (`id`);

--
-- Indexes for table `advisory_board`
--
ALTER TABLE `advisory_board`
  ADD PRIMARY KEY (`id`), ADD UNIQUE KEY `full_name` (`first_name`,`last_name`);

--
-- Indexes for table `authors`
--
ALTER TABLE `authors`
  ADD PRIMARY KEY (`id`), ADD UNIQUE KEY `full_name` (`first_name`,`last_name`);

--
-- Indexes for table `author_break`
--
ALTER TABLE `author_break`
  ADD PRIMARY KEY (`id`), ADD KEY `break_id` (`break_id`), ADD KEY `author_id` (`author_id`);

--
-- Indexes for table `available_breaks`
--
ALTER TABLE `available_breaks`
  ADD PRIMARY KEY (`id`), ADD KEY `subject_id` (`subject_id`);

--
-- Indexes for table `breaks`
--
ALTER TABLE `breaks`
  ADD PRIMARY KEY (`id`), ADD UNIQUE KEY `title` (`title`), ADD KEY `author_id` (`author1_id`), ADD KEY `author2_id` (`author2_id`), ADD KEY `editor_id` (`editor_id`);

--
-- Indexes for table `core_team`
--
ALTER TABLE `core_team`
  ADD PRIMARY KEY (`id`);

--
-- Indexes for table `editor_picks`
--
ALTER TABLE `editor_picks`
  ADD PRIMARY KEY (`id`);

--
-- Indexes for table `subject`
--
ALTER TABLE `subject`
  ADD PRIMARY KEY (`id`), ADD UNIQUE KEY `name` (`name`);

--
-- Indexes for table `subscription`
--
ALTER TABLE `subscription`
  ADD PRIMARY KEY (`id`), ADD UNIQUE KEY `email` (`email`), ADD UNIQUE KEY `email_2` (`email`);

--
-- AUTO_INCREMENT for dumped tables
--

--
-- AUTO_INCREMENT for table `admin`
--
ALTER TABLE `admin`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=2;
--
-- AUTO_INCREMENT for table `advisory_board`
--
ALTER TABLE `advisory_board`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=11;
--
-- AUTO_INCREMENT for table `authors`
--
ALTER TABLE `authors`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=118;
--
-- AUTO_INCREMENT for table `author_break`
--
ALTER TABLE `author_break`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=532;
--
-- AUTO_INCREMENT for table `available_breaks`
--
ALTER TABLE `available_breaks`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=105;
--
-- AUTO_INCREMENT for table `breaks`
--
ALTER TABLE `breaks`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=135;
--
-- AUTO_INCREMENT for table `core_team`
--
ALTER TABLE `core_team`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=19;
--
-- AUTO_INCREMENT for table `editor_picks`
--
ALTER TABLE `editor_picks`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=9;
--
-- AUTO_INCREMENT for table `subject`
--
ALTER TABLE `subject`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=8;
--
-- AUTO_INCREMENT for table `subscription`
--
ALTER TABLE `subscription`
  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=269;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
